<!DOCTYPE html SYSTEM "about:legacy-compat">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><title>MCAM - Cell surface glycoprotein MUC18 precursor - Homo sapiens (Human) - MCAM gene &amp; protein</title><meta content="IE=edge" http-equiv="X-UA-Compatible"/><meta content="text/html; charset=UTF-8" http-equiv="Content-Type"/><meta content="width=device-width, initial-scale=1" name="viewport"/><link href="/" rel="home"/><link href="http://creativecommons.org/licenses/by-nd/3.0/" rel="license"/><link type="image/vnd.microsoft.icon" href="/favicon.ico" rel="shortcut icon"/><link href="/css-compr.css2017_03" type="text/css" rel="stylesheet"/><script type="text/javascript">
			var BASE = '/';
			var image_x_inverse = '/images/x_inverse.png';
		</script><script src="/scripts/frontier/d3/d3.v3.min.js" type="text/javascript"></script><script type="text/javascript">
			var isInternal = false;
		</script><script src="/js-compr.js2017_03" type="text/javascript"></script><script type="text/javascript">
				uniprot.namespace = 'uniprot';
				onRdyFn(function(){
					if (true) {
						var shouldShowBasket = true;
						switch(uniprot.namespace) {
							case 'uniprot':
							case 'uniref' :
							case 'uniparc' :
								break;
							default:
								shouldShowBasket = uniprot.basket.getTotalBasketSize() !== 0;
							}
						if(!shouldShowBasket) {
							$('#basket').hide();
						}
					}
				},'high');
			</script><script type="text/javascript">uniprot.cookie = new Cookie('uniprot.org');</script><script type="text/javascript">
			;
		</script><link href="opensearch.xml" title="UniProtKB" type="application/opensearchdescription+xml" rel="search"/><link href="P43121.rdf" title="RDF" type="application/rdf+xml" rel="alternate"/><link href="P43121.rss?version=*" type="application/rss+xml" title="RSS" rel="alternate"/><script type="text/javascript">
				// variable to store annotation data
				var annotations = [];
			</script><script type="text/javascript">
				var uniprotEntryView;
				onRdyFn(function(){
					uniprotEntryView = new UniProtEntryView();
					uniprotEntryView.rewriteIsoformURL('P43121');
					
						uniprot.entryViews.init('P43121');
					
					uniprotEntryView.init('P43121',true);
				});
			</script><meta content="Plays a role in cell adhesion, and in cohesion of the endothelial monolayer at intercellular junctions in vascular tissue. Its expression may allow melanoma cells to interact with cellular elements of the vascular system, thereby enhancing hematogeneous tumor spread. Could be an adhesion molecule active in neural crest cells during embryonic development. Acts as surface receptor that triggers tyrosine phosphorylation of FYN and PTK2/FAK1, and a transient increase in the intracellular calcium concentration." name="description"/></head><body xmlns:up="http://purl.uniprot.org/core/" class="namespace-uniprot" typeof="schema:WebPage"><span id="evidenceToolTip" style="display:none">&#xd;
&lt;p>An evidence describes the source of an annotation, e.g. an experiment that has been published in the scientific literature, an orthologous protein, a record from another database, etc.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>&lt;a href="/manual/evidences">More…&lt;/a>&lt;/p>      </span>
			<!--[if lte IE 8]>
				<style type="text/css">
					#browserwarning {
						display:block;
					};
				</style>
			<![endif]-->
	 	<p style="display:none"><a accesskey="2" href="#content">Skip Header</a></p><div id="masthead-container"><div class="masthead" id="local-masthead"><div id="local-title"><a id="logo" accesskey="1" href="/"><img alt="" src="/images/logos/Logo_medium.png" title="UniProt home"/></a></div><div class="namespace-uniprot" id="local-search"><form method="get" action="/uniprot" id="search-form"><div id="namespace-background"><div class="searchBoxIndicator" id="searchBoxIndicator1"> </div><div onclick="location.href=&#039;/help/text-search&#039;;" class="searchBoxIndicator" id="searchBoxIndicator2"> </div><div onclick="location.href=&#039;/help/advanced_search &#039;;" class="searchBoxIndicator" id="searchBoxIndicator3"> </div><script type="text/javascript">
			onRdyFn(uniprot.searchbar.init);
			</script><a class="namespace-select" id="select-namespace" onclick="return false;" href=""><span class="caret_white" id="selected-namespace">UniProtKB</span></a><ul class="select-namespace-options"><a href="#" class="closeBox" id="closeNamespaceOptions">x</a><li><ul><li class="fixedHeight_namespaces"><h3 class="namespace_uniprot"><a class="namespace-option uniprot" href="#" id="uniprot">UniProtKB</a></h3><p>Protein knowledgebase</p></li><li class="fixedHeight_namespaces"><h3 class="namespace_uniparc"><a class="namespace-option uniparc" href="#" id="uniparc">UniParc</a></h3><p>Sequence archive</p></li><li class="fixedHeight_namespaces"><h3 class="namespace_help"><a class="namespace-option help" href="#" id="help">Help</a></h3><p>Help pages, FAQs, UniProtKB manual, documents, news archive and Biocuration projects.</p></li></ul></li><li><ul><li class="fixedHeight_namespaces"><h3 class="namespace_uniref"><a class="namespace-option uniref" href="#" id="uniref">UniRef</a></h3><p>Sequence clusters</p></li><li class="fixedHeight_namespaces"><h3 class="namespace_proteomes"><a class="namespace-option proteomes" href="#" id="proteomes">Proteomes</a></h3><p>Protein sets from fully sequenced genomes</p></li><li class="fixedHeight_namespaces"><h3 class="namespace_unirule">Annotation systems</h3><p class="supportingLeadingText">Systems used to automatically annotate proteins with high accuracy:</p><ul class="supportingDataOptions"><li><a class="namespace-option supporting" href="#" id="unirule">UniRule (Expertly curated rules)</a></li><li><a class="namespace-option supporting" href="#" id="saas">SAAS (System generated rules)</a></li></ul></li></ul></li><li><ul><li class="fixedHeight_namespaces"><h3 class="supporting"><span>Supporting data</span></h3><p class="supportingLeadingText">Select one of the options below to target your search:</p><ul class="supportingDataOptions"><li><a class="namespace-option supporting" href="#" id="citations">Literature citations</a></li><li><a class="namespace-option supporting" href="#" id="taxonomy">Taxonomy</a></li><li><a class="namespace-option supporting" href="#" id="keywords">Keywords</a></li><li><a class="namespace-option supporting" href="#" id="locations">Subcellular locations</a></li><li><a class="namespace-option supporting" href="#" id="database">Cross-referenced databases</a></li><li><a class="namespace-option supporting" href="#" id="diseases">Human diseases</a></li></ul></li></ul></li></ul><input value="" id="topQuery" type="hidden"/><input id="query" value="" accesskey="4" name="query" type="search"/><a class="caret_grey" href="#" id="advanced-search-toggle">Advanced</a><input value="score" name="sort" type="hidden"/><a id="search-button" title="Search" data-icon="1" class="icon icon-functional button" onclick="return false" href="">Search</a><script type="text/javascript">
			onRdyFn(function() {
				nameSpaceSelector();
				uniprot.advSearchUI.template(uniprot.namespace);
			});
			</script><div class="advSearch" id="query-builder-container"><div style="padding-top:1%" class="advSearchHeader"><a href="#" class="closeBox" id="closeAdvanceSearch">x</a>
					Searching in <span id="namespaceInAdv"></span></div><div style="margin-top:2%" class="filterList" id="filterList"></div><div class="advButtons"><a data-icon="1" href="#" class="icon icon-functional button" id="advSearchButton"></a></div></div></div></form></div><div id="nav"><ul id="local-nav"><li class="mobileHomeLink"><a href="/">Home</a></li><li class="first"><a class="tooltipped" title="Find regions of local similarity between sequences" onclick="return uniprot.blast.checkSelectionsAndSubmit();" href="/blast">BLAST</a></li><li><a class="tooltipped" title="Align two or more protein sequences with the Clustal Omega program" onclick="return uniprot.align.checkSelectionsAndSubmit();" accesskey="9" href="/align">Align</a></li><li class="last"><a class="tooltipped" title="Submit a list of identifiers to retrieve the corresponding UniProt entries, or to map them from or to an external database" rel="help" onclick="return (new upload()).uploadLinkHandler();" href="/uploadlists" id="upload-lists-nav">Retrieve/ID mapping</a></li><li class="last"><a class="tooltipped" title="Submit a list of peptides to find UniProt sequences containing them" rel="help" href="/peptidesearch" id="peptidesearch-lists-nav">Peptide search</a></li><li class="functional last"><a accesskey="9" href="/contact" id="contact">Contact</a></li><li class="functional"><a rel="help" href="/help/" id="help">Help</a></li></ul></div></div></div><div class="warn" id="browserwarning">You are using a version of browser that may not display all the features of this website. Please consider upgrading <a href="/help/browser_support">your browser</a>.</div><main class="uniprot" property="schema:about" resource="http://purl.uniprot.org/uniprot/P43121" typeof="schema:CreativeWork" id="content"><section id="page-header"><h2 class="page-title" property="schema:alternateName">UniProtKB - P43121
			<span property="schema:alternateName">(MUC18_HUMAN)</span></h2><div id="top-right-actions"><div id="basket"><a href="#" data-icon="b" class="closed basket-empty icon icon-static caret_white button" id="basket-list">Basket <span id="basket-count">0</span></a></div><div style="z-index: 3;" id="basket-contents"><div id="closeBasket"><span class="basketText">(max 400 entries)</span><a class="closeBox" href="#">x</a></div><div id="emptyBasketContent"><h3>Your basket is currently empty.</h3><p>Select item(s) and click on "Add to basket" to create your own collection here<br/> (400 entries max)<br/></p><div id="emptyBasketContentHelp"></div></div><div style="display:none" id="loadingBasketContent"></div><div id="filledBasketContents"><ul class="tabs"><li><a class="uniprot tab">UniProtKB (<span class="uniprot-basket-count">0</span>)</a></li><li><a class="uniref tab">UniRef (<span class="uniref-basket-count">0</span>)</a></li><li><a class="uniparc tab">UniParc (<span class="uniparc-basket-count">0</span>)</a></li></ul><div class="tabsContent" id="basket-uniprot"></div><div class="tabsContent" id="basket-uniref"></div><div class="tabsContent" id="basket-uniparc"></div></div><div id="basket-actions"><ul class="action-buttons-grouped"><li><a title="Select two or more entries and click here to align them" data-icon="i" class="button icon icon-functional align disabled" href="#" id="basket-align-button">Align</a></li><li><a data-icon="t" title="Select an entry and click here to run a BLAST search on it" class="button icon-functional icon blast disabled" href="#" id="basket-blast-button">BLAST</a></li><li><a title="Map ids to other databases" class="button icon-functional disabled" id="basket-map-button">Map Ids</a></li><li><a data-icon="=" title="Download results" class="button icon icon-functional download disabled" href="#" id="basket-download-button">Download</a></li><li style="float: right;"><a title="Full View" class="button fullView disabled" id="basket-view-button">Full View</a></li><li style="float: right;"><a title="Remove Selected entries" class="button remove disabled" id="basket-remove-button">Remove</a></li><li style="float: right"><a title="Clear basket" class="button clear disabled" href="#">Clear</a></li></ul></div></div><form action="/align/" method="post" id="basket-alignForm"><input value="no" name="landingPage" type="hidden"/><input value="clustalo" name="program" type="hidden"/><input value="yes" name="redirect" type="hidden"/><input value="" name="alignQuery" id="basket-alignQuery" type="hidden"/></form><form action="/blast/" method="post" id="basket-blastForm"><input value="yes" name="redirect" type="hidden"/><input value="no" name="landingPage" type="hidden"/><input value="uniprotkb" name="dataset" type="hidden"/><input value="10" name="threshold" type="hidden"/><input value="" name="matrix" type="hidden"/><input value="false" name="filter" type="hidden"/><input value="true" name="gapped" type="hidden"/><input value="250" name="numal" type="hidden"/><input value="" id="basket-blastQuery" name="blastQuery" type="hidden"/></form><form action="" method="post" id="basket-downloadForm"><input value="" name="query" type="hidden"/></form></div></section> <aside id="entrySidebar" class="sidebar displayThis"><div class="viewBy"><h3>Display</h3><ul><li class="active no-close" id="display-entry"><a class="icon-doc-text" href="/uniprot/P43121">Entry</a></li><li class="no-close" id="display-entry"><a class="icon-doc-publications" href="/uniprot/P43121/publications">Publications</a></li><li class="no-close" id="display-fv"><a class="icon-chart-area" href="#showFeaturesViewer">Feature viewer</a></li><li class="no-close" id="display-ft"><a class="icon-table" id="show-feature-button" href="#showFeaturesTable">Feature table</a></li></ul></div><div class="sidebarsubsection"><hr/><div class="sidebar-h-links"><a id="all">All</a> <a id="none">None</a></div><ul class="swissprot" id="entry_sections"><li class="default active"><a id="jp_function" href="#function" class="heading"><input class="toggle-section" id="tg_function" type="checkbox" checked="checked"/><span class="entryview-name-nav" id="function_name">Function</span></a></li><li class="default active"><a id="jp_names_and_taxonomy" href="#names_and_taxonomy" class="heading"><input class="toggle-section" id="tg_names_and_taxonomy" type="checkbox" checked="checked"/><span class="entryview-name-nav" id="names_and_taxonomy_name">Names &amp; Taxonomy</span></a></li><li class="default active"><a id="jp_subcellular_location" href="#subcellular_location" class="heading"><input class="toggle-section" id="tg_subcellular_location" type="checkbox" checked="checked"/><span class="entryview-name-nav" id="subcellular_location_name">Subcellular location</span><span id="subcellular_location_name_small" class="entryview-name-nav tooltipped">Subcell. location</span><style>#subcellular_location_name_small {display:none;}@media screen and (max-width: 1180px) {#subcellular_location_name {display:none;}#subcellular_location_name_small {display:inline;}}</style></a></li><li class="default active"><a id="jp_pathology_and_biotech" href="#pathology_and_biotech" class="heading"><input class="toggle-section" id="tg_pathology_and_biotech" type="checkbox" checked="checked"/><span class="entryview-name-nav" id="pathology_and_biotech_name">Pathology &amp; Biotech</span><span id="pathology_and_biotech_name_small" class="entryview-name-nav tooltipped">Pathol./Biotech</span><style>#pathology_and_biotech_name_small {display:none;}@media screen and (max-width: 1180px) {#pathology_and_biotech_name {display:none;}#pathology_and_biotech_name_small {display:inline;}}</style></a></li><li class="default active"><a id="jp_ptm_processing" href="#ptm_processing" class="heading"><input class="toggle-section" id="tg_ptm_processing" type="checkbox" checked="checked"/><span class="entryview-name-nav" id="ptm_processing_name">PTM / Processing</span></a></li><li class="default active"><a id="jp_expression" href="#expression" class="heading"><input class="toggle-section" id="tg_expression" type="checkbox" checked="checked"/><span class="entryview-name-nav" id="expression_name">Expression</span></a></li><li class="default active"><a id="jp_interaction" href="#interaction" class="heading"><input class="toggle-section" id="tg_interaction" type="checkbox" checked="checked"/><span class="entryview-name-nav" id="interaction_name">Interaction</span></a></li><li class="default active"><a id="jp_structure" href="#structure" class="heading"><input class="toggle-section" id="tg_structure" type="checkbox" checked="checked"/><span class="entryview-name-nav" id="structure_name">Structure</span></a></li><li class="default active"><a id="jp_family_and_domains" href="#family_and_domains" class="heading"><input class="toggle-section" id="tg_family_and_domains" type="checkbox" checked="checked"/><span class="entryview-name-nav" id="family_and_domains_name">Family &amp; Domains</span></a></li><li class="default active"><a id="jp_sequences" href="#sequences" class="heading"><input class="toggle-section" id="tg_sequences" type="checkbox" checked="checked"/><span class="entryview-name-nav" id="sequences_name">Sequences (2)</span></a></li><li class="default active"><a id="jp_cross_references" href="#cross_references" class="heading"><input class="toggle-section" id="tg_cross_references" type="checkbox" checked="checked"/><span class="entryview-name-nav" id="cross_references_name">Cross-references</span></a></li><li class="default active"><a id="jp_entry_information" href="#entry_information" class="heading"><input class="toggle-section" id="tg_entry_information" type="checkbox" checked="checked"/><span class="entryview-name-nav" id="entry_information_name">Entry information</span></a></li><li class="default active"><a id="jp_miscellaneous" href="#miscellaneous" class="heading"><input class="toggle-section" id="tg_miscellaneous" type="checkbox" checked="checked"/><span class="entryview-name-nav" id="miscellaneous_name">Miscellaneous</span></a></li><li class="default active"><a id="jp_similar_proteins" href="#similar_proteins" class="heading"><input class="toggle-section" id="tg_similar_proteins" type="checkbox" checked="checked"/><span class="entryview-name-nav" id="similar_proteins_name">Similar proteins</span></a></li></ul><a class="caret_top" id="scrolltotop">Top</a></div></aside><div class="main-aside"><input value="P43121" id="entry_id" type="hidden"/><ul class="action-buttons-grouped decorated" id="resultActionButtons"><li><a data-icon="t" class="icon icon-functional button tooltipped" title="BLAST against UniProtKB" id="blastButton" href="#">BLAST</a><form action="/blast/" method="post" id="blastForm"><input value="yes" name="redirect" type="hidden"/><input value="no" name="landingPage" type="hidden"/><input value="uniprotkb" name="dataset" type="hidden"/><input value="10" name="threshold" type="hidden"/><input value="" name="matrix" type="hidden"/><input value="false" name="filter" type="hidden"/><input value="true" name="gapped" type="hidden"/><input value="250" name="numal" type="hidden"/><input value="P43121" id="blastQuery" name="blastQuery" type="hidden"/></form><span id="entrySequence" style="display:none">&gt;sp|P43121|MUC18_HUMAN Cell surface glycoprotein MUC18 OS=Homo sapiens GN=MCAM PE=1 SV=2
MGLPRLVCAFLLAACCCCPRVAGVPGEAEQPAPELVEVEVGSTALLKCGLSQSQGNLSHV
DWFSVHKEKRTLIFRVRQGQGQSEPGEYEQRLSLQDRGATLALTQVTPQDERIFLCQGKR
PRSQEYRIQLRVYKAPEEPNIQVNPLGIPVNSKEPEEVATCVGRNGYPIPQVIWYKNGRP
LKEEKNRVHIQSSQTVESSGLYTLQSILKAQLVKEDKDAQFYCELNYRLPSGNHMKESRE
VTVPVFYPTEKVWLEVEPVGMLKEGDRVEIRCLADGNPPPHFSISKQNPSTREAEEETTN
DNGVLVLEPARKEHSGRYECQGLDLDTMISLLSEPQELLVNYVSDVRVSPAAPERQEGSS
LTLTCEAESSQDLEFQWLREETGQVLERGPVLQLHDLKREAGGGYRCVASVPSIPGLNRT
QLVNVAIFGPPWMAFKERKVWVKENMVLNLSCEASGHPRPTISWNVNGTASEQDQDPQRV
LSTLNVLVTPELLETGVECTASNDLGKNTSILFLELVNLTTLTPDSNTTTGLSTSTASPH
TRANSTSTERKLPEPESRGVVIVAVIVCILVLAVLGAVLYFLYKKGKLPCRRSGKQEITL
PPSRKSELVVEVKSDKLPEEMGLLQGSSGDKRAPGDQGEKYIDLRH
</span></li><li><a data-icon="i" class="icon icon-functional button tooltipped" onclick="$(&#039;#alignForm&#039;).submit(); return false" title="Align this entry with its isoforms" href="#" id="alignIsoforms">Align</a></li><form action="/align/" method="post" id="alignForm"><input value="no" name="landingPage" type="hidden"/><input value="clustalo" name="program" type="hidden"/><input value="yes" name="redirect" type="hidden"/><input value="P43121
P43121-2" name="alignQuery" id="alignQuery" type="hidden"/></form><li><a href="" data-icon="b" class="icon icon-functional button tooltipped" title="View this entry in a different format" id="download-button">Format</a></li><li><a href="" data-icon="b" class="icon icon-static button tooltipped add-basket-button P43121" title="Add this entry to your basket" id="add-entry-basket">Add to basket</a><a href="" data-icon="b" style="display:none" class="icon icon-static button disabled added-basket-button P43121" title="This entry is in your basket" id="added-entry-basket">Added to basket</a></li><li><a href="" data-icon="{" class="icon icon-generic button tooltipped" title="View this entry&#039;s history" id="historyButton">History</a><div id="entry-history-content-container"><span class="version">Entry version 153 (15 Mar 2017)</span><br/><span class="version">Sequence version 2 (10 Jan 2006)</span><br/><a id="previousVersions" href="?version=*">Previous versions</a><span> | </span><a href="P43121.rss?version=*">rss</a></div></li><li class="icon-right"><a data-icon="V" class="icon icon-generic namespaceIcons" target="_blank" href="https://www.youtube.com/channel/UCkCR5RJZCZZoVTQzTYY92aw">Other tutorials and videos</a></li><li class="icon-right"><a data-icon="V" class="icon icon-generic helpVideo" target="_blank" href="https://www.youtube.com/watch?v=x9GNm2DLP-U">Help video</a></li><li class="icon-right" id="feedback"><a data-icon="U" class="icon icon-static tooltipped" href="../update?entry=P43121" title="send updates or corrections" rel="nofollow">Feedback</a></li></ul><div><div style="display: none" id="mainContentLoader" title="Loading" class="loader"><svg xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg" xml:space="preserve" style="enable-background:new 0 0 50 50;" viewBox="0 0 50 50" height="40" width="40" y="0px" x="0px" id="loader-1" version="1.1"><path d="M43.935,25.145c0-10.318-8.364-18.683-18.683-18.683c-10.318,0-18.683,8.365-18.683,18.683h4.068c0-8.071,6.543-14.615,14.615-14.615c8.072,0,14.615,6.543,14.615,14.615H43.935z" fill="#000"><animateTransform repeatCount="indefinite" dur="0.6s" to="360 25 25" from="0 25 25" type="rotate" attributeName="transform" attributeType="xml"/></path></svg></div></div><div class="content entry_view_content up_entry swissprot"><div id="entry-overview"><div><div class="entry-overview-header">Protein</div><div id="content-protein" class="entry-overview-content"><h1 property="schema:name">Cell surface glycoprotein MUC18</h1></div></div><div><div class="entry-overview-header">Gene</div><div id="content-gene" class="entry-overview-content"><h2>MCAM</h2></div></div><div><div class="entry-overview-header">Organism</div><div id="content-organism" class="entry-overview-content"><em>Homo sapiens (Human)</em></div></div><div><div id="content-status-header" class="entry-overview-header entry-header">Status</div><div id="content-status" class="entry-overview-content"><a title="Reviewed (Swiss-Prot)" data-icon="s" class="icon-uniprot reviewed-icon tooltipped" href="/manual/entry_status"><span>Reviewed</span></a><span class="sep">-</span><span>Annotation score: </span><span class="context-help bin-score tooltipped-click mouseOver"><svg width="66px" height="12px"><title>Annotation score: 5 out of 5</title><defs><circle id="o" r="5px" cy="6px" stroke="#00709B" fill="white" cx="8px"></circle><circle id="i" r="3.5px" cy="6px" fill="#00709B"></circle></defs><use xlink:href="#o" x="0px"></use><use xlink:href="#o" x="12px"></use><use xlink:href="#o" x="24px"></use><use xlink:href="#o" x="36px"></use><use xlink:href="#o" x="48px"></use><use xlink:href="#i" x="8px"></use><use xlink:href="#i" x="20px"></use><use xlink:href="#i" x="32px"></use><use xlink:href="#i" x="44px"></use><use xlink:href="#i" x="56px"></use></svg><span class="toolTipContent" style="display:none"><p>Annotation score:5 out of 5</p>
&lt;p>The annotation score provides a heuristic measure of the annotation content of a UniProtKB entry or proteome.&lt;p>&lt;a href='/help/annotation_score' target='_top'>More...&lt;/a>&lt;/p></span></span><span class="context-help tooltipped-click"><span class="sep">-</span>Experimental evidence at protein level<sup>i</sup><span class="toolTipContent" style="display:none">
&lt;p>This indicates the type of evidence that supports the existence of the protein. Note that the ‘protein existence’ evidence does not give information on the accuracy or correctness of the sequence(s) displayed.&lt;p>&lt;a href='/help/protein_existence' target='_top'>More...&lt;/a>&lt;/p></span></span></div></div></div><div style="display:none" id="closedSectionsMsg"><h4>Select a section on the left to see content.</h4></div><div class="section " id="function"><h2><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This section provides any useful information about the protein, mostly biological knowledge.&lt;p>&lt;a href='/help/function_section' target='_top'>More...&lt;/a>&lt;/p></span>Function<sup>i</sup></span></h2><div class="annotation" property="schema:hasPart" typeof="schema:CreativeWork"><span property="schema:text">Plays a role in cell adhesion, and in cohesion of the endothelial monolayer at intercellular junctions in vascular tissue. Its expression may allow melanoma cells to interact with cellular elements of the vascular system, thereby enhancing hematogeneous tumor spread. Could be an adhesion molecule active in neural crest cells during embryonic development. Acts as surface receptor that triggers tyrosine phosphorylation of FYN and PTK2/FAK1, and a transient increase in the intracellular calcium concentration.<span class="attribution ECO269"><span class="attributionHeader ">2 Publications<span class="showHideEvidence caret_grey displayThisInline"></span></span><span style="display:none" class="evidenceContainer"><p class="attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Manually curated information for which there is published experimental evidence.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>&lt;a href="/manual/evidences#ECO:0000269">More…&lt;/a>&lt;/p>      </span>Manual assertion based on experiment in<sup>i</sup></span></p><ul><li><div class="P43121#ref9 referenceAttribution"><div class="reference_header">Ref.9</div><div class="reference_content"><div property="schema:citation" resource="http://purl.uniprot.org/citations/8292890" typeof="schema:ScholarlyArticle"><strong property="schema:name">"The progression associated antigen MUC18: a unique member of the immunoglobulin supergene family."</strong><br/><a href="/uniprot/?query=author:%22Johnson+J.P.%22" rel="nofollow">Johnson J.P.</a>, <a href="/uniprot/?query=author:%22Rothbacher+U.%22" rel="nofollow">Rothbacher U.</a>, <a href="/uniprot/?query=author:%22Sers+C.%22" rel="nofollow">Sers C.</a><br/>Melanoma Res. 3:337-340(1993)  [<a property="schema:sameAs" href="https://www.ncbi.nlm.nih.gov/pubmed/8292890">PubMed</a>] [<a property="schema:sameAs" href="http://europepmc.org/abstract/MED/8292890">Europe PMC</a>] [<a href="/citations/8292890">Abstract</a>]</div><div class="citedFor"><span class="details"><strong>Cited for:</strong></span> FUNCTION.</div></div></div></li><li><div class="P43121#ref10 referenceAttribution"><div class="reference_header">Ref.10</div><div class="reference_content"><div property="schema:citation" resource="http://purl.uniprot.org/citations/11036077" typeof="schema:ScholarlyArticle"><strong property="schema:name">"Outside-in signaling pathway linked to CD146 engagement in human endothelial cells."</strong><br/><a href="/uniprot/?query=author:%22Anfosso+F.%22" rel="nofollow">Anfosso F.</a>, <a href="/uniprot/?query=author:%22Bardin+N.%22" rel="nofollow">Bardin N.</a>, <a href="/uniprot/?query=author:%22Vivier+E.%22" rel="nofollow">Vivier E.</a>, <a href="/uniprot/?query=author:%22Sabatier+F.%22" rel="nofollow">Sabatier F.</a>, <a href="/uniprot/?query=author:%22Sampol+J.%22" rel="nofollow">Sampol J.</a>, <a href="/uniprot/?query=author:%22Dignat-George+F.%22" rel="nofollow">Dignat-George F.</a><br/><a href="http://dx.doi.org/10.1074/jbc.M007065200">J. Biol. Chem. 276:1564-1569(2001)</a>  [<a property="schema:sameAs" href="https://www.ncbi.nlm.nih.gov/pubmed/11036077">PubMed</a>] [<a property="schema:sameAs" href="http://europepmc.org/abstract/MED/11036077">Europe PMC</a>] [<a href="/citations/11036077">Abstract</a>]</div><div class="citedFor"><span class="details"><strong>Cited for:</strong></span> FUNCTION.</div></div></div></li></ul></span></span></span></div><h4><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>The &lt;a href="http://www.geneontology.org/">Gene Ontology (GO)&lt;/a> project provides a set of hierarchical controlled vocabulary split into 3 categories:&lt;p>&lt;a href='/help/gene_ontology' target='_top'>More...&lt;/a>&lt;/p></span>GO - Biological process<sup>i</sup></span></h4><ul class="noNumbering biological_process"><li><a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009653" onclick="uniprot.analytics('UniProt-Entry-View', 'click', 'Display-GO-Term');">anatomical structure morphogenesis</a><span class="attribution ECO304"><span class="attributionHeader"> Source: ProtInc<span class="showHideEvidence caret_grey displayThisInline"></span></span><span class="gosource evidenceContainer" style="display:none"><span class="goa attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Traceable Author Statement&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>Used for information from review articles where the original experiments are traceable through that article and also for information from text books or dictionaries.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>More information in the &lt;a href="http://geneontology.org/page/guide-go-evidence-codes#tas">GO evidence code guide&lt;/a>&lt;/p>      </span>Traceable author statement<sup>i</sup></span></span><ul><li><a href="/citations/8378324" class="pubmed ">8378324</a></li></ul></span></span></li><li><a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0001525" onclick="uniprot.analytics('UniProt-Entry-View', 'click', 'Display-GO-Term');">angiogenesis</a><span class="attribution ECO314"><span class="attributionHeader"> Source: MGI<span class="showHideEvidence caret_grey displayThisInline"></span></span><span class="gosource evidenceContainer" style="display:none"><span class="goa attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Inferred from Direct Assay&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>Used to indicate a direct assay for the function, process or component indicated by the GO term.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>More information in the &lt;a href="http://geneontology.org/page/guide-go-evidence-codes#ida">GO evidence code guide&lt;/a>&lt;/p>      </span>Inferred from direct assay<sup>i</sup></span></span><ul><li><a href="/citations/23878390" class="pubmed ">23878390</a></li></ul></span></span></li><li><a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0007155" onclick="uniprot.analytics('UniProt-Entry-View', 'click', 'Display-GO-Term');">cell adhesion</a><span class="attribution ECO304"><span class="attributionHeader"> Source: ProtInc<span class="showHideEvidence caret_grey displayThisInline"></span></span><span class="gosource evidenceContainer" style="display:none"><span class="goa attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Traceable Author Statement&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>Used for information from review articles where the original experiments are traceable through that article and also for information from text books or dictionaries.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>More information in the &lt;a href="http://geneontology.org/page/guide-go-evidence-codes#tas">GO evidence code guide&lt;/a>&lt;/p>      </span>Traceable author statement<sup>i</sup></span></span><ul><li><a href="/citations/8378324" class="pubmed ">8378324</a></li></ul></span></span></li><li><a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003094" onclick="uniprot.analytics('UniProt-Entry-View', 'click', 'Display-GO-Term');">glomerular filtration</a><span class="attribution ECO270"><span class="attributionHeader"> Source: UniProtKB<span class="showHideEvidence caret_grey displayThisInline"></span></span><span class="gosource evidenceContainer" style="display:none"><span class="goa attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Inferred from Expression Pattern&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>Covers cases where the annotation is inferred from the timing or location of expression of a gene.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>More information in the &lt;a href="http://geneontology.org/page/guide-go-evidence-codes#iep">GO evidence code guide&lt;/a>&lt;/p>      </span>Inferred from expression pattern<sup>i</sup></span></span><ul><li><a href="/citations/20354148" class="pubmed ">20354148</a></li></ul></span></span></li><li><a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0030335" onclick="uniprot.analytics('UniProt-Entry-View', 'click', 'Display-GO-Term');">positive regulation of cell migration</a><span class="attribution ECO501"><span class="attributionHeader tooltipped" title="Inferred from electronic annotation"> Source: Ensembl</span></span></li><li><a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0061042" onclick="uniprot.analytics('UniProt-Entry-View', 'click', 'Display-GO-Term');">vascular wound healing</a><span class="attribution ECO270"><span class="attributionHeader"> Source: UniProtKB<span class="showHideEvidence caret_grey displayThisInline"></span></span><span class="gosource evidenceContainer" style="display:none"><span class="goa attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Inferred from Expression Pattern&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>Covers cases where the annotation is inferred from the timing or location of expression of a gene.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>More information in the &lt;a href="http://geneontology.org/page/guide-go-evidence-codes#iep">GO evidence code guide&lt;/a>&lt;/p>      </span>Inferred from expression pattern<sup>i</sup></span></span><ul><li><a href="/citations/20354148" class="pubmed ">20354148</a></li></ul></span></span></li></ul><div class="gocomplete"><a href="http://www.ebi.ac.uk/QuickGO/GProtein?ac=P43121" onclick="uniprot.analytics('UniProt-Entry-View', 'click', 'Complete-GO-Annotation');">Complete GO annotation...</a></div><h4><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>UniProtKB Keywords constitute a &lt;a href="http://www.uniprot.org/keywords">controlled vocabulary&lt;/a> with a hierarchical structure. Keywords summarise the content of a UniProtKB entry and facilitate the search for proteins of interest.&lt;p>&lt;a href='/help/keywords' target='_top'>More...&lt;/a>&lt;/p></span>Keywords<sup>i</sup></span></h4><table class="databaseTable"><tr><td>Biological process</td><td><span><a href="/keywords/KW-0130">Cell adhesion</a></span></td></tr></table></div><div class="section " id="names_and_taxonomy"><h2><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This section provides information about the protein and gene name(s) and synonym(s) and about the organism that is the source of the protein sequence.&lt;p>&lt;a href='/help/names_and_taxonomy_section' target='_top'>More...&lt;/a>&lt;/p></span>Names &amp; Taxonomy<sup>i</sup></span></h2><table class="databaseTable"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Names and Taxonomy’ section provides an exhaustive list of all names of the protein, from commonly used to obsolete, to allow unambiguous identification of a protein.&lt;p>&lt;a href='/help/protein_names' target='_top'>More...&lt;/a>&lt;/p></span>Protein names<sup>i</sup></span></td><td><div><div><span title="The name recommended by the UniProt consortium." class="inner-title">Recommended name:</span></div><span property="schema:name" class="recommended-name"><strong>Cell surface glycoprotein MUC18</strong></span><div><span title="A synonym of the recommended name." class="inner-title">Alternative name(s):</span></div><div><div property="schema:alternateName" class="listify">Cell surface glycoprotein P1H12</div><div property="schema:alternateName" class="listify">Melanoma cell adhesion molecule</div><div property="schema:alternateName" class="listify">Melanoma-associated antigen A32</div><div property="schema:alternateName" class="listify">Melanoma-associated antigen MUC18</div><div property="schema:alternateName" class="listify">S-endo 1 endothelial-associated antigen</div><div property="schema:alternateName" class="listify"><span title="Human cell differentiation molecule." class="inner-title">CD_antigen: </span>CD146</div></div></div></td></tr><tr><td id="gene_name_table"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Names and taxonomy’ section indicates the name(s) of the gene(s) that code for the protein sequence(s) described in the entry. Four distinct tokens exist: ‘Name’, ‘Synonyms’, ‘Ordered locus names’ and ‘ORF names’.&lt;p>&lt;a href='/help/gene_name' target='_top'>More...&lt;/a>&lt;/p></span>Gene names<sup>i</sup></span></td><td class="noBorders"><div class="inner-title" property="schema:mentions" typeof="schema:CreativeWork">Name:<strong property="schema:name"><a href="http://www.genenames.org/cgi-bin/gene_symbol_report?hgnc_id=HGNC:6934">MCAM</a></strong><div><span>Synonyms:</span><span property="schema:alternateName">MUC18</span></div></div></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Names and taxonomy’ section provides information on the name(s) of the organism that is the source of the protein sequence.&lt;p>&lt;a href='/help/organism-name' target='_top'>More...&lt;/a>&lt;/p></span>Organism<sup>i</sup></span></td><td><strong><a href="/taxonomy/9606">Homo sapiens (Human)</a></strong></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Names and taxonomy’ section shows the unique identifier assigned by the &lt;span class="caps">NCBI&lt;/span> to the source organism of the protein. This is known as the ‘taxonomic identifier’ or ‘taxid’.&lt;p>&lt;a href='/help/taxonomic_identifier' target='_top'>More...&lt;/a>&lt;/p></span>Taxonomic identifier<sup>i</sup></span></td><td><a property="schema:mentions" href="/taxonomy/9606">9606</a>  [<a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?lvl=0&amp;id=9606">NCBI</a>]</td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Names and taxonomy’ section contains the taxonomic hierarchical classification lineage of the source organism. It lists the nodes as they appear top-down in the taxonomic tree, with the more general grouping listed first.&lt;p>&lt;a href='/help/taxonomic_lineage' target='_top'>More...&lt;/a>&lt;/p></span>Taxonomic lineage<sup>i</sup></span></td><td><span style="display:none" class="hiddenTaxon"><a href="/taxonomy/131567" style="color:gray">cellular organisms</a> › </span><a href="/taxonomy/2759">Eukaryota</a> › <span style="display:none" class="hiddenTaxon"><a href="/taxonomy/33154" style="color:gray">Opisthokonta</a> › </span><a href="/taxonomy/33208">Metazoa</a> › <span style="display:none" class="hiddenTaxon"><a href="/taxonomy/6072" style="color:gray">Eumetazoa</a> › </span><span style="display:none" class="hiddenTaxon"><a href="/taxonomy/33213" style="color:gray">Bilateria</a> › </span><span style="display:none" class="hiddenTaxon"><a href="/taxonomy/33511" style="color:gray">Deuterostomia</a> › </span><a href="/taxonomy/7711">Chordata</a> › <a href="/taxonomy/89593">Craniata</a> › <a href="/taxonomy/7742">Vertebrata</a> › <span style="display:none" class="hiddenTaxon"><a href="/taxonomy/7776" style="color:gray">Gnathostomata</a> › </span><span style="display:none" class="hiddenTaxon"><a href="/taxonomy/117570" style="color:gray">Teleostomi</a> › </span><a href="/taxonomy/117571">Euteleostomi</a> › <span style="display:none" class="hiddenTaxon"><a href="/taxonomy/8287" style="color:gray">Sarcopterygii</a> › </span><span style="display:none" class="hiddenTaxon"><a href="/taxonomy/1338369" style="color:gray">Dipnotetrapodomorpha</a> › </span><span style="display:none" class="hiddenTaxon"><a href="/taxonomy/32523" style="color:gray">Tetrapoda</a> › </span><span style="display:none" class="hiddenTaxon"><a href="/taxonomy/32524" style="color:gray">Amniota</a> › </span><a href="/taxonomy/40674">Mammalia</a> › <span style="display:none" class="hiddenTaxon"><a href="/taxonomy/32525" style="color:gray">Theria</a> › </span><a href="/taxonomy/9347">Eutheria</a> › <span style="display:none" class="hiddenTaxon"><a href="/taxonomy/1437010" style="color:gray">Boreoeutheria</a> › </span><a href="/taxonomy/314146">Euarchontoglires</a> › <a href="/taxonomy/9443">Primates</a> › <a href="/taxonomy/376913">Haplorrhini</a> › <span style="display:none" class="hiddenTaxon"><a href="/taxonomy/314293" style="color:gray">Simiiformes</a> › </span><a href="/taxonomy/9526">Catarrhini</a> › <span style="display:none" class="hiddenTaxon"><a href="/taxonomy/314295" style="color:gray">Hominoidea</a> › </span><a href="/taxonomy/9604">Hominidae</a> › <span style="display:none" class="hiddenTaxon"><a href="/taxonomy/207598" style="color:gray">Homininae</a> › </span><a href="/taxonomy/9605">Homo</a> <div title="Show complete lineage" class="clickable taxonomy icon_expand"></div></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the “Names and Taxonomy” section is present for entries that are part of a &lt;a href="http://www.uniprot.org/proteomes">proteome&lt;/a>, i.e. of a set of proteins thought to be expressed by organisms whose genomes have been completely sequenced.&lt;p>&lt;a href='/help/proteomes_manual' target='_top'>More...&lt;/a>&lt;/p></span>Proteomes<sup>i</sup></span></td><td><ul><li><a href="/proteomes/UP000005640">UP000005640</a><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>A UniProt &lt;a href="http://www.uniprot.org/manual/proteomes_manual">proteome&lt;/a> can consist of several components. &lt;br>&lt;/br>The component name refers to the genomic component encoding a set of proteins. &lt;br>&lt;/br>These range from a single component such as Viral genomes to several components as in the case of eukaryotic chromosomes. They may also represent different stages in a genome project and include components such as contigs, scaffolds or Whole Genome Shotgun (WGS) master records.&lt;p>&lt;a href='/help/proteome_component' target='_top'>More...&lt;/a>&lt;/p></span> Component<sup>i</sup></span>: Chromosome 11 </li></ul></td></tr></table><h4 id="section_x-ref_organism" class="xref_category">Organism-specific databases</h4><table class="databaseTable ORGANISM"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Human Gene Nomenclature Database</p><a href="/database/DB-0042">More...</a></span>HGNC<sup>i</sup></span></td><td><a href="http://www.genenames.org/cgi-bin/gene_symbol_report?hgnc_id=HGNC:6934" onclick="uniprot.analytics('DR-lines', 'click', 'DR-HGNC');">HGNC:6934.</a> MCAM. <br/></td></tr></table></div><div class="section " id="subcellular_location"><h2><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This section provides information on the location and the topology of the mature protein in the cell.&lt;p>&lt;a href='/help/subcellular_location_section' target='_top'>More...&lt;/a>&lt;/p></span>Subcellular location<sup>i</sup></span></h2><div class="annotation"><ul class="noNumbering subcellLocations"><li><a href="/locations/SL-0162">Membrane</a>; <a href="/locations/SL-9905">Single-pass type I membrane protein</a></li></ul></div><h4>Topology</h4><table class="featureTable" id="topology_section"><tr><th class="featkey_h">Feature key</th><th class="pos_h">Position(s)</th><th class="description_h">Description<span class="featactions_h">Actions</span></th><th class="graphview_h">Graphical view</th><th class="length_h">Length</th></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;a href="http://www.uniprot.org/help/subcellular_location_section">‘Subcellular location’&lt;/a> section describes the subcellular compartment where each non-membrane region of a membrane-spanning protein is found.&lt;p>&lt;a href='/help/topo_dom' target='_top'>More...&lt;/a>&lt;/p></span>Topological domain<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[24-559]&amp;key=Topological domain">24 – 559</a></td><td class="featdescription"><span property="text">Extracellular</span><span class="attribution ECO255"><span class="attributionHeader tooltipped" title="Manual assertion according to rules">Sequence analysis</span></span><span class="featactions"><a class="icon icon-static tooltipped subsequences" data-icon="b" id="P43121_24-559_" title="Add to basket">Add</a> <a class="icon icon-functional tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">BLAST</a></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="3.560%" width="82.972%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">536</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;a href="http://www.uniprot.org/help/subcellular_location_section">‘Subcellular location’&lt;/a> section describes the extent of a membrane-spanning region of the protein. It denotes the presence of both alpha-helical transmembrane regions and the membrane spanning regions of beta-barrel transmembrane proteins.&lt;p>&lt;a href='/help/transmem' target='_top'>More...&lt;/a>&lt;/p></span>Transmembrane<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[560-583]&amp;key=Transmembrane">560 – 583</a></td><td class="featdescription"><span property="text">Helical</span><span class="attribution ECO255"><span class="attributionHeader tooltipped" title="Manual assertion according to rules">Sequence analysis</span></span><span class="featactions"><a class="icon icon-static tooltipped subsequences" data-icon="b" id="P43121_560-583_" title="Add to basket">Add</a> <a class="icon icon-functional tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">BLAST</a></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="86.532%" width="3.715%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">24</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;a href="http://www.uniprot.org/help/subcellular_location_section">‘Subcellular location’&lt;/a> section describes the subcellular compartment where each non-membrane region of a membrane-spanning protein is found.&lt;p>&lt;a href='/help/topo_dom' target='_top'>More...&lt;/a>&lt;/p></span>Topological domain<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[584-646]&amp;key=Topological domain">584 – 646</a></td><td class="featdescription"><span property="text">Cytoplasmic</span><span class="attribution ECO255"><span class="attributionHeader tooltipped" title="Manual assertion according to rules">Sequence analysis</span></span><span class="featactions"><a class="icon icon-static tooltipped subsequences" data-icon="b" id="P43121_584-646_" title="Add to basket">Add</a> <a class="icon icon-functional tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">BLAST</a></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="90.247%" width="9.752%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">63</td></tr></table><h4><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>The &lt;a href="http://www.geneontology.org/">Gene Ontology (GO)&lt;/a> project provides a set of hierarchical controlled vocabulary split into 3 categories:&lt;p>&lt;a href='/help/gene_ontology' target='_top'>More...&lt;/a>&lt;/p></span>GO - Cellular component<sup>i</sup></span></h4><ul class="noNumbering cellular_component"><li><a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009897" onclick="uniprot.analytics('UniProt-Entry-View', 'click', 'Display-GO-Term');">external side of plasma membrane</a><span class="attribution ECO314"><span class="attributionHeader"> Source: UniProtKB<span class="showHideEvidence caret_grey displayThisInline"></span></span><span class="gosource evidenceContainer" style="display:none"><span class="goa attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Inferred from Direct Assay&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>Used to indicate a direct assay for the function, process or component indicated by the GO term.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>More information in the &lt;a href="http://geneontology.org/page/guide-go-evidence-codes#ida">GO evidence code guide&lt;/a>&lt;/p>      </span>Inferred from direct assay<sup>i</sup></span></span><ul><li><a href="/citations/20354148" class="pubmed ">20354148</a></li></ul></span></span></li><li><a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005615" onclick="uniprot.analytics('UniProt-Entry-View', 'click', 'Display-GO-Term');">extracellular space</a><span class="attribution ECO501"><span class="attributionHeader tooltipped" title="Inferred from electronic annotation"> Source: Ensembl</span></span></li><li><a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005925" onclick="uniprot.analytics('UniProt-Entry-View', 'click', 'Display-GO-Term');">focal adhesion</a><span class="attribution ECO314"><span class="attributionHeader"> Source: UniProtKB<span class="showHideEvidence caret_grey displayThisInline"></span></span><span class="gosource evidenceContainer" style="display:none"><span class="goa attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Inferred from Direct Assay&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>Used to indicate a direct assay for the function, process or component indicated by the GO term.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>More information in the &lt;a href="http://geneontology.org/page/guide-go-evidence-codes#ida">GO evidence code guide&lt;/a>&lt;/p>      </span>Inferred from direct assay<sup>i</sup></span></span><ul><li><a href="/citations/21423176" class="pubmed ">21423176</a></li></ul></span></span></li><li><a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021" onclick="uniprot.analytics('UniProt-Entry-View', 'click', 'Display-GO-Term');">integral component of membrane</a><span class="attribution ECO501"><span class="attributionHeader tooltipped" title="Inferred from electronic annotation"> Source: UniProtKB-KW</span></span></li><li><a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005634" onclick="uniprot.analytics('UniProt-Entry-View', 'click', 'Display-GO-Term');">nucleus</a><span class="attribution ECO501"><span class="attributionHeader tooltipped" title="Inferred from electronic annotation"> Source: Ensembl</span></span></li><li><a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005886" onclick="uniprot.analytics('UniProt-Entry-View', 'click', 'Display-GO-Term');">plasma membrane</a><span class="attribution ECO303"><span class="attributionHeader"> Source: ProtInc<span class="showHideEvidence caret_grey displayThisInline"></span></span><span class="gosource evidenceContainer" style="display:none"><span class="goa attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Non-traceable Author Statement&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>Used for statements in the abstract, introduction or discussion of a paper that cannot be traced back to another publication.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>More information in the &lt;a href="http://geneontology.org/page/guide-go-evidence-codes#nas">GO evidence code guide&lt;/a>&lt;/p>      </span>Non-traceable author statement<sup>i</sup></span></span><ul><li><a href="/citations/8378324" class="pubmed ">8378324</a></li></ul></span></span></li></ul><div class="gocomplete"><a href="http://www.ebi.ac.uk/QuickGO/GProtein?ac=P43121" onclick="uniprot.analytics('UniProt-Entry-View', 'click', 'Complete-GO-Annotation');">Complete GO annotation...</a></div><h4><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>UniProtKB Keywords constitute a &lt;a href="http://www.uniprot.org/keywords">controlled vocabulary&lt;/a> with a hierarchical structure. Keywords summarise the content of a UniProtKB entry and facilitate the search for proteins of interest.&lt;p>&lt;a href='/help/keywords' target='_top'>More...&lt;/a>&lt;/p></span>Keywords - Cellular component<sup>i</sup></span></h4><span><a href="/keywords/KW-0472">Membrane</a></span></div><div class="section " id="pathology_and_biotech"><h2><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This section provides information on the disease(s) and phenotype(s) associated with a protein.&lt;p>&lt;a href='/help/pathology_and_biotech_section' target='_top'>More...&lt;/a>&lt;/p></span>Pathology &amp; Biotech<sup>i</sup></span></h2><h4 id="section_x-ref_organism" class="xref_category">Organism-specific databases</h4><table class="databaseTable ORGANISM"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>DisGeNET</p><a href="/database/DB-0218">More...</a></span>DisGeNET<sup>i</sup></span></td><td><a href="http://disgenet.org/search?q=4162" onclick="uniprot.analytics('DR-lines', 'click', 'DR-DisGeNET');">4162.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Open Targets</p><a href="/database/DB-0219">More...</a></span>OpenTargets<sup>i</sup></span></td><td><a href="https://www.targetvalidation.org/target/ENSG00000076706/associations" onclick="uniprot.analytics('DR-lines', 'click', 'DR-OpenTargets');">ENSG00000076706.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>The Pharmacogenetics and Pharmacogenomics Knowledge Base</p><a href="/database/DB-0074">More...</a></span>PharmGKB<sup>i</sup></span></td><td><a href="https://www.pharmgkb.org/gene/PA30678" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PharmGKB');">PA30678.</a> <br/></td></tr></table><h4 id="section_x-ref_polymorphism" class="xref_category">Polymorphism and mutation databases</h4><table class="databaseTable POLYMORPHISM"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>BioMuta curated single-nucleotide variation and disease association database</p><a href="/database/DB-0192">More...</a></span>BioMuta<sup>i</sup></span></td><td><a href="https://hive.biochemistry.gwu.edu/tools/biomuta/biomuta.php?gene=MCAM" onclick="uniprot.analytics('DR-lines', 'click', 'DR-BioMuta');">MCAM.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Domain mapping of disease mutations (DMDM)</p><a href="/database/DB-0166">More...</a></span>DMDM<sup>i</sup></span></td><td><a href="http://bioinf.umbc.edu/dmdm/gene_prot_page.php?search_type=protein&amp;id=85681878" onclick="uniprot.analytics('DR-lines', 'click', 'DR-DMDM');">85681878.</a> <br/></td></tr></table></div><div class="section " id="ptm_processing"><h2><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This section describes post-translational modifications (PTMs) and/or processing events.&lt;p>&lt;a href='/help/ptm_processing_section' target='_top'>More...&lt;/a>&lt;/p></span>PTM / Processing<sup>i</sup></span></h2><h4>Molecule processing</h4><table class="featureTable" id="peptides_section"><tr><th class="featkey_h">Feature key</th><th class="pos_h">Position(s)</th><th class="description_h">Description<span class="featactions_h">Actions</span></th><th class="graphview_h">Graphical view</th><th class="length_h">Length</th></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘PTM / Processing’ section denotes the presence of an N-terminal signal peptide.&lt;p>&lt;a href='/help/signal' target='_top'>More...&lt;/a>&lt;/p></span>Signal peptide<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[1-23]&amp;key=Signal peptide">1 – 23</a></td><td class="featdescription"><span class="attribution ECO269"><span class="attributionHeader ">1 Publication<span class="showHideEvidence caret_grey displayThisInline"></span></span><span style="display:none" class="evidenceContainer"><p class="attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Manually curated information for which there is published experimental evidence.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>&lt;a href="/manual/evidences#ECO:0000269">More…&lt;/a>&lt;/p>      </span>Manual assertion based on experiment in<sup>i</sup></span></p><ul><li><div class="P43121#ref7 referenceAttribution"><div class="reference_header">Ref.7</div><div class="reference_content"><div property="schema:citation" resource="http://purl.uniprot.org/citations/8162602" typeof="schema:ScholarlyArticle"><strong property="schema:name">"Isolation and functional characterization of the A32 melanoma-associated antigen."</strong><br/><a href="/uniprot/?query=author:%22Shih+I.-M.%22" rel="nofollow">Shih I.-M.</a>, <a href="/uniprot/?query=author:%22Eleder+D.E.%22" rel="nofollow">Eleder D.E.</a>, <a href="/uniprot/?query=author:%22Speicher+D.%22" rel="nofollow">Speicher D.</a>, <a href="/uniprot/?query=author:%22Johnson+J.P.%22" rel="nofollow">Johnson J.P.</a>, <a href="/uniprot/?query=author:%22Herlyn+M.%22" rel="nofollow">Herlyn M.</a><br/>Cancer Res. 54:2514-2520(1994)  [<a property="schema:sameAs" href="https://www.ncbi.nlm.nih.gov/pubmed/8162602">PubMed</a>] [<a property="schema:sameAs" href="http://europepmc.org/abstract/MED/8162602">Europe PMC</a>] [<a href="/citations/8162602">Abstract</a>]</div><div class="citedFor"><span class="details"><strong>Cited for:</strong></span> PROTEIN SEQUENCE OF <a href="/blast/?about=P43121[24-44]" class="tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">24-44</a>; <a href="/blast/?about=P43121[98-112]" class="tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">98-112</a>; <a href="/blast/?about=P43121[135-153]" class="tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">135-153</a>; <a href="/blast/?about=P43121[240-260]" class="tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">240-260</a>; <a href="/blast/?about=P43121[379-389]" class="tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">379-389</a> AND <a href="/blast/?about=P43121[460-478]" class="tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">460-478</a>.</div></div></div></li></ul></span></span><span class="featactions"><a class="icon icon-static tooltipped subsequences" data-icon="b" id="P43121_1-23_" title="Add to basket">Add</a> <a class="icon icon-functional tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">BLAST</a></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="0%" width="3.560%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">23</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork" resource="http://purl.uniprot.org/annotation/PRO_0000014891" id="PRO_0000014891"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘PTM / Processing’ section describes the extent of a polypeptide chain in the mature protein following processing.&lt;p>&lt;a href='/help/chain' target='_top'>More...&lt;/a>&lt;/p></span>Chain<sup>i</sup></span><span class="featureId">PRO_0000014891</span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[24-646]&amp;key=Chain&amp;id=PRO_0000014891">24 – 646</a></td><td class="featdescription"><span property="text">Cell surface glycoprotein MUC18</span><span class="featactions"><a class="icon icon-static tooltipped subsequences" data-icon="b" id="P43121_24-646_" title="Add to basket">Add</a> <a class="icon icon-functional tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">BLAST</a></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="3.560%" width="96.439%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">623</td></tr></table><h4>Amino acid modifications</h4><table class="featureTable" id="aaMod_section"><tr><th class="featkey_h">Feature key</th><th class="pos_h">Position(s)</th><th class="description_h">Description<span class="featactions_h">Actions</span></th><th class="graphview_h">Graphical view</th><th class="length_h">Length</th></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;span class="caps">PTM&lt;/span> / Processing”:/help/ptm_processing_section section describes the positions of cysteine residues participating in disulfide bonds.&lt;p>&lt;a href='/help/disulfid' target='_top'>More...&lt;/a>&lt;/p></span>Disulfide bond<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[48,116]&amp;key=Disulfide bond">48 ↔ 116</a></td><td class="featdescription"><span class="attribution ECO305"><span class="attributionHeader tooltipped" title="Manual assertion inferred by curator">Curated</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="7.275%" width="10.681%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric"></td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;a href="http://www.uniprot.org/help/ptm_processing_section">&lt;span class="caps">PTM&lt;/span> / Processing&lt;/a> section specifies the position and type of each covalently attached glycan group (mono-, di-, or polysaccharide).&lt;p>&lt;a href='/help/carbohyd' target='_top'>More...&lt;/a>&lt;/p></span>Glycosylation<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[56]&amp;key=Glycosylation">56</a></td><td class="featdescription"><span property="text">N-linked (GlcNAc...)</span><span class="attribution ECO255"><span class="attributionHeader tooltipped" title="Manual assertion according to rules">Sequence analysis</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="8.513%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;span class="caps">PTM&lt;/span> / Processing”:/help/ptm_processing_section section describes the positions of cysteine residues participating in disulfide bonds.&lt;p>&lt;a href='/help/disulfid' target='_top'>More...&lt;/a>&lt;/p></span>Disulfide bond<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[161,223]&amp;key=Disulfide bond">161 ↔ 223</a></td><td class="featdescription"><span class="attribution ECO305"><span class="attributionHeader tooltipped" title="Manual assertion inferred by curator">Curated</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="24.767%" width="9.752%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric"></td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;span class="caps">PTM&lt;/span> / Processing”:/help/ptm_processing_section section describes the positions of cysteine residues participating in disulfide bonds.&lt;p>&lt;a href='/help/disulfid' target='_top'>More...&lt;/a>&lt;/p></span>Disulfide bond<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[272,320]&amp;key=Disulfide bond">272 ↔ 320</a></td><td class="featdescription"><span class="attribution ECO305"><span class="attributionHeader tooltipped" title="Manual assertion inferred by curator">Curated</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="41.950%" width="7.585%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric"></td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;span class="caps">PTM&lt;/span> / Processing”:/help/ptm_processing_section section describes the positions of cysteine residues participating in disulfide bonds.&lt;p>&lt;a href='/help/disulfid' target='_top'>More...&lt;/a>&lt;/p></span>Disulfide bond<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[365,407]&amp;key=Disulfide bond">365 ↔ 407</a></td><td class="featdescription"><span class="attribution ECO305"><span class="attributionHeader tooltipped" title="Manual assertion inferred by curator">Curated</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="56.346%" width="6.656%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric"></td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;a href="http://www.uniprot.org/help/ptm_processing_section">&lt;span class="caps">PTM&lt;/span> / Processing&lt;/a> section specifies the position and type of each covalently attached glycan group (mono-, di-, or polysaccharide).&lt;p>&lt;a href='/help/carbohyd' target='_top'>More...&lt;/a>&lt;/p></span>Glycosylation<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[418]&amp;key=Glycosylation">418</a></td><td class="featdescription"><span property="text">N-linked (GlcNAc...)</span><span class="attribution ECO255"><span class="attributionHeader tooltipped" title="Manual assertion according to rules">Sequence analysis</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="64.551%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;a href="http://www.uniprot.org/help/ptm_processing_section">&lt;span class="caps">PTM&lt;/span> / Processing&lt;/a> section specifies the position and type of each covalently attached glycan group (mono-, di-, or polysaccharide).&lt;p>&lt;a href='/help/carbohyd' target='_top'>More...&lt;/a>&lt;/p></span>Glycosylation<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[449]&amp;key=Glycosylation">449</a></td><td class="featdescription"><span property="text">N-linked (GlcNAc...)</span><span class="attribution ECO255"><span class="attributionHeader tooltipped" title="Manual assertion according to rules">Sequence analysis</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="69.349%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;span class="caps">PTM&lt;/span> / Processing”:/help/ptm_processing_section section describes the positions of cysteine residues participating in disulfide bonds.&lt;p>&lt;a href='/help/disulfid' target='_top'>More...&lt;/a>&lt;/p></span>Disulfide bond<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[452,499]&amp;key=Disulfide bond">452 ↔ 499</a></td><td class="featdescription"><span class="attribution ECO305"><span class="attributionHeader tooltipped" title="Manual assertion inferred by curator">Curated</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="69.814%" width="7.430%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric"></td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;a href="http://www.uniprot.org/help/ptm_processing_section">&lt;span class="caps">PTM&lt;/span> / Processing&lt;/a> section specifies the position and type of each covalently attached glycan group (mono-, di-, or polysaccharide).&lt;p>&lt;a href='/help/carbohyd' target='_top'>More...&lt;/a>&lt;/p></span>Glycosylation<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[467]&amp;key=Glycosylation">467</a></td><td class="featdescription"><span property="text">N-linked (GlcNAc...)</span><span class="attribution ECO269"><span class="attributionHeader ">1 Publication<span class="showHideEvidence caret_grey displayThisInline"></span></span><span style="display:none" class="evidenceContainer"><p class="attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Manually curated information for which there is published experimental evidence.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>&lt;a href="/manual/evidences#ECO:0000269">More…&lt;/a>&lt;/p>      </span>Manual assertion based on experiment in<sup>i</sup></span></p><ul><li><div class="P43121#ref12 referenceAttribution"><div class="reference_header">Ref.12</div><div class="reference_content"><div property="schema:citation" resource="http://purl.uniprot.org/citations/19159218" typeof="schema:ScholarlyArticle"><strong property="schema:name">"Glycoproteomics analysis of human liver tissue by combination of multiple enzyme digestion and hydrazide chemistry."</strong><br/><a href="/uniprot/?query=author:%22Chen+R.%22" rel="nofollow">Chen R.</a>, <a href="/uniprot/?query=author:%22Jiang+X.%22" rel="nofollow">Jiang X.</a>, <a href="/uniprot/?query=author:%22Sun+D.%22" rel="nofollow">Sun D.</a>, <a href="/uniprot/?query=author:%22Han+G.%22" rel="nofollow">Han G.</a>, <a href="/uniprot/?query=author:%22Wang+F.%22" rel="nofollow">Wang F.</a>, <a href="/uniprot/?query=author:%22Ye+M.%22" rel="nofollow">Ye M.</a>, <a href="/uniprot/?query=author:%22Wang+L.%22" rel="nofollow">Wang L.</a>, <a href="/uniprot/?query=author:%22Zou+H.%22" rel="nofollow">Zou H.</a><br/><a href="http://dx.doi.org/10.1021/pr8008012">J. Proteome Res. 8:651-661(2009)</a>  [<a property="schema:sameAs" href="https://www.ncbi.nlm.nih.gov/pubmed/19159218">PubMed</a>] [<a property="schema:sameAs" href="http://europepmc.org/abstract/MED/19159218">Europe PMC</a>] [<a href="/citations/19159218">Abstract</a>]</div><div class="citedFor"><span class="details"><strong>Cited for:</strong></span> GLYCOSYLATION [LARGE SCALE ANALYSIS] AT ASN-467.</div></div></div></li></ul></span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="72.136%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;a href="http://www.uniprot.org/help/ptm_processing_section">&lt;span class="caps">PTM&lt;/span> / Processing&lt;/a> section specifies the position and type of each covalently attached glycan group (mono-, di-, or polysaccharide).&lt;p>&lt;a href='/help/carbohyd' target='_top'>More...&lt;/a>&lt;/p></span>Glycosylation<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[508]&amp;key=Glycosylation">508</a></td><td class="featdescription"><span property="text">N-linked (GlcNAc...)</span><span class="attribution ECO255"><span class="attributionHeader tooltipped" title="Manual assertion according to rules">Sequence analysis</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="78.482%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;a href="http://www.uniprot.org/help/ptm_processing_section">&lt;span class="caps">PTM&lt;/span> / Processing&lt;/a> section specifies the position and type of each covalently attached glycan group (mono-, di-, or polysaccharide).&lt;p>&lt;a href='/help/carbohyd' target='_top'>More...&lt;/a>&lt;/p></span>Glycosylation<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[518]&amp;key=Glycosylation">518</a></td><td class="featdescription"><span property="text">N-linked (GlcNAc...)</span><span class="attribution ECO255"><span class="attributionHeader tooltipped" title="Manual assertion according to rules">Sequence analysis</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="80.030%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;a href="http://www.uniprot.org/help/ptm_processing_section">&lt;span class="caps">PTM&lt;/span> / Processing&lt;/a> section specifies the position and type of each covalently attached glycan group (mono-, di-, or polysaccharide).&lt;p>&lt;a href='/help/carbohyd' target='_top'>More...&lt;/a>&lt;/p></span>Glycosylation<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[527]&amp;key=Glycosylation">527</a></td><td class="featdescription"><span property="text">N-linked (GlcNAc...)</span><span class="attribution ECO255"><span class="attributionHeader tooltipped" title="Manual assertion according to rules">Sequence analysis</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="81.424%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;a href="http://www.uniprot.org/help/ptm_processing_section">&lt;span class="caps">PTM&lt;/span> / Processing&lt;/a> section specifies the position and type of each covalently attached glycan group (mono-, di-, or polysaccharide).&lt;p>&lt;a href='/help/carbohyd' target='_top'>More...&lt;/a>&lt;/p></span>Glycosylation<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[544]&amp;key=Glycosylation">544</a></td><td class="featdescription"><span property="text">N-linked (GlcNAc...)</span><span class="attribution ECO255"><span class="attributionHeader tooltipped" title="Manual assertion according to rules">Sequence analysis</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="84.055%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘PTM / Processing’ section specifies the position and type of each modified residue excluding &lt;a href="http://www.uniprot.org/manual/lipid">lipids&lt;/a>, &lt;a href="http://www.uniprot.org/manual/carbohyd">glycans&lt;/a> and &lt;a href="http://www.uniprot.org/manual/crosslnk">protein cross-links&lt;/a>.&lt;p>&lt;a href='/help/mod_res' target='_top'>More...&lt;/a>&lt;/p></span>Modified residue<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[606]&amp;key=Modified residue">606</a></td><td class="featdescription"><span property="text">Phosphoserine</span><span class="attribution ECO250"><span class="attributionHeader ">By similarity<span class="showHideEvidence caret_grey displayThisInline"></span></span><span style="display:none" class="evidenceContainer"><p class="attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Manually curated information which has been propagated from a related experimentally characterized protein.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>&lt;a href="/manual/evidences#ECO:0000250">More…&lt;/a>&lt;/p>      </span>Manual assertion inferred from sequence similarity to<sup>i</sup></span></p><ul><li><a href="Q8R2Y2">UniProtKB:Q8R2Y2 (MUC18_MOUSE)</a></li></ul></span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="93.653%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘PTM / Processing’ section specifies the position and type of each modified residue excluding &lt;a href="http://www.uniprot.org/manual/lipid">lipids&lt;/a>, &lt;a href="http://www.uniprot.org/manual/carbohyd">glycans&lt;/a> and &lt;a href="http://www.uniprot.org/manual/crosslnk">protein cross-links&lt;/a>.&lt;p>&lt;a href='/help/mod_res' target='_top'>More...&lt;/a>&lt;/p></span>Modified residue<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[614]&amp;key=Modified residue">614</a></td><td class="featdescription"><span property="text">Phosphoserine</span><span class="attribution ECO244"><span class="attributionHeader ">Combined sources<span class="showHideEvidence caret_grey displayThisInline"></span></span><span style="display:none" class="evidenceContainer"><p class="attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Manually validated information inferred from a combination of experimental and computational evidence.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>&lt;a href="/manual/evidences#ECO:0000244">More…&lt;/a>&lt;/p>      </span>Manual assertion inferred from combination of experimental and computational evidence<sup>i</sup></span></p><ul><li><div class="P43121#ref11 referenceAttribution"><div class="reference_header">Ref.11</div><div class="reference_content"><div property="schema:citation" resource="http://purl.uniprot.org/citations/18669648" typeof="schema:ScholarlyArticle"><strong property="schema:name">"A quantitative atlas of mitotic phosphorylation."</strong><br/><a href="/uniprot/?query=author:%22Dephoure+N.%22" rel="nofollow">Dephoure N.</a>, <a href="/uniprot/?query=author:%22Zhou+C.%22" rel="nofollow">Zhou C.</a>, <a href="/uniprot/?query=author:%22Villen+J.%22" rel="nofollow">Villen J.</a>, <a href="/uniprot/?query=author:%22Beausoleil+S.A.%22" rel="nofollow">Beausoleil S.A.</a>, <a href="/uniprot/?query=author:%22Bakalarski+C.E.%22" rel="nofollow">Bakalarski C.E.</a>, <a href="/uniprot/?query=author:%22Elledge+S.J.%22" rel="nofollow">Elledge S.J.</a>, <a href="/uniprot/?query=author:%22Gygi+S.P.%22" rel="nofollow">Gygi S.P.</a><br/><a href="http://dx.doi.org/10.1073/pnas.0805139105">Proc. Natl. Acad. Sci. U.S.A. 105:10762-10767(2008)</a>  [<a property="schema:sameAs" href="https://www.ncbi.nlm.nih.gov/pubmed/18669648">PubMed</a>] [<a property="schema:sameAs" href="http://europepmc.org/abstract/MED/18669648">Europe PMC</a>] [<a href="/citations/18669648">Abstract</a>]</div><div class="citedFor"><span class="details"><strong>Cited for:</strong></span> PHOSPHORYLATION [LARGE SCALE ANALYSIS] AT SER-614, IDENTIFICATION BY MASS SPECTROMETRY [LARGE SCALE ANALYSIS].</div></div></div></li><li><div class="P43121#ref13 referenceAttribution"><div class="reference_header">Ref.13</div><div class="reference_content"><div property="schema:citation" resource="http://purl.uniprot.org/citations/20068231" typeof="schema:ScholarlyArticle"><strong property="schema:name">"Quantitative phosphoproteomics reveals widespread full phosphorylation site occupancy during mitosis."</strong><br/><a href="/uniprot/?query=author:%22Olsen+J.V.%22" rel="nofollow">Olsen J.V.</a>, <a href="/uniprot/?query=author:%22Vermeulen+M.%22" rel="nofollow">Vermeulen M.</a>, <a href="/uniprot/?query=author:%22Santamaria+A.%22" rel="nofollow">Santamaria A.</a>, <a href="/uniprot/?query=author:%22Kumar+C.%22" rel="nofollow">Kumar C.</a>, <a href="/uniprot/?query=author:%22Miller+M.L.%22" rel="nofollow">Miller M.L.</a>, <a href="/uniprot/?query=author:%22Jensen+L.J.%22" rel="nofollow">Jensen L.J.</a>, <a href="/uniprot/?query=author:%22Gnad+F.%22" rel="nofollow">Gnad F.</a>, <a href="/uniprot/?query=author:%22Cox+J.%22" rel="nofollow">Cox J.</a>, <a href="/uniprot/?query=author:%22Jensen+T.S.%22" rel="nofollow">Jensen T.S.</a>, <a href="/uniprot/?query=author:%22Nigg+E.A.%22" rel="nofollow">Nigg E.A.</a>, <a href="/uniprot/?query=author:%22Brunak+S.%22" rel="nofollow">Brunak S.</a>, <a href="/uniprot/?query=author:%22Mann+M.%22" rel="nofollow">Mann M.</a><br/><a href="http://dx.doi.org/10.1126/scisignal.2000475">Sci. Signal. 3:RA3-RA3(2010)</a>  [<a property="schema:sameAs" href="https://www.ncbi.nlm.nih.gov/pubmed/20068231">PubMed</a>] [<a property="schema:sameAs" href="http://europepmc.org/abstract/MED/20068231">Europe PMC</a>] [<a href="/citations/20068231">Abstract</a>]</div><div class="citedFor"><span class="details"><strong>Cited for:</strong></span> PHOSPHORYLATION [LARGE SCALE ANALYSIS] AT SER-614 AND SER-628, IDENTIFICATION BY MASS SPECTROMETRY [LARGE SCALE ANALYSIS].</div></div></div></li></ul></span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="94.891%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘PTM / Processing’ section specifies the position and type of each modified residue excluding &lt;a href="http://www.uniprot.org/manual/lipid">lipids&lt;/a>, &lt;a href="http://www.uniprot.org/manual/carbohyd">glycans&lt;/a> and &lt;a href="http://www.uniprot.org/manual/crosslnk">protein cross-links&lt;/a>.&lt;p>&lt;a href='/help/mod_res' target='_top'>More...&lt;/a>&lt;/p></span>Modified residue<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[628]&amp;key=Modified residue">628</a></td><td class="featdescription"><span property="text">Phosphoserine</span><span class="attribution ECO244"><span class="attributionHeader ">Combined sources<span class="showHideEvidence caret_grey displayThisInline"></span></span><span style="display:none" class="evidenceContainer"><p class="attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Manually validated information inferred from a combination of experimental and computational evidence.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>&lt;a href="/manual/evidences#ECO:0000244">More…&lt;/a>&lt;/p>      </span>Manual assertion inferred from combination of experimental and computational evidence<sup>i</sup></span></p><ul><li><div class="P43121#ref13 referenceAttribution"><div class="reference_header">Ref.13</div><div class="reference_content"><div property="schema:citation" resource="http://purl.uniprot.org/citations/20068231" typeof="schema:ScholarlyArticle"><strong property="schema:name">"Quantitative phosphoproteomics reveals widespread full phosphorylation site occupancy during mitosis."</strong><br/><a href="/uniprot/?query=author:%22Olsen+J.V.%22" rel="nofollow">Olsen J.V.</a>, <a href="/uniprot/?query=author:%22Vermeulen+M.%22" rel="nofollow">Vermeulen M.</a>, <a href="/uniprot/?query=author:%22Santamaria+A.%22" rel="nofollow">Santamaria A.</a>, <a href="/uniprot/?query=author:%22Kumar+C.%22" rel="nofollow">Kumar C.</a>, <a href="/uniprot/?query=author:%22Miller+M.L.%22" rel="nofollow">Miller M.L.</a>, <a href="/uniprot/?query=author:%22Jensen+L.J.%22" rel="nofollow">Jensen L.J.</a>, <a href="/uniprot/?query=author:%22Gnad+F.%22" rel="nofollow">Gnad F.</a>, <a href="/uniprot/?query=author:%22Cox+J.%22" rel="nofollow">Cox J.</a>, <a href="/uniprot/?query=author:%22Jensen+T.S.%22" rel="nofollow">Jensen T.S.</a>, <a href="/uniprot/?query=author:%22Nigg+E.A.%22" rel="nofollow">Nigg E.A.</a>, <a href="/uniprot/?query=author:%22Brunak+S.%22" rel="nofollow">Brunak S.</a>, <a href="/uniprot/?query=author:%22Mann+M.%22" rel="nofollow">Mann M.</a><br/><a href="http://dx.doi.org/10.1126/scisignal.2000475">Sci. Signal. 3:RA3-RA3(2010)</a>  [<a property="schema:sameAs" href="https://www.ncbi.nlm.nih.gov/pubmed/20068231">PubMed</a>] [<a property="schema:sameAs" href="http://europepmc.org/abstract/MED/20068231">Europe PMC</a>] [<a href="/citations/20068231">Abstract</a>]</div><div class="citedFor"><span class="details"><strong>Cited for:</strong></span> PHOSPHORYLATION [LARGE SCALE ANALYSIS] AT SER-614 AND SER-628, IDENTIFICATION BY MASS SPECTROMETRY [LARGE SCALE ANALYSIS].</div></div></div></li></ul></span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="97.058%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr></table><h4><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>UniProtKB Keywords constitute a &lt;a href="http://www.uniprot.org/keywords">controlled vocabulary&lt;/a> with a hierarchical structure. Keywords summarise the content of a UniProtKB entry and facilitate the search for proteins of interest.&lt;p>&lt;a href='/help/keywords' target='_top'>More...&lt;/a>&lt;/p></span>Keywords - PTM<sup>i</sup></span></h4><span><a href="/keywords/KW-1015">Disulfide bond</a>, <a href="/keywords/KW-0325">Glycoprotein</a>, <a href="/keywords/KW-0597">Phosphoprotein</a></span><h4 id="section_x-ref_proteomic" class="xref_category">Proteomic databases</h4><table class="databaseTable PROTEOMIC"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Encyclopedia of Proteome Dynamics</p><a href="/database/DB-0205">More...</a></span>EPD<sup>i</sup></span></td><td><a href="https://www.peptracker.com/epd/protein/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EPD');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>MaxQB - The MaxQuant DataBase</p><a href="/database/DB-0186">More...</a></span>MaxQB<sup>i</sup></span></td><td><a href="http://maxqb.biochem.mpg.de/mxdb/protein/show/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-MaxQB');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>PaxDb, a database of protein abundance averages across all three domains of life</p><a href="/database/DB-0173">More...</a></span>PaxDb<sup>i</sup></span></td><td><a href="http://pax-db.org/#!protein/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PaxDb');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>PeptideAtlas</p><a href="/database/DB-0071">More...</a></span>PeptideAtlas<sup>i</sup></span></td><td><a href="https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/Search?action=GO&amp;search_key=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PeptideAtlas');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>PRoteomics IDEntifications database</p><a href="/database/DB-0130">More...</a></span>PRIDE<sup>i</sup></span></td><td><a href="http://www.ebi.ac.uk/pride/searchSummary.do?queryTypeSelected=identification%20accession%20number&amp;identificationAccessionNumber=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PRIDE');">P43121.</a> <br/></td></tr></table><h4 id="section_x-ref_ptm" class="xref_category">PTM databases</h4><table class="databaseTable PTM"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>iPTMnet integrated resource for PTMs in systems biology context</p><a href="/database/DB-0200">More...</a></span>iPTMnet<sup>i</sup></span></td><td><a href="http://research.bioinformatics.udel.edu/iptmnet/entry/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-iPTMnet');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Comprehensive resource for the study of protein post-translational modifications (PTMs) in human, mouse and rat.</p><a href="/database/DB-0123">More...</a></span>PhosphoSitePlus<sup>i</sup></span></td><td><a href="http://www.phosphosite.org/uniprotAccAction?id=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PhosphoSitePlus');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>SwissPalm database of S-palmitoylation events</p><a href="/database/DB-0201">More...</a></span>SwissPalm<sup>i</sup></span></td><td><a href="http://swisspalm.epfl.ch/proteins?up_ac=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-SwissPalm');">P43121.</a> <br/></td></tr></table></div><div class="section " id="expression"><h2><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This section provides information on the expression of a gene at the mRNA or protein level in cells or in tissues of multicellular organisms.&lt;p>&lt;a href='/help/expression_section' target='_top'>More...&lt;/a>&lt;/p></span>Expression<sup>i</sup></span></h2><h4><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Expression’ section provides information on the expression of a gene at the mRNA or protein level in cells or in tissues of multicellular organisms. By default, the information is derived from experiments at the mRNA level, unless specified ‘at protein level’.&lt;br>&lt;/br>Examples: &lt;a href="http://www.uniprot.org/uniprot/P92958#expression">&lt;span class="caps">P92958&lt;/span>&lt;/a>, &lt;a href="http://www.uniprot.org/uniprot/Q8TDN4#expression">&lt;span class="caps">Q8TDN4&lt;/span>&lt;/a>, &lt;a href="http://www.uniprot.org/uniprot/O14734#expression">&lt;span class="caps">O14734&lt;/span>&lt;/a>&lt;p>&lt;a href='/help/tissue_specificity' target='_top'>More...&lt;/a>&lt;/p></span>Tissue specificity<sup>i</sup></span></h4><div class="annotation" property="schema:hasPart" typeof="schema:CreativeWork"><span property="schema:text">Detected in endothelial cells in vascular tissue throughout the body. May appear at the surface of neural crest cells during their embryonic migration. Appears to be limited to vascular smooth muscle in normal adult tissues. Associated with tumor progression and the development of metastasis in human malignant melanoma. Expressed most strongly on metastatic lesions and advanced primary tumors and is only rarely detected in benign melanocytic nevi and thin primary melanomas with a low probability of metastasis.</span></div><h4 id="section_x-ref_expression" class="xref_category">Gene expression databases</h4><table class="databaseTable EXPRESSION"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Bgee dataBase for Gene Expression Evolution</p><a href="/database/DB-0133">More...</a></span>Bgee<sup>i</sup></span></td><td><a href="http://bgee.org/?page=gene&amp;gene_id=ENSG00000076706" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Bgee');">ENSG00000076706.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>CleanEx database of gene expression profiles</p><a href="/database/DB-0008">More...</a></span>CleanEx<sup>i</sup></span></td><td><a href="http://cleanex.vital-it.ch/cgi-bin/get_doc?db=cleanex&amp;format=nice&amp;entry=HS_MCAM" onclick="uniprot.analytics('DR-lines', 'click', 'DR-CleanEx');">HS_MCAM.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>ExpressionAtlas, Differential and Baseline Expression</p><a href="/database/DB-0004">More...</a></span>ExpressionAtlas<sup>i</sup></span></td><td><a href="http://www.ebi.ac.uk/gxa/query?geneQuery=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-ExpressionAtlas');">P43121.</a> baseline and differential. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Genevisible search portal to normalized and curated expression data from Genevestigator</p><a href="/database/DB-0194">More...</a></span>Genevisible<sup>i</sup></span></td><td><a href="https://genevisible.com/tissues/HS/UniProt/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Genevisible');">P43121.</a> HS. <br/></td></tr></table><h4 id="section_x-ref_organism" class="xref_category">Organism-specific databases</h4><table class="databaseTable ORGANISM"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Human Protein Atlas</p><a href="/database/DB-0046">More...</a></span>HPA<sup>i</sup></span></td><td><a href="http://www.proteinatlas.org/tissue_profile.php?antibody_id=CAB002147" onclick="uniprot.analytics('DR-lines', 'click', 'DR-HPA');">CAB002147.</a> <br/><a href="http://www.proteinatlas.org/tissue_profile.php?antibody_id=HPA008848" onclick="uniprot.analytics('DR-lines', 'click', 'DR-HPA');">HPA008848.</a> <br/></td></tr></table></div><div class="section " id="interaction"><h2><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This section provides information on the quaternary structure of a protein and on interaction(s) with other proteins or protein complexes.&lt;p>&lt;a href='/help/interaction_section' target='_top'>More...&lt;/a>&lt;/p></span>Interaction<sup>i</sup></span></h2><h4 id="section_x-ref_interaction" class="xref_category">Protein-protein interaction databases</h4><table class="databaseTable INTERACTION"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>The Biological General Repository for Interaction Datasets (BioGrid)</p><a href="/database/DB-0184">More...</a></span>BioGrid<sup>i</sup></span></td><td><a href="https://thebiogrid.org/110332" onclick="uniprot.analytics('DR-lines', 'click', 'DR-BioGrid');">110332.</a> 13 interactors.<br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Database of interacting proteins</p><a href="/database/DB-0016">More...</a></span>DIP<sup>i</sup></span></td><td><a href="http://dip.doe-mbi.ucla.edu/dip/Browse.cgi?ID=DIP-52791N" onclick="uniprot.analytics('DR-lines', 'click', 'DR-DIP');">DIP-52791N.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Protein interaction database and analysis system</p><a href="/database/DB-0051">More...</a></span>IntAct<sup>i</sup></span></td><td><a href="https://www.ebi.ac.uk/intact/interactors/id:P43121*" onclick="uniprot.analytics('DR-lines', 'click', 'DR-IntAct');">P43121.</a> 5 interactors.<br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Molecular INTeraction database</p><a href="/database/DB-0158">More...</a></span>MINT<sup>i</sup></span></td><td><a href="http://mint.bio.uniroma2.it/mint/search/search.do?queryType=protein&amp;interactorAc=MINT-1184800" onclick="uniprot.analytics('DR-lines', 'click', 'DR-MINT');">MINT-1184800.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>STRING: functional protein association networks</p><a href="/database/DB-0141">More...</a></span>STRING<sup>i</sup></span></td><td><a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifier=9606.ENSP00000264036" onclick="uniprot.analytics('DR-lines', 'click', 'DR-STRING');">9606.ENSP00000264036.</a> <br/></td></tr></table></div><div class="section " id="structure"><h2><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This section provides information on the tertiary and secondary structure of a protein.&lt;p>&lt;a href='/help/structure_section' target='_top'>More...&lt;/a>&lt;/p></span>Structure<sup>i</sup></span></h2><h4 id="section_x-ref_structure" class="xref_category">3D structure databases</h4><table class="databaseTable STRUCTURE"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Protein Model Portal of the PSI-Nature Structural Biology Knowledgebase</p><a href="/database/DB-0159">More...</a></span>ProteinModelPortal<sup>i</sup></span></td><td><a href="http://www.proteinmodelportal.org/query/uniprot/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-ProteinModelPortal');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Database of comparative protein structure models</p><a href="/database/DB-0063">More...</a></span>ModBase<sup>i</sup></span></td><td><a href="http://salilab.org/modbase-cgi/model_search.cgi?searchkw=name&amp;kword=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-ModBase');">Search...</a><br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>MobiDB: a database of protein disorder and mobility annotations</p><a href="/database/DB-0183">More...</a></span>MobiDB<sup>i</sup></span></td><td><a href="http://mobidb.bio.unipd.it/entries/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-MobiDB');">Search...</a><br/></td></tr></table></div><div class="section " id="family_and_domains"><h2><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This section provides information on sequence similarities with other proteins and the domain(s) present in a protein.&lt;p>&lt;a href='/help/family_and_domains_section' target='_top'>More...&lt;/a>&lt;/p></span>Family &amp; Domains<sup>i</sup></span></h2><h4>Domains and Repeats</h4><table class="featureTable" id="domainsAnno_section"><tr><th class="featkey_h">Feature key</th><th class="pos_h">Position(s)</th><th class="description_h">Description<span class="featactions_h">Actions</span></th><th class="graphview_h">Graphical view</th><th class="length_h">Length</th></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Family and Domains’ section describes the position and type of a domain, which is defined as a specific combination of secondary structures organized into a characteristic three-dimensional structure or fold.&lt;p>&lt;a href='/help/domain' target='_top'>More...&lt;/a>&lt;/p></span>Domain<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[24-129]&amp;key=Domain">24 – 129</a></td><td class="featdescription"><span property="text">Ig-like V-type 1</span><span class="featactions"><a class="icon icon-static tooltipped subsequences" data-icon="b" id="P43121_24-129_" title="Add to basket">Add</a> <a class="icon icon-functional tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">BLAST</a></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="3.560%" width="16.408%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">106</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Family and Domains’ section describes the position and type of a domain, which is defined as a specific combination of secondary structures organized into a characteristic three-dimensional structure or fold.&lt;p>&lt;a href='/help/domain' target='_top'>More...&lt;/a>&lt;/p></span>Domain<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[139-242]&amp;key=Domain">139 – 242</a></td><td class="featdescription"><span property="text">Ig-like V-type 2</span><span class="featactions"><a class="icon icon-static tooltipped subsequences" data-icon="b" id="P43121_139-242_" title="Add to basket">Add</a> <a class="icon icon-functional tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">BLAST</a></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="21.362%" width="16.099%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">104</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Family and Domains’ section describes the position and type of a domain, which is defined as a specific combination of secondary structures organized into a characteristic three-dimensional structure or fold.&lt;p>&lt;a href='/help/domain' target='_top'>More...&lt;/a>&lt;/p></span>Domain<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[244-330]&amp;key=Domain">244 – 330</a></td><td class="featdescription"><span property="text">Ig-like C2-type 1</span><span class="featactions"><a class="icon icon-static tooltipped subsequences" data-icon="b" id="P43121_244-330_" title="Add to basket">Add</a> <a class="icon icon-functional tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">BLAST</a></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="37.616%" width="13.467%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">87</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Family and Domains’ section describes the position and type of a domain, which is defined as a specific combination of secondary structures organized into a characteristic three-dimensional structure or fold.&lt;p>&lt;a href='/help/domain' target='_top'>More...&lt;/a>&lt;/p></span>Domain<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[335-424]&amp;key=Domain">335 – 424</a></td><td class="featdescription"><span property="text">Ig-like C2-type 2</span><span class="featactions"><a class="icon icon-static tooltipped subsequences" data-icon="b" id="P43121_335-424_" title="Add to basket">Add</a> <a class="icon icon-functional tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">BLAST</a></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="51.702%" width="13.931%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">90</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Family and Domains’ section describes the position and type of a domain, which is defined as a specific combination of secondary structures organized into a characteristic three-dimensional structure or fold.&lt;p>&lt;a href='/help/domain' target='_top'>More...&lt;/a>&lt;/p></span>Domain<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[430-510]&amp;key=Domain">430 – 510</a></td><td class="featdescription"><span property="text">Ig-like C2-type 3</span><span class="featactions"><a class="icon icon-static tooltipped subsequences" data-icon="b" id="P43121_430-510_" title="Add to basket">Add</a> <a class="icon icon-functional tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">BLAST</a></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="66.408%" width="12.538%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">81</td></tr></table><h4><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>UniProtKB Keywords constitute a &lt;a href="http://www.uniprot.org/keywords">controlled vocabulary&lt;/a> with a hierarchical structure. Keywords summarise the content of a UniProtKB entry and facilitate the search for proteins of interest.&lt;p>&lt;a href='/help/keywords' target='_top'>More...&lt;/a>&lt;/p></span>Keywords - Domain<sup>i</sup></span></h4><span><a href="/keywords/KW-0393">Immunoglobulin domain</a>, <a href="/keywords/KW-0677">Repeat</a>, <a href="/keywords/KW-0732">Signal</a>, <a href="/keywords/KW-0812">Transmembrane</a>, <a href="/keywords/KW-1133">Transmembrane helix</a></span><h4 id="section_x-ref_phylogenomic" class="xref_category">Phylogenomic databases</h4><table class="databaseTable PHYLOGENOMIC"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>evolutionary genealogy of genes: Non-supervised Orthologous Groups</p><a href="/database/DB-0152">More...</a></span>eggNOG<sup>i</sup></span></td><td><a href="http://eggnogdb.embl.de/#/app/results?seqid=P43121&amp;target_nogs=ENOG410IEC2" onclick="uniprot.analytics('DR-lines', 'click', 'DR-eggNOG');">ENOG410IEC2.</a> Eukaryota. <br/><a href="http://eggnogdb.embl.de/#/app/results?seqid=P43121&amp;target_nogs=ENOG4110RPG" onclick="uniprot.analytics('DR-lines', 'click', 'DR-eggNOG');">ENOG4110RPG.</a> LUCA. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Ensembl GeneTree</p><a href="/database/DB-0162">More...</a></span>GeneTree<sup>i</sup></span></td><td><a href="http://www.ensemblgenomes.org/id-genetree/ENSGT00530000063457" onclick="uniprot.analytics('DR-lines', 'click', 'DR-GeneTree');">ENSGT00530000063457.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>The HOGENOM Database of Homologous Genes from Fully Sequenced Organisms</p><a href="/database/DB-0044">More...</a></span>HOGENOM<sup>i</sup></span></td><td><a href="http://pbil.univ-lyon1.fr/cgi-bin/acnuc-ac2tree?query=P43121&amp;db=HOGENOM" onclick="uniprot.analytics('DR-lines', 'click', 'DR-HOGENOM');">HOG000015427.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>The HOVERGEN Database of Homologous Vertebrate Genes</p><a href="/database/DB-0045">More...</a></span>HOVERGEN<sup>i</sup></span></td><td><a href="http://pbil.univ-lyon1.fr/cgi-bin/acnuc-ac2tree?query=P43121&amp;db=HOVERGEN" onclick="uniprot.analytics('DR-lines', 'click', 'DR-HOVERGEN');">HBG002808.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>InParanoid: Eukaryotic Ortholog Groups</p><a href="/database/DB-0146">More...</a></span>InParanoid<sup>i</sup></span></td><td><a href="http://inparanoid.sbc.su.se/cgi-bin/gene_search.cgi?id=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InParanoid');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>KEGG Orthology (KO)</p><a href="/database/DB-0164">More...</a></span>KO<sup>i</sup></span></td><td><a href="http://www.genome.jp/dbget-bin/www_bget?ko:K06534" onclick="uniprot.analytics('DR-lines', 'click', 'DR-KO');">K06534.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Identification of Orthologs from Complete Genome Data</p><a href="/database/DB-0137">More...</a></span>OMA<sup>i</sup></span></td><td><a href="http://omabrowser.org/cgi-bin/gateway.pl?f=DisplayGroup&amp;p1=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-OMA');">PSRKSEF.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Database of Orthologous Groups</p><a href="/database/DB-0143">More...</a></span>OrthoDB<sup>i</sup></span></td><td><a href="http://www.orthodb.org/?query=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-OrthoDB');">EOG091G072E.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Database for complete collections of gene phylogenies</p><a href="/database/DB-0144">More...</a></span>PhylomeDB<sup>i</sup></span></td><td><a href="http://phylomedb.org/?q=search_tree&amp;seqid=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PhylomeDB');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>TreeFam database of animal gene trees</p><a href="/database/DB-0185">More...</a></span>TreeFam<sup>i</sup></span></td><td><a href="http://www.treefam.org/family/TF330534" onclick="uniprot.analytics('DR-lines', 'click', 'DR-TreeFam');">TF330534.</a> <br/></td></tr></table><h4 id="section_x-ref_domain" class="xref_category">Family and domain databases</h4><table class="databaseTable DOMAIN"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Gene3D Structural and Functional Annotation of Protein Families</p><a href="/database/DB-0029">More...</a></span>Gene3D<sup>i</sup></span></td><td><a href="http://www.cathdb.info/superfamily/2.60.40.10" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Gene3D');">2.60.40.10.</a> 5 hits. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Integrated resource of protein families, domains and functional sites</p><a href="/database/DB-0052">More...</a></span>InterPro<sup>i</sup></span></td><td><a href="http://www.ebi.ac.uk/interpro/protein/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">View protein in InterPro</a><br/><a href="http://www.ebi.ac.uk/interpro/entry/IPR013162" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">IPR013162.</a> CD80_C2-set. <br/><a href="http://www.ebi.ac.uk/interpro/entry/IPR007110" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">IPR007110.</a> Ig-like_dom. <br/><a href="http://www.ebi.ac.uk/interpro/entry/IPR013783" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">IPR013783.</a> Ig-like_fold. <br/><a href="http://www.ebi.ac.uk/interpro/entry/IPR003599" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">IPR003599.</a> Ig_sub. <br/><a href="http://www.ebi.ac.uk/interpro/entry/IPR003598" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">IPR003598.</a> Ig_sub2. <br/><a href="http://www.ebi.ac.uk/interpro/entry/IPR013106" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">IPR013106.</a> Ig_V-set. <br/><a href="http://www.ebi.ac.uk/interpro/entry/IPR013151" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">IPR013151.</a> Immunoglobulin. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Pfam protein domain database</p><a href="/database/DB-0073">More...</a></span>Pfam<sup>i</sup></span></td><td><a href="http://pfam.xfam.org/protein/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Pfam-Graphical-View');">View protein in Pfam</a><br/><a href="http://pfam.xfam.org/family/PF08205" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Pfam');">PF08205.</a> C2-set_2. 1 hit. <br/><a href="http://pfam.xfam.org/family/PF00047" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Pfam');">PF00047.</a> ig. 1 hit. <br/><a href="http://pfam.xfam.org/family/PF07686" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Pfam');">PF07686.</a> V-set. 1 hit. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Simple Modular Architecture Research Tool; a protein domain database</p><a href="/database/DB-0097">More...</a></span>SMART<sup>i</sup></span></td><td><a href="http://smart.embl.de/smart/show_motifs.pl?ID=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-SMART-Graphical-View');">View protein in SMART</a><br/><a href="http://smart.embl.de/smart/do_annotation.pl?DOMAIN=SM00409" onclick="uniprot.analytics('DR-lines', 'click', 'DR-SMART');">SM00409.</a> IG. 5 hits. <br/><a href="http://smart.embl.de/smart/do_annotation.pl?DOMAIN=SM00408" onclick="uniprot.analytics('DR-lines', 'click', 'DR-SMART');">SM00408.</a> IGc2. 3 hits. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Superfamily database of structural and functional annotation</p><a href="/database/DB-0155">More...</a></span>SUPFAM<sup>i</sup></span></td><td><a href="http://supfam.org/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF48726" onclick="uniprot.analytics('DR-lines', 'click', 'DR-SUPFAM');">SSF48726.</a> SSF48726. 5 hits. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>PROSITE; a protein domain and family database</p><a href="/database/DB-0084">More...</a></span>PROSITE<sup>i</sup></span></td><td><a href="http://prosite.expasy.org/cgi-bin/prosite/PSScan.cgi?seq=P43121&amp;output=nice" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PROSITE-Graphical-View');">View protein in PROSITE</a><br/><a href="http://prosite.expasy.org/doc/PS50835" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PROSITE');">PS50835.</a> IG_LIKE. 5 hits. <br/></td></tr></table></div><div class="section " id="sequences"><h2><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This section displays by default the canonical protein sequence and upon request all isoforms described in the entry. It also includes information pertinent to the sequence(s), including length and molecular weight.&lt;p>&lt;a href='/help/sequences_section' target='_top'>More...&lt;/a>&lt;/p></span>Sequence<span>s (2)<sup>i</sup></span></span></h2><p class="details"><span><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Sequence’ section indicates if the &lt;a href="http://www.uniprot.org/help/canonical_and_isoforms">canonical sequence&lt;/a> displayed by default in the entry is complete or not.&lt;p>&lt;a href='/help/sequence_status' target='_top'>More...&lt;/a>&lt;/p></span>Sequence status<sup>i</sup></span></span><span class="tooltipped" title="The displayed sequence is complete.">: Complete.</span></p><p class="details"><span><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;a href="http://www.uniprot.org/help/sequences_section">Sequence&lt;/a> section indicates if the &lt;a href="http://www.uniprot.org/help/canonical_and_isoforms">canonical sequence&lt;/a> displayed by default in the entry is in its mature form or if it represents the precursor.&lt;p>&lt;a href='/help/sequence_processing' target='_top'>More...&lt;/a>&lt;/p></span>Sequence processing<sup>i</sup></span>: The displayed sequence is further processed into a mature form.</span></p><p>This entry describes <strong>2</strong><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Sequence’ section lists the alternative protein sequences (isoforms) that can be generated from the same gene by a single or by the combination of up to four biological events (alternative promoter usage, alternative splicing, alternative initiation and ribosomal frameshifting). Additionally, this section gives relevant information on each alternative protein isoform.&lt;p>&lt;a href='/help/alternative_products' target='_top'>More...&lt;/a>&lt;/p></span> isoforms<sup>i</sup></span> produced by <span class="alternative_products"><strong>alternative splicing</strong></span>. <a href="#" onclick="$('#alignIsoforms').click(); return false" title="Align this entry with its isoforms" class="icon icon-functional button tooltipped" data-icon="i">Align</a><script type="text/javascript">var isoforms = ['P43121','P43121-2'];</script><a id="add-all-isoforms" title="Add all isoforms to basket" class="icon icon-static button tooltipped add-isoforms-basket" data-icon="b" onclick="uniprot.basket.addItemsTobasket(isoforms,uniprot.namespace); return false;">Add to basket</a><a id="added-all-isoforms" title="All isoforms in basket" class="icon icon-static button disabled tooltipped added-isoforms-basket" style="display:none" data-icon="b">Added to basket</a></p><div id="sequences-section"><div class="sequence-isoform"><input type="checkbox" class="cart-item" onclick="addOrAppendCart('P43121', 'section_seq')" id="checkbox_P43121"/><div class="sequence-isoform-leftcol"><span class="sequence-isoform-header" id="P43121"><a id="P43121-1" href="../blast/?about=P43121-1">Isoform 1</a></span> (identifier: <strong>P43121-1</strong>)  [<a href="/uniparc/?query=uniprot:P43121&amp;direct=yes">UniParc</a>]<a href="/uniprot/P43121.fasta" class="tooltipped icon icon-functional button displayThis" data-icon="=" title="Sequence data in FASTA format" rel="nofollow">FASTA</a><a id="add-isoform-P43121-basket" class="icon icon-static button tooltipped add-basket-button P43121" title="Add this isoform to basket" data-icon="b">Add to basket</a><a id="added-isoform-P43121-basket" class="icon icon-static button disabled added-basket-button P43121" title="This isoform is in your basket" style="display:none" data-icon="b">Added to basket</a><div class="details"></div><p class="details"><i>This isoform has been chosen as the 'canonical' sequence. All positional information in this entry refers to it. This is also the sequence that appears in the downloadable versions of the entry.</i></p><div class="sequence" style="display:block" id="P43121-details"><p style="clear: both" class="details"><a href="#" onclick="$('#P43121-details').hide(); $('#P43121-show-link').show(); return false;">« Hide</a></p><pre class="sequence">        10         20         30         40         50<br/>MGLPRLVCAF LLAACCCCPR VAGVPGEAEQ PAPELVEVEV GSTALLKCGL <br/>        60         70         80         90        100<br/>SQSQGNLSHV DWFSVHKEKR TLIFRVRQGQ GQSEPGEYEQ RLSLQDRGAT <br/>       110        120        130        140        150<br/>LALTQVTPQD ERIFLCQGKR PRSQEYRIQL RVYKAPEEPN IQVNPLGIPV <br/>       160        170        180        190        200<br/>NSKEPEEVAT CVGRNGYPIP QVIWYKNGRP LKEEKNRVHI QSSQTVESSG <br/>       210        220        230        240        250<br/>LYTLQSILKA QLVKEDKDAQ FYCELNYRLP SGNHMKESRE VTVPVFYPTE <br/>       260        270        280        290        300<br/>KVWLEVEPVG MLKEGDRVEI RCLADGNPPP HFSISKQNPS TREAEEETTN <br/>       310        320        330        340        350<br/>DNGVLVLEPA RKEHSGRYEC QGLDLDTMIS LLSEPQELLV NYVSDVRVSP <br/>       360        370        380        390        400<br/>AAPERQEGSS LTLTCEAESS QDLEFQWLRE ETGQVLERGP VLQLHDLKRE <br/>       410        420        430        440        450<br/>AGGGYRCVAS VPSIPGLNRT QLVNVAIFGP PWMAFKERKV WVKENMVLNL <br/>       460        470        480        490        500<br/>SCEASGHPRP TISWNVNGTA SEQDQDPQRV LSTLNVLVTP ELLETGVECT <br/>       510        520        530        540        550<br/>ASNDLGKNTS ILFLELVNLT TLTPDSNTTT GLSTSTASPH TRANSTSTER <br/>       560        570        580        590        600<br/>KLPEPESRGV VIVAVIVCIL VLAVLGAVLY FLYKKGKLPC RRSGKQEITL <br/>       610        620        630        640 <br/>PPSRKSELVV EVKSDKLPEE MGLLQGSSGD KRAPGDQGEK YIDLRH <br/></pre></div><span id="P43121-show-link" class="show-link" style="display:none"><a href="#" onclick="$('#P43121-details').show(); $('#P43121-show-link').hide(); return false;">Show </a>»</span></div><div class="sequence-isoform-rightcol"><div><span class="sequence-field-header tooltiped" title="Sequence length.">Length:</span><span>646</span></div><div><span title="The mass of the unprocessed protein, in Daltons." class="sequence-field-header tooltiped">Mass (Da):</span><span>71,607</span></div><div><span class="sequence-field-header">Last modified:</span><span>January 10, 2006 - v2</span></div><div><span class="sequence-field-header"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
				&#xd;
				&lt;p>The checksum is a form of redundancy check that is calculated &#xd;
					from the sequence. It is useful for tracking sequence updates.&lt;/p>&#xd;
&#xd;
				&lt;p>It should be noted that while, in theory, two different sequences could &#xd;
					have the same checksum value, the likelihood that this would happen &#xd;
					is extremely low.&lt;/p>&#xd;
&#xd;
				&lt;p>However UniProtKB may contain entries with identical sequences in case &#xd;
					of multiple genes (paralogs).&lt;/p>	&#xd;
				&#xd;
				&lt;p>The checksum is computed as the sequence 64-bit Cyclic Redundancy Check value (CRC64)&#xd;
					using the generator polynomial: x&lt;sup>64&lt;/sup> + x&lt;sup>4&lt;/sup> + x&lt;sup>3&lt;/sup> + x + 1.&#xd;
					The algorithm is described in the ISO 3309 standard. &#xd;
				&lt;/p>&#xd;
				&#xd;
				&lt;p class="publication">Press W.H., Flannery B.P., Teukolsky S.A. and Vetterling W.T.&lt;br /> &#xd;
					&lt;strong>Cyclic redundancy and other checksums&lt;/strong>&lt;br />&#xd;
					&lt;a href="http://www.nrbook.com/b/bookcpdf.php">Numerical recipes in C 2nd ed., pp896-902, Cambridge University Press (1993)&lt;/a>)&lt;/p>&#xd;
			      </span>Checksum:<sup>i</sup></span></span>E46CB8AC7BA0738E</div><div class="tools displayThis"><span class="no-wrap"><select id="tools-P43121"><option value="P43121">BLAST</option><option value="http://web.expasy.org/cgi-bin/protparam/protparam?P43121">ProtParam</option><option value="http://web.expasy.org/cgi-bin/protscale/protscale.pl?P43121">ProtScale</option><option value="http://web.expasy.org/cgi-bin/compute_pi/pi_tool?P43121">Compute pI/MW</option><option value="http://web.expasy.org/cgi-bin/peptide_mass/peptide-mass.pl?P43121">PeptideMass</option><option value="http://web.expasy.org/cgi-bin/peptide_cutter/peptidecutter.pl?P43121">PeptideCutter</option></select><a class="button go run-isoform-tool" href="">GO</a></span></div></div></div><div class="sequence-isoform"><input type="checkbox" class="cart-item" onclick="addOrAppendCart('P43121-2', 'section_seq')" id="checkbox_P43121-2"/><div class="sequence-isoform-leftcol"><span class="sequence-isoform-header"><a id="P43121-2" href="../blast/?about=P43121-2">Isoform 2</a></span> (identifier: <strong>P43121-2</strong>)  [<a href="/uniparc/?query=isoform:P43121-2&amp;direct=yes">UniParc</a>]<a href="/uniprot/P43121-2.fasta" class="tooltipped icon icon-functional button displayThis" data-icon="=" title="Sequence data in FASTA format" rel="nofollow">FASTA</a><a id="add-isoform-P43121-2-basket" class="icon icon-static button tooltipped add-basket-button P43121-2" title="Add this isoform to basket" data-icon="b">Add to basket</a><a id="added-isoform-P43121-2-basket" class="icon icon-static button disabled added-basket-button P43121-2" title="This isoform is in your basket" style="display:none" data-icon="b">Added to basket</a><div class="details"></div><p><i>The sequence of this isoform differs from the canonical sequence as follows:</i><br/>     <a href="/blast/?about=P43121%5B1-187%5D">1-187</a>: MGLPRLVCAF...GRPLKEEKNR → MVYIVRQFLL...SLPPLPPCPG<br/>     <a href="/blast/?about=P43121%5B549-646%5D">549-646</a>: ERKLPEPESR...QGEKYIDLRH → GKPGLAREQGCARASFLPCPSPESPVQKGE<br/></p><div class="sequence" style="display:none" id="P43121-2-details"><p style="clear: both" class="details"><a href="#" onclick="$('#P43121-2-details').hide(); $('#P43121-2-show-link').show(); return false;">« Hide</a></p><pre class="sequence">        10         20         30         40         50<br/>MVYIVRQFLL YNVSGSVYLD QLIVLLTAKF SILRIAGSRV HHSPFSGHLD <br/>        60         70         80         90        100<br/>GCSFLSLQHS LHTSLDMSRH ENVFLGLTLS SKSAGLKGFQ LAFVPGLLQG <br/>       110        120        130        140        150<br/>TGGYLDGPLP TPVDNPRVGL EVGLRLSLPP LPPCPGVHIQ SSQTVESSGL <br/>       160        170        180        190        200<br/>YTLQSILKAQ LVKEDKDAQF YCELNYRLPS GNHMKESREV TVPVFYPTEK <br/>       210        220        230        240        250<br/>VWLEVEPVGM LKEGDRVEIR CLADGNPPPH FSISKQNPST REAEEETTND <br/>       260        270        280        290        300<br/>NGVLVLEPAR KEHSGRYECQ GLDLDTMISL LSEPQELLVN YVSDVRVSPA <br/>       310        320        330        340        350<br/>APERQEGSSL TLTCEAESSQ DLEFQWLREE TGQVLERGPV LQLHDLKREA <br/>       360        370        380        390        400<br/>GGGYRCVASV PSIPGLNRTQ LVNVAIFGPP WMAFKERKVW VKENMVLNLS <br/>       410        420        430        440        450<br/>CEASGHPRPT ISWNVNGTAS EQDQDPQRVL STLNVLVTPE LLETGVECTA <br/>       460        470        480        490        500<br/>SNDLGKNTSI LFLELVNLTT LTPDSNTTTG LSTSTASPHT RANSTSTGKP <br/>       510        520 <br/>GLAREQGCAR ASFLPCPSPE SPVQKGE                       <br/></pre></div><div class="details"><span class="annotation_note">Note:</span> <span>No experimental confirmation available.</span></div><span id="P43121-2-show-link" class="show-link"><a href="#" onclick="$('#P43121-2-details').show(); $('#P43121-2-show-link').hide(); return false;">Show </a>»</span></div><div class="sequence-isoform-rightcol"><div><span class="sequence-field-header tooltiped" title="Sequence length.">Length:</span><span>527</span></div><div><span title="The mass of the unprocessed protein, in Daltons." class="sequence-field-header tooltiped">Mass (Da):</span><span>57,605</span></div><div><span class="sequence-field-header"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
				&#xd;
				&lt;p>The checksum is a form of redundancy check that is calculated &#xd;
					from the sequence. It is useful for tracking sequence updates.&lt;/p>&#xd;
&#xd;
				&lt;p>It should be noted that while, in theory, two different sequences could &#xd;
					have the same checksum value, the likelihood that this would happen &#xd;
					is extremely low.&lt;/p>&#xd;
&#xd;
				&lt;p>However UniProtKB may contain entries with identical sequences in case &#xd;
					of multiple genes (paralogs).&lt;/p>	&#xd;
				&#xd;
				&lt;p>The checksum is computed as the sequence 64-bit Cyclic Redundancy Check value (CRC64)&#xd;
					using the generator polynomial: x&lt;sup>64&lt;/sup> + x&lt;sup>4&lt;/sup> + x&lt;sup>3&lt;/sup> + x + 1.&#xd;
					The algorithm is described in the ISO 3309 standard. &#xd;
				&lt;/p>&#xd;
				&#xd;
				&lt;p class="publication">Press W.H., Flannery B.P., Teukolsky S.A. and Vetterling W.T.&lt;br /> &#xd;
					&lt;strong>Cyclic redundancy and other checksums&lt;/strong>&lt;br />&#xd;
					&lt;a href="http://www.nrbook.com/b/bookcpdf.php">Numerical recipes in C 2nd ed., pp896-902, Cambridge University Press (1993)&lt;/a>)&lt;/p>&#xd;
			      </span>Checksum:<sup>i</sup></span></span>68F5C36C5FFD36D6</div><div class="tools displayThis"><span class="no-wrap"><select id="tools-P43121-2"><option value="P43121-2">BLAST</option><option value="http://web.expasy.org/cgi-bin/protparam/protparam?P43121-2">ProtParam</option><option value="http://web.expasy.org/cgi-bin/protscale/protscale.pl?P43121-2">ProtScale</option><option value="http://web.expasy.org/cgi-bin/compute_pi/pi_tool?P43121-2">Compute pI/MW</option><option value="http://web.expasy.org/cgi-bin/peptide_mass/peptide-mass.pl?P43121-2">PeptideMass</option><option value="http://web.expasy.org/cgi-bin/peptide_cutter/peptidecutter.pl?P43121-2">PeptideCutter</option></select><a class="button go run-isoform-tool" href="">GO</a></span></div></div></div></div><h4><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Sequence’ section reports difference(s) between the protein sequence shown in the UniProtKB entry and other available protein sequences derived from the same gene.&lt;p>&lt;a href='/help/sequence_caution' target='_top'>More...&lt;/a>&lt;/p></span>Sequence caution<sup>i</sup></span></h4><div class="annotation" property="schema:hasPart" typeof="schema:CreativeWork">The sequence <a href="http://www.ebi.ac.uk/ena/data/view/BAD93162">BAD93162</a> differs from that shown. Reason: Erroneous initiation.<span class="attribution ECO305"><span class="attributionHeader tooltipped" title="Manual assertion inferred by curator">Curated</span></span></div><h4>Experimental Info</h4><table class="featureTable" id="Sequence_conflict_section"><tr><th class="featkey_h">Feature key</th><th class="pos_h">Position(s)</th><th class="description_h">Description<span class="featactions_h">Actions</span></th><th class="graphview_h">Graphical view</th><th class="length_h">Length</th></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Sequence’ section reports difference(s) between the canonical sequence (displayed by default in the entry) and the different sequence submissions merged in the entry. These various submissions may originate from different sequencing projects, different types of experiments, or different biological samples. Sequence conflicts are usually of unknown origin.&lt;p>&lt;a href='/help/conflict' target='_top'>More...&lt;/a>&lt;/p></span>Sequence conflict<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[322-324]&amp;key=Sequence conflict">322 – 324</a></td><td class="featdescription">GLD → AWN (PubMed:<a href="/citations/2602381">2602381</a>).<span class="attribution ECO305"><span class="attributionHeader tooltipped" title="Manual assertion inferred by curator">Curated</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="49.690%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">3</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Sequence’ section reports difference(s) between the canonical sequence (displayed by default in the entry) and the different sequence submissions merged in the entry. These various submissions may originate from different sequencing projects, different types of experiments, or different biological samples. Sequence conflicts are usually of unknown origin.&lt;p>&lt;a href='/help/conflict' target='_top'>More...&lt;/a>&lt;/p></span>Sequence conflict<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[322-324]&amp;key=Sequence conflict">322 – 324</a></td><td class="featdescription">GLD → AWN (PubMed:<a href="/citations/8378324">8378324</a>).<span class="attribution ECO305"><span class="attributionHeader tooltipped" title="Manual assertion inferred by curator">Curated</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="49.690%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">3</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Sequence’ section reports difference(s) between the canonical sequence (displayed by default in the entry) and the different sequence submissions merged in the entry. These various submissions may originate from different sequencing projects, different types of experiments, or different biological samples. Sequence conflicts are usually of unknown origin.&lt;p>&lt;a href='/help/conflict' target='_top'>More...&lt;/a>&lt;/p></span>Sequence conflict<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[383]&amp;key=Sequence conflict">383</a></td><td class="featdescription">G → D (PubMed:<a href="/citations/2602381">2602381</a>).<span class="attribution ECO305"><span class="attributionHeader tooltipped" title="Manual assertion inferred by curator">Curated</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="59.133%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Sequence’ section reports difference(s) between the canonical sequence (displayed by default in the entry) and the different sequence submissions merged in the entry. These various submissions may originate from different sequencing projects, different types of experiments, or different biological samples. Sequence conflicts are usually of unknown origin.&lt;p>&lt;a href='/help/conflict' target='_top'>More...&lt;/a>&lt;/p></span>Sequence conflict<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[383]&amp;key=Sequence conflict">383</a></td><td class="featdescription">G → D (PubMed:<a href="/citations/8378324">8378324</a>).<span class="attribution ECO305"><span class="attributionHeader tooltipped" title="Manual assertion inferred by curator">Curated</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="59.133%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Sequence’ section reports difference(s) between the canonical sequence (displayed by default in the entry) and the different sequence submissions merged in the entry. These various submissions may originate from different sequencing projects, different types of experiments, or different biological samples. Sequence conflicts are usually of unknown origin.&lt;p>&lt;a href='/help/conflict' target='_top'>More...&lt;/a>&lt;/p></span>Sequence conflict<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[383]&amp;key=Sequence conflict">383</a></td><td class="featdescription">G → D (PubMed:<a href="/citations/11709656">11709656</a>).<span class="attribution ECO305"><span class="attributionHeader tooltipped" title="Manual assertion inferred by curator">Curated</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="59.133%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Sequence’ section reports difference(s) between the canonical sequence (displayed by default in the entry) and the different sequence submissions merged in the entry. These various submissions may originate from different sequencing projects, different types of experiments, or different biological samples. Sequence conflicts are usually of unknown origin.&lt;p>&lt;a href='/help/conflict' target='_top'>More...&lt;/a>&lt;/p></span>Sequence conflict<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[424-425]&amp;key=Sequence conflict">424 – 425</a></td><td class="featdescription">NV → KL (PubMed:<a href="/citations/2602381">2602381</a>).<span class="attribution ECO305"><span class="attributionHeader tooltipped" title="Manual assertion inferred by curator">Curated</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="65.479%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">2</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Sequence’ section reports difference(s) between the canonical sequence (displayed by default in the entry) and the different sequence submissions merged in the entry. These various submissions may originate from different sequencing projects, different types of experiments, or different biological samples. Sequence conflicts are usually of unknown origin.&lt;p>&lt;a href='/help/conflict' target='_top'>More...&lt;/a>&lt;/p></span>Sequence conflict<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[424-425]&amp;key=Sequence conflict">424 – 425</a></td><td class="featdescription">NV → KL (PubMed:<a href="/citations/8378324">8378324</a>).<span class="attribution ECO305"><span class="attributionHeader tooltipped" title="Manual assertion inferred by curator">Curated</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="65.479%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">2</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Sequence’ section reports difference(s) between the canonical sequence (displayed by default in the entry) and the different sequence submissions merged in the entry. These various submissions may originate from different sequencing projects, different types of experiments, or different biological samples. Sequence conflicts are usually of unknown origin.&lt;p>&lt;a href='/help/conflict' target='_top'>More...&lt;/a>&lt;/p></span>Sequence conflict<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[606]&amp;key=Sequence conflict">606</a></td><td class="featdescription">S → T (PubMed:<a href="/citations/2602381">2602381</a>).<span class="attribution ECO305"><span class="attributionHeader tooltipped" title="Manual assertion inferred by curator">Curated</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="93.653%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Sequence’ section reports difference(s) between the canonical sequence (displayed by default in the entry) and the different sequence submissions merged in the entry. These various submissions may originate from different sequencing projects, different types of experiments, or different biological samples. Sequence conflicts are usually of unknown origin.&lt;p>&lt;a href='/help/conflict' target='_top'>More...&lt;/a>&lt;/p></span>Sequence conflict<sup>i</sup></span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[606]&amp;key=Sequence conflict">606</a></td><td class="featdescription">S → T (PubMed:<a href="/citations/8378324">8378324</a>).<span class="attribution ECO305"><span class="attributionHeader tooltipped" title="Manual assertion inferred by curator">Curated</span></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="93.653%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr></table><h4>Natural variant</h4><table class="featureTable" id="Natural_variant_section"><tr><th class="featkey_h">Feature key</th><th class="pos_h">Position(s)</th><th class="description_h">Description<span class="featactions_h">Actions</span></th><th class="graphview_h">Graphical view</th><th class="length_h">Length</th></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork" resource="http://purl.uniprot.org/annotation/VAR_049915" id="VAR_049915"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Sequence’ section describes natural variant(s) of the protein sequence.&lt;p>&lt;a href='/help/variant' target='_top'>More...&lt;/a>&lt;/p></span>Natural variant<sup>i</sup></span><span class="featureId">VAR_049915</span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[89]&amp;key=Natural variant&amp;id=VAR_049915">89</a></td><td class="featdescription"><a href="http://web.expasy.org/variant_pages/VAR_049915.html">E → G</a><span class="attribution ECO269"><span class="attributionHeader ">1 Publication<span class="showHideEvidence caret_grey displayThisInline"></span></span><span style="display:none" class="evidenceContainer"><p class="attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Manually curated information for which there is published experimental evidence.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>&lt;a href="/manual/evidences#ECO:0000269">More…&lt;/a>&lt;/p>      </span>Manual assertion based on experiment in<sup>i</sup></span></p><ul><li><div class="P43121#ref5 referenceAttribution"><div class="reference_header">Ref.5</div><div class="reference_content"><div property="schema:citation" typeof="schema:CreativeWork"><a href="/uniprot/?query=author:%22Totoki+Y.%22" rel="nofollow">Totoki Y.</a>, <a href="/uniprot/?query=author:%22Toyoda+A.%22" rel="nofollow">Toyoda A.</a>, <a href="/uniprot/?query=author:%22Takeda+T.%22" rel="nofollow">Takeda T.</a>, <a href="/uniprot/?query=author:%22Sakaki+Y.%22" rel="nofollow">Sakaki Y.</a>, <a href="/uniprot/?query=author:%22Tanaka+A.%22" rel="nofollow">Tanaka A.</a>, <a href="/uniprot/?query=author:%22Yokoyama+S.%22" rel="nofollow">Yokoyama S.</a>, <a href="/uniprot/?query=author:%22Ohara+O.%22" rel="nofollow">Ohara O.</a>, <a href="/uniprot/?query=author:%22Nagase+T.%22" rel="nofollow">Nagase T.</a>, <a href="/uniprot/?query=author:%22Kikuno+R.F.%22" rel="nofollow">Kikuno R.F.</a><br/>Submitted (MAR-2005) to the EMBL/GenBank/DDBJ databases</div><div class="citedFor"><span class="details"><strong>Cited for:</strong></span> NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA] (ISOFORM 1), VARIANT GLY-89.</div></div></div></li></ul></span></span>Corresponds to variant <a href="https://www.ncbi.nlm.nih.gov/SNP/snp_ref.cgi?type=rs&amp;rs=34587557">dbSNP:rs34587557</a><span class="featactions"><span class="dbsnp rs34587557"><span class="acronym" title="Ensembl eukaryotic genome annotation project"><a href="http://www.ensembl.org/Homo_sapiens/Variation/Explore?v=rs34587557">Ensembl</a></span></span>.</span></td><td class="ft-sequence"><svg class="featureImage"><rect x="13.622%" width="1%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">1</td></tr></table><h4>Alternative sequence</h4><table class="featureTable" id="Alternative_sequence_section"><tr><th class="featkey_h">Feature key</th><th class="pos_h">Position(s)</th><th class="description_h">Description<span class="featactions_h">Actions</span></th><th class="graphview_h">Graphical view</th><th class="length_h">Length</th></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork" resource="http://purl.uniprot.org/annotation/VSP_016938" id="VSP_016938"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Sequence’ section describes the sequence of naturally occurring alternative protein isoform(s). The changes in the amino acid sequence may be due to alternative splicing, alternative promoter usage, alternative initiation, or ribosomal frameshifting. The information stored in this subsection is used to automatically construct alternative protein sequence(s) for display.&lt;p>&lt;a href='/help/var_seq' target='_top'>More...&lt;/a>&lt;/p></span>Alternative sequence<sup>i</sup></span><span class="featureId">VSP_016938</span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[1-187]&amp;key=Alternative sequence&amp;id=VSP_016938">1 – 187</a></td><td class="featdescription">MGLPR…EEKNR → MVYIVRQFLLYNVSGSVYLD QLIVLLTAKFSILRIAGSRV HHSPFSGHLDGCSFLSLQHS LHTSLDMSRHENVFLGLTLS SKSAGLKGFQLAFVPGLLQG TGGYLDGPLPTPVDNPRVGL EVGLRLSLPPLPPCPG <span property="text">in isoform <a href="#P43121-2">2</a>.</span> <span class="attribution ECO303"><span class="attributionHeader ">1 Publication<span class="showHideEvidence caret_grey displayThisInline"></span></span><span style="display:none" class="evidenceContainer"><p class="attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Manually curated information that is based on statements in scientific articles for which there is no experimental support.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>&lt;a href="/manual/evidences#ECO:0000303">More…&lt;/a>&lt;/p>      </span>Manual assertion based on opinion in<sup>i</sup></span></p><ul><li><div class="P43121#ref4 referenceAttribution"><div class="reference_header">Ref.4</div><div class="reference_content"><div property="schema:citation" resource="http://purl.uniprot.org/citations/14702039" typeof="schema:ScholarlyArticle"><strong property="schema:name">"Complete sequencing and characterization of 21,243 full-length human cDNAs."</strong><br/><a href="/uniprot/?query=author:%22Ota+T.%22" rel="nofollow">Ota T.</a>, <a href="/uniprot/?query=author:%22Suzuki+Y.%22" rel="nofollow">Suzuki Y.</a>, <a href="/uniprot/?query=author:%22Nishikawa+T.%22" rel="nofollow">Nishikawa T.</a>, <a href="/uniprot/?query=author:%22Otsuki+T.%22" rel="nofollow">Otsuki T.</a>, <a href="/uniprot/?query=author:%22Sugiyama+T.%22" rel="nofollow">Sugiyama T.</a>, <a href="/uniprot/?query=author:%22Irie+R.%22" rel="nofollow">Irie R.</a>, <a href="/uniprot/?query=author:%22Wakamatsu+A.%22" rel="nofollow">Wakamatsu A.</a>, <a href="/uniprot/?query=author:%22Hayashi+K.%22" rel="nofollow">Hayashi K.</a>, <a href="/uniprot/?query=author:%22Sato+H.%22" rel="nofollow">Sato H.</a>, <a href="/uniprot/?query=author:%22Nagai+K.%22" rel="nofollow">Nagai K.</a>, <a href="/uniprot/?query=author:%22Kimura+K.%22" rel="nofollow">Kimura K.</a>, <a href="/uniprot/?query=author:%22Makita+H.%22" rel="nofollow">Makita H.</a>, <a href="/uniprot/?query=author:%22Sekine+M.%22" rel="nofollow">Sekine M.</a>, <a href="/uniprot/?query=author:%22Obayashi+M.%22" rel="nofollow">Obayashi M.</a>, <a href="/uniprot/?query=author:%22Nishi+T.%22" rel="nofollow">Nishi T.</a>, <a href="/uniprot/?query=author:%22Shibahara+T.%22" rel="nofollow">Shibahara T.</a>, <a href="/uniprot/?query=author:%22Tanaka+T.%22" rel="nofollow">Tanaka T.</a>, <a href="/uniprot/?query=author:%22Ishii+S.%22" rel="nofollow">Ishii S.</a><span> <div class="clickable authors icon_expand" title="Show all authors"></div> </span><span style="display:none">, <a href="/uniprot/?query=author:%22Yamamoto+J.%22" rel="nofollow">Yamamoto J.</a>, <a href="/uniprot/?query=author:%22Saito+K.%22" rel="nofollow">Saito K.</a>, <a href="/uniprot/?query=author:%22Kawai+Y.%22" rel="nofollow">Kawai Y.</a>, <a href="/uniprot/?query=author:%22Isono+Y.%22" rel="nofollow">Isono Y.</a>, <a href="/uniprot/?query=author:%22Nakamura+Y.%22" rel="nofollow">Nakamura Y.</a>, <a href="/uniprot/?query=author:%22Nagahari+K.%22" rel="nofollow">Nagahari K.</a>, <a href="/uniprot/?query=author:%22Murakami+K.%22" rel="nofollow">Murakami K.</a>, <a href="/uniprot/?query=author:%22Yasuda+T.%22" rel="nofollow">Yasuda T.</a>, <a href="/uniprot/?query=author:%22Iwayanagi+T.%22" rel="nofollow">Iwayanagi T.</a>, <a href="/uniprot/?query=author:%22Wagatsuma+M.%22" rel="nofollow">Wagatsuma M.</a>, <a href="/uniprot/?query=author:%22Shiratori+A.%22" rel="nofollow">Shiratori A.</a>, <a href="/uniprot/?query=author:%22Sudo+H.%22" rel="nofollow">Sudo H.</a>, <a href="/uniprot/?query=author:%22Hosoiri+T.%22" rel="nofollow">Hosoiri T.</a>, <a href="/uniprot/?query=author:%22Kaku+Y.%22" rel="nofollow">Kaku Y.</a>, <a href="/uniprot/?query=author:%22Kodaira+H.%22" rel="nofollow">Kodaira H.</a>, <a href="/uniprot/?query=author:%22Kondo+H.%22" rel="nofollow">Kondo H.</a>, <a href="/uniprot/?query=author:%22Sugawara+M.%22" rel="nofollow">Sugawara M.</a>, <a href="/uniprot/?query=author:%22Takahashi+M.%22" rel="nofollow">Takahashi M.</a>, <a href="/uniprot/?query=author:%22Kanda+K.%22" rel="nofollow">Kanda K.</a>, <a href="/uniprot/?query=author:%22Yokoi+T.%22" rel="nofollow">Yokoi T.</a>, <a href="/uniprot/?query=author:%22Furuya+T.%22" rel="nofollow">Furuya T.</a>, <a href="/uniprot/?query=author:%22Kikkawa+E.%22" rel="nofollow">Kikkawa E.</a>, <a href="/uniprot/?query=author:%22Omura+Y.%22" rel="nofollow">Omura Y.</a>, <a href="/uniprot/?query=author:%22Abe+K.%22" rel="nofollow">Abe K.</a>, <a href="/uniprot/?query=author:%22Kamihara+K.%22" rel="nofollow">Kamihara K.</a>, <a href="/uniprot/?query=author:%22Katsuta+N.%22" rel="nofollow">Katsuta N.</a>, <a href="/uniprot/?query=author:%22Sato+K.%22" rel="nofollow">Sato K.</a>, <a href="/uniprot/?query=author:%22Tanikawa+M.%22" rel="nofollow">Tanikawa M.</a>, <a href="/uniprot/?query=author:%22Yamazaki+M.%22" rel="nofollow">Yamazaki M.</a>, <a href="/uniprot/?query=author:%22Ninomiya+K.%22" rel="nofollow">Ninomiya K.</a>, <a href="/uniprot/?query=author:%22Ishibashi+T.%22" rel="nofollow">Ishibashi T.</a>, <a href="/uniprot/?query=author:%22Yamashita+H.%22" rel="nofollow">Yamashita H.</a>, <a href="/uniprot/?query=author:%22Murakawa+K.%22" rel="nofollow">Murakawa K.</a>, <a href="/uniprot/?query=author:%22Fujimori+K.%22" rel="nofollow">Fujimori K.</a>, <a href="/uniprot/?query=author:%22Tanai+H.%22" rel="nofollow">Tanai H.</a>, <a href="/uniprot/?query=author:%22Kimata+M.%22" rel="nofollow">Kimata M.</a>, <a href="/uniprot/?query=author:%22Watanabe+M.%22" rel="nofollow">Watanabe M.</a>, <a href="/uniprot/?query=author:%22Hiraoka+S.%22" rel="nofollow">Hiraoka S.</a>, <a href="/uniprot/?query=author:%22Chiba+Y.%22" rel="nofollow">Chiba Y.</a>, <a href="/uniprot/?query=author:%22Ishida+S.%22" rel="nofollow">Ishida S.</a>, <a href="/uniprot/?query=author:%22Ono+Y.%22" rel="nofollow">Ono Y.</a>, <a href="/uniprot/?query=author:%22Takiguchi+S.%22" rel="nofollow">Takiguchi S.</a>, <a href="/uniprot/?query=author:%22Watanabe+S.%22" rel="nofollow">Watanabe S.</a>, <a href="/uniprot/?query=author:%22Yosida+M.%22" rel="nofollow">Yosida M.</a>, <a href="/uniprot/?query=author:%22Hotuta+T.%22" rel="nofollow">Hotuta T.</a>, <a href="/uniprot/?query=author:%22Kusano+J.%22" rel="nofollow">Kusano J.</a>, <a href="/uniprot/?query=author:%22Kanehori+K.%22" rel="nofollow">Kanehori K.</a>, <a href="/uniprot/?query=author:%22Takahashi-Fujii+A.%22" rel="nofollow">Takahashi-Fujii A.</a>, <a href="/uniprot/?query=author:%22Hara+H.%22" rel="nofollow">Hara H.</a>, <a href="/uniprot/?query=author:%22Tanase+T.-O.%22" rel="nofollow">Tanase T.-O.</a>, <a href="/uniprot/?query=author:%22Nomura+Y.%22" rel="nofollow">Nomura Y.</a>, <a href="/uniprot/?query=author:%22Togiya+S.%22" rel="nofollow">Togiya S.</a>, <a href="/uniprot/?query=author:%22Komai+F.%22" rel="nofollow">Komai F.</a>, <a href="/uniprot/?query=author:%22Hara+R.%22" rel="nofollow">Hara R.</a>, <a href="/uniprot/?query=author:%22Takeuchi+K.%22" rel="nofollow">Takeuchi K.</a>, <a href="/uniprot/?query=author:%22Arita+M.%22" rel="nofollow">Arita M.</a>, <a href="/uniprot/?query=author:%22Imose+N.%22" rel="nofollow">Imose N.</a>, <a href="/uniprot/?query=author:%22Musashino+K.%22" rel="nofollow">Musashino K.</a>, <a href="/uniprot/?query=author:%22Yuuki+H.%22" rel="nofollow">Yuuki H.</a>, <a href="/uniprot/?query=author:%22Oshima+A.%22" rel="nofollow">Oshima A.</a>, <a href="/uniprot/?query=author:%22Sasaki+N.%22" rel="nofollow">Sasaki N.</a>, <a href="/uniprot/?query=author:%22Aotsuka+S.%22" rel="nofollow">Aotsuka S.</a>, <a href="/uniprot/?query=author:%22Yoshikawa+Y.%22" rel="nofollow">Yoshikawa Y.</a>, <a href="/uniprot/?query=author:%22Matsunawa+H.%22" rel="nofollow">Matsunawa H.</a>, <a href="/uniprot/?query=author:%22Ichihara+T.%22" rel="nofollow">Ichihara T.</a>, <a href="/uniprot/?query=author:%22Shiohata+N.%22" rel="nofollow">Shiohata N.</a>, <a href="/uniprot/?query=author:%22Sano+S.%22" rel="nofollow">Sano S.</a>, <a href="/uniprot/?query=author:%22Moriya+S.%22" rel="nofollow">Moriya S.</a>, <a href="/uniprot/?query=author:%22Momiyama+H.%22" rel="nofollow">Momiyama H.</a>, <a href="/uniprot/?query=author:%22Satoh+N.%22" rel="nofollow">Satoh N.</a>, <a href="/uniprot/?query=author:%22Takami+S.%22" rel="nofollow">Takami S.</a>, <a href="/uniprot/?query=author:%22Terashima+Y.%22" rel="nofollow">Terashima Y.</a>, <a href="/uniprot/?query=author:%22Suzuki+O.%22" rel="nofollow">Suzuki O.</a>, <a href="/uniprot/?query=author:%22Nakagawa+S.%22" rel="nofollow">Nakagawa S.</a>, <a href="/uniprot/?query=author:%22Senoh+A.%22" rel="nofollow">Senoh A.</a>, <a href="/uniprot/?query=author:%22Mizoguchi+H.%22" rel="nofollow">Mizoguchi H.</a>, <a href="/uniprot/?query=author:%22Goto+Y.%22" rel="nofollow">Goto Y.</a>, <a href="/uniprot/?query=author:%22Shimizu+F.%22" rel="nofollow">Shimizu F.</a>, <a href="/uniprot/?query=author:%22Wakebe+H.%22" rel="nofollow">Wakebe H.</a>, <a href="/uniprot/?query=author:%22Hishigaki+H.%22" rel="nofollow">Hishigaki H.</a>, <a href="/uniprot/?query=author:%22Watanabe+T.%22" rel="nofollow">Watanabe T.</a>, <a href="/uniprot/?query=author:%22Sugiyama+A.%22" rel="nofollow">Sugiyama A.</a>, <a href="/uniprot/?query=author:%22Takemoto+M.%22" rel="nofollow">Takemoto M.</a>, <a href="/uniprot/?query=author:%22Kawakami+B.%22" rel="nofollow">Kawakami B.</a>, <a href="/uniprot/?query=author:%22Yamazaki+M.%22" rel="nofollow">Yamazaki M.</a>, <a href="/uniprot/?query=author:%22Watanabe+K.%22" rel="nofollow">Watanabe K.</a>, <a href="/uniprot/?query=author:%22Kumagai+A.%22" rel="nofollow">Kumagai A.</a>, <a href="/uniprot/?query=author:%22Itakura+S.%22" rel="nofollow">Itakura S.</a>, <a href="/uniprot/?query=author:%22Fukuzumi+Y.%22" rel="nofollow">Fukuzumi Y.</a>, <a href="/uniprot/?query=author:%22Fujimori+Y.%22" rel="nofollow">Fujimori Y.</a>, <a href="/uniprot/?query=author:%22Komiyama+M.%22" rel="nofollow">Komiyama M.</a>, <a href="/uniprot/?query=author:%22Tashiro+H.%22" rel="nofollow">Tashiro H.</a>, <a href="/uniprot/?query=author:%22Tanigami+A.%22" rel="nofollow">Tanigami A.</a>, <a href="/uniprot/?query=author:%22Fujiwara+T.%22" rel="nofollow">Fujiwara T.</a>, <a href="/uniprot/?query=author:%22Ono+T.%22" rel="nofollow">Ono T.</a>, <a href="/uniprot/?query=author:%22Yamada+K.%22" rel="nofollow">Yamada K.</a>, <a href="/uniprot/?query=author:%22Fujii+Y.%22" rel="nofollow">Fujii Y.</a>, <a href="/uniprot/?query=author:%22Ozaki+K.%22" rel="nofollow">Ozaki K.</a>, <a href="/uniprot/?query=author:%22Hirao+M.%22" rel="nofollow">Hirao M.</a>, <a href="/uniprot/?query=author:%22Ohmori+Y.%22" rel="nofollow">Ohmori Y.</a>, <a href="/uniprot/?query=author:%22Kawabata+A.%22" rel="nofollow">Kawabata A.</a>, <a href="/uniprot/?query=author:%22Hikiji+T.%22" rel="nofollow">Hikiji T.</a>, <a href="/uniprot/?query=author:%22Kobatake+N.%22" rel="nofollow">Kobatake N.</a>, <a href="/uniprot/?query=author:%22Inagaki+H.%22" rel="nofollow">Inagaki H.</a>, <a href="/uniprot/?query=author:%22Ikema+Y.%22" rel="nofollow">Ikema Y.</a>, <a href="/uniprot/?query=author:%22Okamoto+S.%22" rel="nofollow">Okamoto S.</a>, <a href="/uniprot/?query=author:%22Okitani+R.%22" rel="nofollow">Okitani R.</a>, <a href="/uniprot/?query=author:%22Kawakami+T.%22" rel="nofollow">Kawakami T.</a>, <a href="/uniprot/?query=author:%22Noguchi+S.%22" rel="nofollow">Noguchi S.</a>, <a href="/uniprot/?query=author:%22Itoh+T.%22" rel="nofollow">Itoh T.</a>, <a href="/uniprot/?query=author:%22Shigeta+K.%22" rel="nofollow">Shigeta K.</a>, <a href="/uniprot/?query=author:%22Senba+T.%22" rel="nofollow">Senba T.</a>, <a href="/uniprot/?query=author:%22Matsumura+K.%22" rel="nofollow">Matsumura K.</a>, <a href="/uniprot/?query=author:%22Nakajima+Y.%22" rel="nofollow">Nakajima Y.</a>, <a href="/uniprot/?query=author:%22Mizuno+T.%22" rel="nofollow">Mizuno T.</a>, <a href="/uniprot/?query=author:%22Morinaga+M.%22" rel="nofollow">Morinaga M.</a>, <a href="/uniprot/?query=author:%22Sasaki+M.%22" rel="nofollow">Sasaki M.</a>, <a href="/uniprot/?query=author:%22Togashi+T.%22" rel="nofollow">Togashi T.</a>, <a href="/uniprot/?query=author:%22Oyama+M.%22" rel="nofollow">Oyama M.</a>, <a href="/uniprot/?query=author:%22Hata+H.%22" rel="nofollow">Hata H.</a>, <a href="/uniprot/?query=author:%22Watanabe+M.%22" rel="nofollow">Watanabe M.</a>, <a href="/uniprot/?query=author:%22Komatsu+T.%22" rel="nofollow">Komatsu T.</a>, <a href="/uniprot/?query=author:%22Mizushima-Sugano+J.%22" rel="nofollow">Mizushima-Sugano J.</a>, <a href="/uniprot/?query=author:%22Satoh+T.%22" rel="nofollow">Satoh T.</a>, <a href="/uniprot/?query=author:%22Shirai+Y.%22" rel="nofollow">Shirai Y.</a>, <a href="/uniprot/?query=author:%22Takahashi+Y.%22" rel="nofollow">Takahashi Y.</a>, <a href="/uniprot/?query=author:%22Nakagawa+K.%22" rel="nofollow">Nakagawa K.</a>, <a href="/uniprot/?query=author:%22Okumura+K.%22" rel="nofollow">Okumura K.</a>, <a href="/uniprot/?query=author:%22Nagase+T.%22" rel="nofollow">Nagase T.</a>, <a href="/uniprot/?query=author:%22Nomura+N.%22" rel="nofollow">Nomura N.</a>, <a href="/uniprot/?query=author:%22Kikuchi+H.%22" rel="nofollow">Kikuchi H.</a>, <a href="/uniprot/?query=author:%22Masuho+Y.%22" rel="nofollow">Masuho Y.</a>, <a href="/uniprot/?query=author:%22Yamashita+R.%22" rel="nofollow">Yamashita R.</a>, <a href="/uniprot/?query=author:%22Nakai+K.%22" rel="nofollow">Nakai K.</a>, <a href="/uniprot/?query=author:%22Yada+T.%22" rel="nofollow">Yada T.</a>, <a href="/uniprot/?query=author:%22Nakamura+Y.%22" rel="nofollow">Nakamura Y.</a>, <a href="/uniprot/?query=author:%22Ohara+O.%22" rel="nofollow">Ohara O.</a>, <a href="/uniprot/?query=author:%22Isogai+T.%22" rel="nofollow">Isogai T.</a>, </span><a href="/uniprot/?query=author:%22Sugano+S.%22" rel="nofollow">Sugano S.</a><br/><a href="http://dx.doi.org/10.1038/ng1285">Nat. Genet. 36:40-45(2004)</a>  [<a property="schema:sameAs" href="https://www.ncbi.nlm.nih.gov/pubmed/14702039">PubMed</a>] [<a property="schema:sameAs" href="http://europepmc.org/abstract/MED/14702039">Europe PMC</a>] [<a href="/citations/14702039">Abstract</a>]</div><div class="citedFor"><span class="details"><strong>Cited for:</strong></span> NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA] (ISOFORM 2).</div></div></div></li></ul></span></span><span class="featactions"><a class="icon icon-static tooltipped subsequences" data-icon="b" id="P43121_1-187_" title="Add to basket">Add</a> <a class="icon icon-functional tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">BLAST</a></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="0%" width="28.947%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">187</td></tr><tr class="feature_row" property="schema:hasPart" typeof="schema:CreativeWork" resource="http://purl.uniprot.org/annotation/VSP_016939" id="VSP_016939"><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Sequence’ section describes the sequence of naturally occurring alternative protein isoform(s). The changes in the amino acid sequence may be due to alternative splicing, alternative promoter usage, alternative initiation, or ribosomal frameshifting. The information stored in this subsection is used to automatically construct alternative protein sequence(s) for display.&lt;p>&lt;a href='/help/var_seq' target='_top'>More...&lt;/a>&lt;/p></span>Alternative sequence<sup>i</sup></span><span class="featureId">VSP_016939</span></td><td class="numeric"><a class="position tooltipped" title="BLAST subsequence" href="/blast/?about=P43121[549-646]&amp;key=Alternative sequence&amp;id=VSP_016939">549 – 646</a></td><td class="featdescription">ERKLP…IDLRH → GKPGLAREQGCARASFLPCP SPESPVQKGE <span property="text">in isoform <a href="#P43121-2">2</a>.</span> <span class="attribution ECO303"><span class="attributionHeader ">1 Publication<span class="showHideEvidence caret_grey displayThisInline"></span></span><span style="display:none" class="evidenceContainer"><p class="attributionExplain"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">&#xd;
&lt;p>Manually curated information that is based on statements in scientific articles for which there is no experimental support.&lt;/p>&#xd;
&#xd;
&#xd;
	&lt;p>&lt;a href="/manual/evidences#ECO:0000303">More…&lt;/a>&lt;/p>      </span>Manual assertion based on opinion in<sup>i</sup></span></p><ul><li><div class="P43121#ref4 referenceAttribution"><div class="reference_header">Ref.4</div><div class="reference_content"><div property="schema:citation" resource="http://purl.uniprot.org/citations/14702039" typeof="schema:ScholarlyArticle"><strong property="schema:name">"Complete sequencing and characterization of 21,243 full-length human cDNAs."</strong><br/><a href="/uniprot/?query=author:%22Ota+T.%22" rel="nofollow">Ota T.</a>, <a href="/uniprot/?query=author:%22Suzuki+Y.%22" rel="nofollow">Suzuki Y.</a>, <a href="/uniprot/?query=author:%22Nishikawa+T.%22" rel="nofollow">Nishikawa T.</a>, <a href="/uniprot/?query=author:%22Otsuki+T.%22" rel="nofollow">Otsuki T.</a>, <a href="/uniprot/?query=author:%22Sugiyama+T.%22" rel="nofollow">Sugiyama T.</a>, <a href="/uniprot/?query=author:%22Irie+R.%22" rel="nofollow">Irie R.</a>, <a href="/uniprot/?query=author:%22Wakamatsu+A.%22" rel="nofollow">Wakamatsu A.</a>, <a href="/uniprot/?query=author:%22Hayashi+K.%22" rel="nofollow">Hayashi K.</a>, <a href="/uniprot/?query=author:%22Sato+H.%22" rel="nofollow">Sato H.</a>, <a href="/uniprot/?query=author:%22Nagai+K.%22" rel="nofollow">Nagai K.</a>, <a href="/uniprot/?query=author:%22Kimura+K.%22" rel="nofollow">Kimura K.</a>, <a href="/uniprot/?query=author:%22Makita+H.%22" rel="nofollow">Makita H.</a>, <a href="/uniprot/?query=author:%22Sekine+M.%22" rel="nofollow">Sekine M.</a>, <a href="/uniprot/?query=author:%22Obayashi+M.%22" rel="nofollow">Obayashi M.</a>, <a href="/uniprot/?query=author:%22Nishi+T.%22" rel="nofollow">Nishi T.</a>, <a href="/uniprot/?query=author:%22Shibahara+T.%22" rel="nofollow">Shibahara T.</a>, <a href="/uniprot/?query=author:%22Tanaka+T.%22" rel="nofollow">Tanaka T.</a>, <a href="/uniprot/?query=author:%22Ishii+S.%22" rel="nofollow">Ishii S.</a><span> <div class="clickable authors icon_expand" title="Show all authors"></div> </span><span style="display:none">, <a href="/uniprot/?query=author:%22Yamamoto+J.%22" rel="nofollow">Yamamoto J.</a>, <a href="/uniprot/?query=author:%22Saito+K.%22" rel="nofollow">Saito K.</a>, <a href="/uniprot/?query=author:%22Kawai+Y.%22" rel="nofollow">Kawai Y.</a>, <a href="/uniprot/?query=author:%22Isono+Y.%22" rel="nofollow">Isono Y.</a>, <a href="/uniprot/?query=author:%22Nakamura+Y.%22" rel="nofollow">Nakamura Y.</a>, <a href="/uniprot/?query=author:%22Nagahari+K.%22" rel="nofollow">Nagahari K.</a>, <a href="/uniprot/?query=author:%22Murakami+K.%22" rel="nofollow">Murakami K.</a>, <a href="/uniprot/?query=author:%22Yasuda+T.%22" rel="nofollow">Yasuda T.</a>, <a href="/uniprot/?query=author:%22Iwayanagi+T.%22" rel="nofollow">Iwayanagi T.</a>, <a href="/uniprot/?query=author:%22Wagatsuma+M.%22" rel="nofollow">Wagatsuma M.</a>, <a href="/uniprot/?query=author:%22Shiratori+A.%22" rel="nofollow">Shiratori A.</a>, <a href="/uniprot/?query=author:%22Sudo+H.%22" rel="nofollow">Sudo H.</a>, <a href="/uniprot/?query=author:%22Hosoiri+T.%22" rel="nofollow">Hosoiri T.</a>, <a href="/uniprot/?query=author:%22Kaku+Y.%22" rel="nofollow">Kaku Y.</a>, <a href="/uniprot/?query=author:%22Kodaira+H.%22" rel="nofollow">Kodaira H.</a>, <a href="/uniprot/?query=author:%22Kondo+H.%22" rel="nofollow">Kondo H.</a>, <a href="/uniprot/?query=author:%22Sugawara+M.%22" rel="nofollow">Sugawara M.</a>, <a href="/uniprot/?query=author:%22Takahashi+M.%22" rel="nofollow">Takahashi M.</a>, <a href="/uniprot/?query=author:%22Kanda+K.%22" rel="nofollow">Kanda K.</a>, <a href="/uniprot/?query=author:%22Yokoi+T.%22" rel="nofollow">Yokoi T.</a>, <a href="/uniprot/?query=author:%22Furuya+T.%22" rel="nofollow">Furuya T.</a>, <a href="/uniprot/?query=author:%22Kikkawa+E.%22" rel="nofollow">Kikkawa E.</a>, <a href="/uniprot/?query=author:%22Omura+Y.%22" rel="nofollow">Omura Y.</a>, <a href="/uniprot/?query=author:%22Abe+K.%22" rel="nofollow">Abe K.</a>, <a href="/uniprot/?query=author:%22Kamihara+K.%22" rel="nofollow">Kamihara K.</a>, <a href="/uniprot/?query=author:%22Katsuta+N.%22" rel="nofollow">Katsuta N.</a>, <a href="/uniprot/?query=author:%22Sato+K.%22" rel="nofollow">Sato K.</a>, <a href="/uniprot/?query=author:%22Tanikawa+M.%22" rel="nofollow">Tanikawa M.</a>, <a href="/uniprot/?query=author:%22Yamazaki+M.%22" rel="nofollow">Yamazaki M.</a>, <a href="/uniprot/?query=author:%22Ninomiya+K.%22" rel="nofollow">Ninomiya K.</a>, <a href="/uniprot/?query=author:%22Ishibashi+T.%22" rel="nofollow">Ishibashi T.</a>, <a href="/uniprot/?query=author:%22Yamashita+H.%22" rel="nofollow">Yamashita H.</a>, <a href="/uniprot/?query=author:%22Murakawa+K.%22" rel="nofollow">Murakawa K.</a>, <a href="/uniprot/?query=author:%22Fujimori+K.%22" rel="nofollow">Fujimori K.</a>, <a href="/uniprot/?query=author:%22Tanai+H.%22" rel="nofollow">Tanai H.</a>, <a href="/uniprot/?query=author:%22Kimata+M.%22" rel="nofollow">Kimata M.</a>, <a href="/uniprot/?query=author:%22Watanabe+M.%22" rel="nofollow">Watanabe M.</a>, <a href="/uniprot/?query=author:%22Hiraoka+S.%22" rel="nofollow">Hiraoka S.</a>, <a href="/uniprot/?query=author:%22Chiba+Y.%22" rel="nofollow">Chiba Y.</a>, <a href="/uniprot/?query=author:%22Ishida+S.%22" rel="nofollow">Ishida S.</a>, <a href="/uniprot/?query=author:%22Ono+Y.%22" rel="nofollow">Ono Y.</a>, <a href="/uniprot/?query=author:%22Takiguchi+S.%22" rel="nofollow">Takiguchi S.</a>, <a href="/uniprot/?query=author:%22Watanabe+S.%22" rel="nofollow">Watanabe S.</a>, <a href="/uniprot/?query=author:%22Yosida+M.%22" rel="nofollow">Yosida M.</a>, <a href="/uniprot/?query=author:%22Hotuta+T.%22" rel="nofollow">Hotuta T.</a>, <a href="/uniprot/?query=author:%22Kusano+J.%22" rel="nofollow">Kusano J.</a>, <a href="/uniprot/?query=author:%22Kanehori+K.%22" rel="nofollow">Kanehori K.</a>, <a href="/uniprot/?query=author:%22Takahashi-Fujii+A.%22" rel="nofollow">Takahashi-Fujii A.</a>, <a href="/uniprot/?query=author:%22Hara+H.%22" rel="nofollow">Hara H.</a>, <a href="/uniprot/?query=author:%22Tanase+T.-O.%22" rel="nofollow">Tanase T.-O.</a>, <a href="/uniprot/?query=author:%22Nomura+Y.%22" rel="nofollow">Nomura Y.</a>, <a href="/uniprot/?query=author:%22Togiya+S.%22" rel="nofollow">Togiya S.</a>, <a href="/uniprot/?query=author:%22Komai+F.%22" rel="nofollow">Komai F.</a>, <a href="/uniprot/?query=author:%22Hara+R.%22" rel="nofollow">Hara R.</a>, <a href="/uniprot/?query=author:%22Takeuchi+K.%22" rel="nofollow">Takeuchi K.</a>, <a href="/uniprot/?query=author:%22Arita+M.%22" rel="nofollow">Arita M.</a>, <a href="/uniprot/?query=author:%22Imose+N.%22" rel="nofollow">Imose N.</a>, <a href="/uniprot/?query=author:%22Musashino+K.%22" rel="nofollow">Musashino K.</a>, <a href="/uniprot/?query=author:%22Yuuki+H.%22" rel="nofollow">Yuuki H.</a>, <a href="/uniprot/?query=author:%22Oshima+A.%22" rel="nofollow">Oshima A.</a>, <a href="/uniprot/?query=author:%22Sasaki+N.%22" rel="nofollow">Sasaki N.</a>, <a href="/uniprot/?query=author:%22Aotsuka+S.%22" rel="nofollow">Aotsuka S.</a>, <a href="/uniprot/?query=author:%22Yoshikawa+Y.%22" rel="nofollow">Yoshikawa Y.</a>, <a href="/uniprot/?query=author:%22Matsunawa+H.%22" rel="nofollow">Matsunawa H.</a>, <a href="/uniprot/?query=author:%22Ichihara+T.%22" rel="nofollow">Ichihara T.</a>, <a href="/uniprot/?query=author:%22Shiohata+N.%22" rel="nofollow">Shiohata N.</a>, <a href="/uniprot/?query=author:%22Sano+S.%22" rel="nofollow">Sano S.</a>, <a href="/uniprot/?query=author:%22Moriya+S.%22" rel="nofollow">Moriya S.</a>, <a href="/uniprot/?query=author:%22Momiyama+H.%22" rel="nofollow">Momiyama H.</a>, <a href="/uniprot/?query=author:%22Satoh+N.%22" rel="nofollow">Satoh N.</a>, <a href="/uniprot/?query=author:%22Takami+S.%22" rel="nofollow">Takami S.</a>, <a href="/uniprot/?query=author:%22Terashima+Y.%22" rel="nofollow">Terashima Y.</a>, <a href="/uniprot/?query=author:%22Suzuki+O.%22" rel="nofollow">Suzuki O.</a>, <a href="/uniprot/?query=author:%22Nakagawa+S.%22" rel="nofollow">Nakagawa S.</a>, <a href="/uniprot/?query=author:%22Senoh+A.%22" rel="nofollow">Senoh A.</a>, <a href="/uniprot/?query=author:%22Mizoguchi+H.%22" rel="nofollow">Mizoguchi H.</a>, <a href="/uniprot/?query=author:%22Goto+Y.%22" rel="nofollow">Goto Y.</a>, <a href="/uniprot/?query=author:%22Shimizu+F.%22" rel="nofollow">Shimizu F.</a>, <a href="/uniprot/?query=author:%22Wakebe+H.%22" rel="nofollow">Wakebe H.</a>, <a href="/uniprot/?query=author:%22Hishigaki+H.%22" rel="nofollow">Hishigaki H.</a>, <a href="/uniprot/?query=author:%22Watanabe+T.%22" rel="nofollow">Watanabe T.</a>, <a href="/uniprot/?query=author:%22Sugiyama+A.%22" rel="nofollow">Sugiyama A.</a>, <a href="/uniprot/?query=author:%22Takemoto+M.%22" rel="nofollow">Takemoto M.</a>, <a href="/uniprot/?query=author:%22Kawakami+B.%22" rel="nofollow">Kawakami B.</a>, <a href="/uniprot/?query=author:%22Yamazaki+M.%22" rel="nofollow">Yamazaki M.</a>, <a href="/uniprot/?query=author:%22Watanabe+K.%22" rel="nofollow">Watanabe K.</a>, <a href="/uniprot/?query=author:%22Kumagai+A.%22" rel="nofollow">Kumagai A.</a>, <a href="/uniprot/?query=author:%22Itakura+S.%22" rel="nofollow">Itakura S.</a>, <a href="/uniprot/?query=author:%22Fukuzumi+Y.%22" rel="nofollow">Fukuzumi Y.</a>, <a href="/uniprot/?query=author:%22Fujimori+Y.%22" rel="nofollow">Fujimori Y.</a>, <a href="/uniprot/?query=author:%22Komiyama+M.%22" rel="nofollow">Komiyama M.</a>, <a href="/uniprot/?query=author:%22Tashiro+H.%22" rel="nofollow">Tashiro H.</a>, <a href="/uniprot/?query=author:%22Tanigami+A.%22" rel="nofollow">Tanigami A.</a>, <a href="/uniprot/?query=author:%22Fujiwara+T.%22" rel="nofollow">Fujiwara T.</a>, <a href="/uniprot/?query=author:%22Ono+T.%22" rel="nofollow">Ono T.</a>, <a href="/uniprot/?query=author:%22Yamada+K.%22" rel="nofollow">Yamada K.</a>, <a href="/uniprot/?query=author:%22Fujii+Y.%22" rel="nofollow">Fujii Y.</a>, <a href="/uniprot/?query=author:%22Ozaki+K.%22" rel="nofollow">Ozaki K.</a>, <a href="/uniprot/?query=author:%22Hirao+M.%22" rel="nofollow">Hirao M.</a>, <a href="/uniprot/?query=author:%22Ohmori+Y.%22" rel="nofollow">Ohmori Y.</a>, <a href="/uniprot/?query=author:%22Kawabata+A.%22" rel="nofollow">Kawabata A.</a>, <a href="/uniprot/?query=author:%22Hikiji+T.%22" rel="nofollow">Hikiji T.</a>, <a href="/uniprot/?query=author:%22Kobatake+N.%22" rel="nofollow">Kobatake N.</a>, <a href="/uniprot/?query=author:%22Inagaki+H.%22" rel="nofollow">Inagaki H.</a>, <a href="/uniprot/?query=author:%22Ikema+Y.%22" rel="nofollow">Ikema Y.</a>, <a href="/uniprot/?query=author:%22Okamoto+S.%22" rel="nofollow">Okamoto S.</a>, <a href="/uniprot/?query=author:%22Okitani+R.%22" rel="nofollow">Okitani R.</a>, <a href="/uniprot/?query=author:%22Kawakami+T.%22" rel="nofollow">Kawakami T.</a>, <a href="/uniprot/?query=author:%22Noguchi+S.%22" rel="nofollow">Noguchi S.</a>, <a href="/uniprot/?query=author:%22Itoh+T.%22" rel="nofollow">Itoh T.</a>, <a href="/uniprot/?query=author:%22Shigeta+K.%22" rel="nofollow">Shigeta K.</a>, <a href="/uniprot/?query=author:%22Senba+T.%22" rel="nofollow">Senba T.</a>, <a href="/uniprot/?query=author:%22Matsumura+K.%22" rel="nofollow">Matsumura K.</a>, <a href="/uniprot/?query=author:%22Nakajima+Y.%22" rel="nofollow">Nakajima Y.</a>, <a href="/uniprot/?query=author:%22Mizuno+T.%22" rel="nofollow">Mizuno T.</a>, <a href="/uniprot/?query=author:%22Morinaga+M.%22" rel="nofollow">Morinaga M.</a>, <a href="/uniprot/?query=author:%22Sasaki+M.%22" rel="nofollow">Sasaki M.</a>, <a href="/uniprot/?query=author:%22Togashi+T.%22" rel="nofollow">Togashi T.</a>, <a href="/uniprot/?query=author:%22Oyama+M.%22" rel="nofollow">Oyama M.</a>, <a href="/uniprot/?query=author:%22Hata+H.%22" rel="nofollow">Hata H.</a>, <a href="/uniprot/?query=author:%22Watanabe+M.%22" rel="nofollow">Watanabe M.</a>, <a href="/uniprot/?query=author:%22Komatsu+T.%22" rel="nofollow">Komatsu T.</a>, <a href="/uniprot/?query=author:%22Mizushima-Sugano+J.%22" rel="nofollow">Mizushima-Sugano J.</a>, <a href="/uniprot/?query=author:%22Satoh+T.%22" rel="nofollow">Satoh T.</a>, <a href="/uniprot/?query=author:%22Shirai+Y.%22" rel="nofollow">Shirai Y.</a>, <a href="/uniprot/?query=author:%22Takahashi+Y.%22" rel="nofollow">Takahashi Y.</a>, <a href="/uniprot/?query=author:%22Nakagawa+K.%22" rel="nofollow">Nakagawa K.</a>, <a href="/uniprot/?query=author:%22Okumura+K.%22" rel="nofollow">Okumura K.</a>, <a href="/uniprot/?query=author:%22Nagase+T.%22" rel="nofollow">Nagase T.</a>, <a href="/uniprot/?query=author:%22Nomura+N.%22" rel="nofollow">Nomura N.</a>, <a href="/uniprot/?query=author:%22Kikuchi+H.%22" rel="nofollow">Kikuchi H.</a>, <a href="/uniprot/?query=author:%22Masuho+Y.%22" rel="nofollow">Masuho Y.</a>, <a href="/uniprot/?query=author:%22Yamashita+R.%22" rel="nofollow">Yamashita R.</a>, <a href="/uniprot/?query=author:%22Nakai+K.%22" rel="nofollow">Nakai K.</a>, <a href="/uniprot/?query=author:%22Yada+T.%22" rel="nofollow">Yada T.</a>, <a href="/uniprot/?query=author:%22Nakamura+Y.%22" rel="nofollow">Nakamura Y.</a>, <a href="/uniprot/?query=author:%22Ohara+O.%22" rel="nofollow">Ohara O.</a>, <a href="/uniprot/?query=author:%22Isogai+T.%22" rel="nofollow">Isogai T.</a>, </span><a href="/uniprot/?query=author:%22Sugano+S.%22" rel="nofollow">Sugano S.</a><br/><a href="http://dx.doi.org/10.1038/ng1285">Nat. Genet. 36:40-45(2004)</a>  [<a property="schema:sameAs" href="https://www.ncbi.nlm.nih.gov/pubmed/14702039">PubMed</a>] [<a property="schema:sameAs" href="http://europepmc.org/abstract/MED/14702039">Europe PMC</a>] [<a href="/citations/14702039">Abstract</a>]</div><div class="citedFor"><span class="details"><strong>Cited for:</strong></span> NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA] (ISOFORM 2).</div></div></div></li></ul></span></span><span class="featactions"><a class="icon icon-static tooltipped subsequences" data-icon="b" id="P43121_549-646_" title="Add to basket">Add</a> <a class="icon icon-functional tooltipped blastSubsequences" data-icon="t" title="View this subsequence or run a BLAST search on it.">BLAST</a></span></td><td class="ft-sequence"><svg class="featureImage"><rect x="84.829%" width="15.170%" height="100%" class="featureRect"></rect></svg></td><td class="featlength numeric">98</td></tr></table><h4 id="section_x-ref_sequence" class="xref_category">Sequence databases</h4><table class="databaseTable SEQUENCE"><tr><td><span class="emblRadioContainer">Select the link destinations:<br/><input type="radio" name="insd-target-selector-sequences" value="EMBL" onclick="uniprot.entryTags.configureLink('insd', 'embl',this); uniprot.entryTags.configureLink('insd_cds', 'embl_cds',this);showDatabaseTarget(this);" checked="checked" class="embl"/><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>EMBL nucleotide sequence database</p><a href="/database/DB-0022">More...</a></span>EMBL<sup>i</sup></span><br/><input type="radio" name="insd-target-selector-sequences" value="GenBank" onclick="uniprot.entryTags.configureLink('insd', 'genbank',this); uniprot.entryTags.configureLink('insd_cds', 'genbank_cds',this);showDatabaseTarget(this);" class="genbank"/><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>GenBank nucleotide sequence database</p><a href="/database/DB-0028">More...</a></span>GenBank<sup>i</sup></span><br/><input type="radio" name="insd-target-selector-sequences" value="DDBJ" onclick="uniprot.entryTags.configureLink('insd', 'ddbj',this); uniprot.entryTags.configureLink('insd_cds', 'ddbj_cds',this);showDatabaseTarget(this);" class="ddbj"/><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>DNA Data Bank of Japan; a nucleotide sequence database</p><a href="/database/DB-0014">More...</a></span>DDBJ<sup>i</sup></span><div class="updatedFlasher">Links Updated</div></span></td><td><a href="http://www.ebi.ac.uk/ena/data/view/M29277" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">M29277</a> mRNA. Translation: <a href="http://www.ebi.ac.uk/ena/data/view/AAA20824" class="embl_cds" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL-CDS');">AAA20824.1</a>.<br/><a href="http://www.ebi.ac.uk/ena/data/view/M28882" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">M28882</a> mRNA. Translation: <a href="http://www.ebi.ac.uk/ena/data/view/AAA20922" class="embl_cds" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL-CDS');">AAA20922.1</a>.<br/><a href="http://www.ebi.ac.uk/ena/data/view/X68264" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">X68264</a><span> <div id="embl_img1_-987646653" class="clickable icon_expand" title="Show all EMBL-ACs" onclick="var regexC=/collapse/; if(regexC.test(document.getElementById('embl_img1_-987646653').className)){document.getElementById('embl_img1_-987646653').className='icon_expand';document.getElementById('embl_img1_-987646653').title='Show all EMBL-ACs';$(this).parent().next().hide(); return;}; var regexE=/expand/; if(regexE.test(document.getElementById('embl_img1_-987646653').className)){document.getElementById('embl_img1_-987646653').className='icon_collapse'; document.getElementById('embl_img1_-987646653').title='Hide EMBL-ACs';$(this).parent().next().show(); return;};"></div> </span><span style="display:none">, <a href="http://www.ebi.ac.uk/ena/data/view/X68265" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">X68265</a>, <a href="http://www.ebi.ac.uk/ena/data/view/X68266" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">X68266</a>, <a href="http://www.ebi.ac.uk/ena/data/view/X68267" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">X68267</a>, <a href="http://www.ebi.ac.uk/ena/data/view/X68268" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">X68268</a>, <a href="http://www.ebi.ac.uk/ena/data/view/X68270" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">X68270</a>, </span><a href="http://www.ebi.ac.uk/ena/data/view/X68271" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">X68271</a> Genomic DNA. Translation: <a href="http://www.ebi.ac.uk/ena/data/view/CAA48332" class="embl_cds" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL-CDS');">CAA48332.1</a>.<br/><a href="http://www.ebi.ac.uk/ena/data/view/AF089868" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">AF089868</a> mRNA. Translation: <a href="http://www.ebi.ac.uk/ena/data/view/AAD17799" class="embl_cds" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL-CDS');">AAD17799.1</a>.<br/><a href="http://www.ebi.ac.uk/ena/data/view/AK126303" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">AK126303</a> mRNA. Translation: <a href="http://www.ebi.ac.uk/ena/data/view/BAC86520" class="embl_cds" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL-CDS');">BAC86520.1</a>.<br/><a href="http://www.ebi.ac.uk/ena/data/view/AB209925" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">AB209925</a> mRNA. Translation: <a href="http://www.ebi.ac.uk/ena/data/view/BAD93162" class="embl_cds" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL-CDS');">BAD93162.1</a>.  Different initiation.<br/><a href="http://www.ebi.ac.uk/ena/data/view/BC056418" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">BC056418</a> mRNA. Translation: <a href="http://www.ebi.ac.uk/ena/data/view/AAH56418" class="embl_cds" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL-CDS');">AAH56418.1</a>.<br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>The Consensus CDS (CCDS) project</p><a href="/database/DB-0187">More...</a></span>CCDS<sup>i</sup></span></td><td><a href="https://www.ncbi.nlm.nih.gov/CCDS/CcdsBrowse.cgi?REQUEST=CCDS&amp;GO=MainBrowse&amp;DATA=CCDS31690.1" onclick="uniprot.analytics('DR-lines', 'click', 'DR-CCDS');">CCDS31690.1.</a>   [<a href="/uniprot/P43121#P43121-1" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Isoform P43121-1');">P43121-1</a>]<br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Protein sequence database of the Protein Information Resource</p><a href="/database/DB-0078">More...</a></span>PIR<sup>i</sup></span></td><td><a href="http://pir.georgetown.edu/cgi-bin/nbrfget?uid=I38049" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PIR');">I38049.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>NCBI Reference Sequences</p><a href="/database/DB-0117">More...</a></span>RefSeq<sup>i</sup></span></td><td><a href="https://www.ncbi.nlm.nih.gov/protein/NP_006491.2" onclick="uniprot.analytics('DR-lines', 'click', 'DR-RefSeq');">NP_006491.2.</a> <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_006500.2">NM_006500.2. </a>  [<a href="/uniprot/P43121#P43121-1" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Isoform P43121-1');">P43121-1</a>]<br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>UniGene gene-oriented nucleotide sequence clusters</p><a href="/database/DB-0107">More...</a></span>UniGene<sup>i</sup></span></td><td><a href="https://www.ncbi.nlm.nih.gov/UniGene/clust.cgi?ORG=Hs&amp;CID=599039" onclick="uniprot.analytics('DR-lines', 'click', 'DR-UniGene');">Hs.599039.</a> <br/></td></tr></table><h4 id="section_x-ref_genome" class="xref_category">Genome annotation databases</h4><table class="databaseTable GENOME"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Ensembl eukaryotic genome annotation project</p><a href="/database/DB-0023">More...</a></span>Ensembl<sup>i</sup></span></td><td><a href="http://www.ensembl.org/id/ENST00000264036" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Ensembl-Transcript');">ENST00000264036</a>; <a href="http://www.ensembl.org/id/ENSP00000264036" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Ensembl-Protein');">ENSP00000264036</a>; <a href="http://www.ensembl.org/id/ENSG00000076706" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Ensembl-Gene');">ENSG00000076706</a>.   [<a href="/uniprot/P43121#P43121-1" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Isoform P43121-1');">P43121-1</a>]<br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Database of genes from NCBI RefSeq genomes</p><a href="/database/DB-0118">More...</a></span>GeneID<sup>i</sup></span></td><td><a href="https://www.ncbi.nlm.nih.gov/gene/4162" onclick="uniprot.analytics('DR-lines', 'click', 'DR-GeneID');">4162.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>KEGG: Kyoto Encyclopedia of Genes and Genomes</p><a href="/database/DB-0053">More...</a></span>KEGG<sup>i</sup></span></td><td><a href="http://www.genome.jp/dbget-bin/www_bget?hsa:4162" onclick="uniprot.analytics('DR-lines', 'click', 'DR-KEGG');">hsa:4162.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>UCSC genome browser</p><a href="/database/DB-0139">More...</a></span>UCSC<sup>i</sup></span></td><td><a href="http://genome.ucsc.edu/cgi-bin/hgGene?hgg_gene=uc001pwf.4&amp;org=human" onclick="uniprot.analytics('DR-lines', 'click', 'DR-UCSC');">uc001pwf.4.</a> human.   [<a href="/uniprot/P43121#P43121-1" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Isoform P43121-1');">P43121-1</a>]<br/></td></tr></table><h4><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>UniProtKB Keywords constitute a &lt;a href="http://www.uniprot.org/keywords">controlled vocabulary&lt;/a> with a hierarchical structure. Keywords summarise the content of a UniProtKB entry and facilitate the search for proteins of interest.&lt;p>&lt;a href='/help/keywords' target='_top'>More...&lt;/a>&lt;/p></span>Keywords - Coding sequence diversity<sup>i</sup></span></h4><span><a href="/keywords/KW-0025">Alternative splicing</a>, <a href="/keywords/KW-0621">Polymorphism</a></span></div><div class="section " id="cross_references"><h2><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This section is used to point to information related to entries and found in data collections other than UniProtKB.&lt;p>&lt;a href='/help/cross_references_section' target='_top'>More...&lt;/a>&lt;/p></span>Cross-references<sup>i</sup></span></h2><h4><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the &lt;a href="http://www.uniprot.org/manual/cross_references_section">Cross-references&lt;/a> section provides links to various web resources that are relevant for a specific protein.&lt;p>&lt;a href='/help/web_resource' target='_top'>More...&lt;/a>&lt;/p></span>Web resources<sup>i</sup></span></h4><table><tbody><tr><td><a href="http://atlasgeneticsoncology.org/Genes/MCAMID41314ch11q23.html">Atlas of Genetics and Cytogenetics in Oncology and Haematology </a></td></tr></tbody></table><h4 id="section_x-ref_sequence" class="xref_category">Sequence databases</h4><table class="databaseTable SEQUENCE"><tr><td><span class="emblRadioContainer">Select the link destinations:<br/><input type="radio" name="insd-target-selector-cross_references" value="EMBL" onclick="uniprot.entryTags.configureLink('insd', 'embl',this); uniprot.entryTags.configureLink('insd_cds', 'embl_cds',this);showDatabaseTarget(this);" checked="checked" class="embl"/><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>EMBL nucleotide sequence database</p><a href="/database/DB-0022">More...</a></span>EMBL<sup>i</sup></span><br/><input type="radio" name="insd-target-selector-cross_references" value="GenBank" onclick="uniprot.entryTags.configureLink('insd', 'genbank',this); uniprot.entryTags.configureLink('insd_cds', 'genbank_cds',this);showDatabaseTarget(this);" class="genbank"/><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>GenBank nucleotide sequence database</p><a href="/database/DB-0028">More...</a></span>GenBank<sup>i</sup></span><br/><input type="radio" name="insd-target-selector-cross_references" value="DDBJ" onclick="uniprot.entryTags.configureLink('insd', 'ddbj',this); uniprot.entryTags.configureLink('insd_cds', 'ddbj_cds',this);showDatabaseTarget(this);" class="ddbj"/><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>DNA Data Bank of Japan; a nucleotide sequence database</p><a href="/database/DB-0014">More...</a></span>DDBJ<sup>i</sup></span><div class="updatedFlasher">Links Updated</div></span></td><td><a href="http://www.ebi.ac.uk/ena/data/view/M29277" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">M29277</a> mRNA. Translation: <a href="http://www.ebi.ac.uk/ena/data/view/AAA20824" class="embl_cds" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL-CDS');">AAA20824.1</a>.<br/><a href="http://www.ebi.ac.uk/ena/data/view/M28882" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">M28882</a> mRNA. Translation: <a href="http://www.ebi.ac.uk/ena/data/view/AAA20922" class="embl_cds" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL-CDS');">AAA20922.1</a>.<br/><a href="http://www.ebi.ac.uk/ena/data/view/X68264" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">X68264</a><span> <div id="embl_img1_1635687864" class="clickable icon_expand" title="Show all EMBL-ACs" onclick="var regexC=/collapse/; if(regexC.test(document.getElementById('embl_img1_1635687864').className)){document.getElementById('embl_img1_1635687864').className='icon_expand';document.getElementById('embl_img1_1635687864').title='Show all EMBL-ACs';$(this).parent().next().hide(); return;}; var regexE=/expand/; if(regexE.test(document.getElementById('embl_img1_1635687864').className)){document.getElementById('embl_img1_1635687864').className='icon_collapse'; document.getElementById('embl_img1_1635687864').title='Hide EMBL-ACs';$(this).parent().next().show(); return;};"></div> </span><span style="display:none">, <a href="http://www.ebi.ac.uk/ena/data/view/X68265" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">X68265</a>, <a href="http://www.ebi.ac.uk/ena/data/view/X68266" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">X68266</a>, <a href="http://www.ebi.ac.uk/ena/data/view/X68267" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">X68267</a>, <a href="http://www.ebi.ac.uk/ena/data/view/X68268" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">X68268</a>, <a href="http://www.ebi.ac.uk/ena/data/view/X68270" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">X68270</a>, </span><a href="http://www.ebi.ac.uk/ena/data/view/X68271" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">X68271</a> Genomic DNA. Translation: <a href="http://www.ebi.ac.uk/ena/data/view/CAA48332" class="embl_cds" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL-CDS');">CAA48332.1</a>.<br/><a href="http://www.ebi.ac.uk/ena/data/view/AF089868" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">AF089868</a> mRNA. Translation: <a href="http://www.ebi.ac.uk/ena/data/view/AAD17799" class="embl_cds" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL-CDS');">AAD17799.1</a>.<br/><a href="http://www.ebi.ac.uk/ena/data/view/AK126303" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">AK126303</a> mRNA. Translation: <a href="http://www.ebi.ac.uk/ena/data/view/BAC86520" class="embl_cds" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL-CDS');">BAC86520.1</a>.<br/><a href="http://www.ebi.ac.uk/ena/data/view/AB209925" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">AB209925</a> mRNA. Translation: <a href="http://www.ebi.ac.uk/ena/data/view/BAD93162" class="embl_cds" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL-CDS');">BAD93162.1</a>.  Different initiation.<br/><a href="http://www.ebi.ac.uk/ena/data/view/BC056418" class="embl" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL');">BC056418</a> mRNA. Translation: <a href="http://www.ebi.ac.uk/ena/data/view/AAH56418" class="embl_cds" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EMBL-CDS');">AAH56418.1</a>.<br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>The Consensus CDS (CCDS) project</p><a href="/database/DB-0187">More...</a></span>CCDS<sup>i</sup></span></td><td><a href="https://www.ncbi.nlm.nih.gov/CCDS/CcdsBrowse.cgi?REQUEST=CCDS&amp;GO=MainBrowse&amp;DATA=CCDS31690.1" onclick="uniprot.analytics('DR-lines', 'click', 'DR-CCDS');">CCDS31690.1.</a>   [<a href="/uniprot/P43121#P43121-1" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Isoform P43121-1');">P43121-1</a>]<br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Protein sequence database of the Protein Information Resource</p><a href="/database/DB-0078">More...</a></span>PIR<sup>i</sup></span></td><td><a href="http://pir.georgetown.edu/cgi-bin/nbrfget?uid=I38049" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PIR');">I38049.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>NCBI Reference Sequences</p><a href="/database/DB-0117">More...</a></span>RefSeq<sup>i</sup></span></td><td><a href="https://www.ncbi.nlm.nih.gov/protein/NP_006491.2" onclick="uniprot.analytics('DR-lines', 'click', 'DR-RefSeq');">NP_006491.2.</a> <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_006500.2">NM_006500.2. </a>  [<a href="/uniprot/P43121#P43121-1" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Isoform P43121-1');">P43121-1</a>]<br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>UniGene gene-oriented nucleotide sequence clusters</p><a href="/database/DB-0107">More...</a></span>UniGene<sup>i</sup></span></td><td><a href="https://www.ncbi.nlm.nih.gov/UniGene/clust.cgi?ORG=Hs&amp;CID=599039" onclick="uniprot.analytics('DR-lines', 'click', 'DR-UniGene');">Hs.599039.</a> <br/></td></tr></table><h4 id="section_x-ref_structure" class="xref_category">3D structure databases</h4><table class="databaseTable STRUCTURE"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Protein Model Portal of the PSI-Nature Structural Biology Knowledgebase</p><a href="/database/DB-0159">More...</a></span>ProteinModelPortal<sup>i</sup></span></td><td><a href="http://www.proteinmodelportal.org/query/uniprot/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-ProteinModelPortal');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Database of comparative protein structure models</p><a href="/database/DB-0063">More...</a></span>ModBase<sup>i</sup></span></td><td><a href="http://salilab.org/modbase-cgi/model_search.cgi?searchkw=name&amp;kword=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-ModBase');">Search...</a><br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>MobiDB: a database of protein disorder and mobility annotations</p><a href="/database/DB-0183">More...</a></span>MobiDB<sup>i</sup></span></td><td><a href="http://mobidb.bio.unipd.it/entries/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-MobiDB');">Search...</a><br/></td></tr></table><h4 id="section_x-ref_interaction" class="xref_category">Protein-protein interaction databases</h4><table class="databaseTable INTERACTION"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>The Biological General Repository for Interaction Datasets (BioGrid)</p><a href="/database/DB-0184">More...</a></span>BioGrid<sup>i</sup></span></td><td><a href="https://thebiogrid.org/110332" onclick="uniprot.analytics('DR-lines', 'click', 'DR-BioGrid');">110332.</a> 13 interactors.<br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Database of interacting proteins</p><a href="/database/DB-0016">More...</a></span>DIP<sup>i</sup></span></td><td><a href="http://dip.doe-mbi.ucla.edu/dip/Browse.cgi?ID=DIP-52791N" onclick="uniprot.analytics('DR-lines', 'click', 'DR-DIP');">DIP-52791N.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Protein interaction database and analysis system</p><a href="/database/DB-0051">More...</a></span>IntAct<sup>i</sup></span></td><td><a href="https://www.ebi.ac.uk/intact/interactors/id:P43121*" onclick="uniprot.analytics('DR-lines', 'click', 'DR-IntAct');">P43121.</a> 5 interactors.<br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Molecular INTeraction database</p><a href="/database/DB-0158">More...</a></span>MINT<sup>i</sup></span></td><td><a href="http://mint.bio.uniroma2.it/mint/search/search.do?queryType=protein&amp;interactorAc=MINT-1184800" onclick="uniprot.analytics('DR-lines', 'click', 'DR-MINT');">MINT-1184800.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>STRING: functional protein association networks</p><a href="/database/DB-0141">More...</a></span>STRING<sup>i</sup></span></td><td><a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifier=9606.ENSP00000264036" onclick="uniprot.analytics('DR-lines', 'click', 'DR-STRING');">9606.ENSP00000264036.</a> <br/></td></tr></table><h4 id="section_x-ref_ptm" class="xref_category">PTM databases</h4><table class="databaseTable PTM"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>iPTMnet integrated resource for PTMs in systems biology context</p><a href="/database/DB-0200">More...</a></span>iPTMnet<sup>i</sup></span></td><td><a href="http://research.bioinformatics.udel.edu/iptmnet/entry/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-iPTMnet');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Comprehensive resource for the study of protein post-translational modifications (PTMs) in human, mouse and rat.</p><a href="/database/DB-0123">More...</a></span>PhosphoSitePlus<sup>i</sup></span></td><td><a href="http://www.phosphosite.org/uniprotAccAction?id=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PhosphoSitePlus');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>SwissPalm database of S-palmitoylation events</p><a href="/database/DB-0201">More...</a></span>SwissPalm<sup>i</sup></span></td><td><a href="http://swisspalm.epfl.ch/proteins?up_ac=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-SwissPalm');">P43121.</a> <br/></td></tr></table><h4 id="section_x-ref_polymorphism" class="xref_category">Polymorphism and mutation databases</h4><table class="databaseTable POLYMORPHISM"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>BioMuta curated single-nucleotide variation and disease association database</p><a href="/database/DB-0192">More...</a></span>BioMuta<sup>i</sup></span></td><td><a href="https://hive.biochemistry.gwu.edu/tools/biomuta/biomuta.php?gene=MCAM" onclick="uniprot.analytics('DR-lines', 'click', 'DR-BioMuta');">MCAM.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Domain mapping of disease mutations (DMDM)</p><a href="/database/DB-0166">More...</a></span>DMDM<sup>i</sup></span></td><td><a href="http://bioinf.umbc.edu/dmdm/gene_prot_page.php?search_type=protein&amp;id=85681878" onclick="uniprot.analytics('DR-lines', 'click', 'DR-DMDM');">85681878.</a> <br/></td></tr></table><h4 id="section_x-ref_proteomic" class="xref_category">Proteomic databases</h4><table class="databaseTable PROTEOMIC"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Encyclopedia of Proteome Dynamics</p><a href="/database/DB-0205">More...</a></span>EPD<sup>i</sup></span></td><td><a href="https://www.peptracker.com/epd/protein/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-EPD');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>MaxQB - The MaxQuant DataBase</p><a href="/database/DB-0186">More...</a></span>MaxQB<sup>i</sup></span></td><td><a href="http://maxqb.biochem.mpg.de/mxdb/protein/show/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-MaxQB');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>PaxDb, a database of protein abundance averages across all three domains of life</p><a href="/database/DB-0173">More...</a></span>PaxDb<sup>i</sup></span></td><td><a href="http://pax-db.org/#!protein/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PaxDb');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>PeptideAtlas</p><a href="/database/DB-0071">More...</a></span>PeptideAtlas<sup>i</sup></span></td><td><a href="https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/Search?action=GO&amp;search_key=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PeptideAtlas');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>PRoteomics IDEntifications database</p><a href="/database/DB-0130">More...</a></span>PRIDE<sup>i</sup></span></td><td><a href="http://www.ebi.ac.uk/pride/searchSummary.do?queryTypeSelected=identification%20accession%20number&amp;identificationAccessionNumber=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PRIDE');">P43121.</a> <br/></td></tr></table><h4 id="section_x-ref_protocol" class="xref_category">Protocols and materials databases</h4><table class="databaseTable PROTOCOL"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>The DNASU plasmid repository</p><a href="/database/DB-0167">More...</a></span>DNASU<sup>i</sup></span></td><td><a href="http://dnasu.org/DNASU/AdvancedSearchOptions.do?geneName=4162" onclick="uniprot.analytics('DR-lines', 'click', 'DR-DNASU');">4162.</a> <br/></td></tr><tr><td><span>Structural Biology Knowledgebase</span></td><td><a href="http://sbkb.org/uid/P43121/uniprot" onclick="uniprot.analytics('DR-lines', 'click', 'DR-SBKB');">Search...</a><br/></td></tr></table><h4 id="section_x-ref_genome" class="xref_category">Genome annotation databases</h4><table class="databaseTable GENOME"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Ensembl eukaryotic genome annotation project</p><a href="/database/DB-0023">More...</a></span>Ensembl<sup>i</sup></span></td><td><a href="http://www.ensembl.org/id/ENST00000264036" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Ensembl-Transcript');">ENST00000264036</a>; <a href="http://www.ensembl.org/id/ENSP00000264036" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Ensembl-Protein');">ENSP00000264036</a>; <a href="http://www.ensembl.org/id/ENSG00000076706" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Ensembl-Gene');">ENSG00000076706</a>.   [<a href="/uniprot/P43121#P43121-1" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Isoform P43121-1');">P43121-1</a>]<br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Database of genes from NCBI RefSeq genomes</p><a href="/database/DB-0118">More...</a></span>GeneID<sup>i</sup></span></td><td><a href="https://www.ncbi.nlm.nih.gov/gene/4162" onclick="uniprot.analytics('DR-lines', 'click', 'DR-GeneID');">4162.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>KEGG: Kyoto Encyclopedia of Genes and Genomes</p><a href="/database/DB-0053">More...</a></span>KEGG<sup>i</sup></span></td><td><a href="http://www.genome.jp/dbget-bin/www_bget?hsa:4162" onclick="uniprot.analytics('DR-lines', 'click', 'DR-KEGG');">hsa:4162.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>UCSC genome browser</p><a href="/database/DB-0139">More...</a></span>UCSC<sup>i</sup></span></td><td><a href="http://genome.ucsc.edu/cgi-bin/hgGene?hgg_gene=uc001pwf.4&amp;org=human" onclick="uniprot.analytics('DR-lines', 'click', 'DR-UCSC');">uc001pwf.4.</a> human.   [<a href="/uniprot/P43121#P43121-1" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Isoform P43121-1');">P43121-1</a>]<br/></td></tr></table><h4 id="section_x-ref_organism" class="xref_category">Organism-specific databases</h4><table class="databaseTable ORGANISM"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Comparative Toxicogenomics Database</p><a href="/database/DB-0140">More...</a></span>CTD<sup>i</sup></span></td><td><a href="http://ctdbase.org/detail.go?type=gene&amp;db=GENE&amp;acc=4162" onclick="uniprot.analytics('DR-lines', 'click', 'DR-CTD');">4162.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>DisGeNET</p><a href="/database/DB-0218">More...</a></span>DisGeNET<sup>i</sup></span></td><td><a href="http://disgenet.org/search?q=4162" onclick="uniprot.analytics('DR-lines', 'click', 'DR-DisGeNET');">4162.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>GeneCards: human genes, protein and diseases</p><a href="/database/DB-0030">More...</a></span>GeneCards<sup>i</sup></span></td><td><a href="http://www.genecards.org/cgi-bin/carddisp.pl?gene=MCAM" onclick="uniprot.analytics('DR-lines', 'click', 'DR-GeneCards');">MCAM.</a> <br/><a href="http://www.genecards.org/cgi-bin/carddisp.pl?gene=MIR6756" onclick="uniprot.analytics('DR-lines', 'click', 'DR-GeneCards');">MIR6756.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Human Gene Nomenclature Database</p><a href="/database/DB-0042">More...</a></span>HGNC<sup>i</sup></span></td><td><a href="http://www.genenames.org/cgi-bin/gene_symbol_report?hgnc_id=HGNC:6934" onclick="uniprot.analytics('DR-lines', 'click', 'DR-HGNC');">HGNC:6934.</a> MCAM. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Human Protein Atlas</p><a href="/database/DB-0046">More...</a></span>HPA<sup>i</sup></span></td><td><a href="http://www.proteinatlas.org/tissue_profile.php?antibody_id=CAB002147" onclick="uniprot.analytics('DR-lines', 'click', 'DR-HPA');">CAB002147.</a> <br/><a href="http://www.proteinatlas.org/tissue_profile.php?antibody_id=HPA008848" onclick="uniprot.analytics('DR-lines', 'click', 'DR-HPA');">HPA008848.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Online Mendelian Inheritance in Man (OMIM)</p><a href="/database/DB-0062">More...</a></span>MIM<sup>i</sup></span></td><td><a href="http://www.omim.org/entry/155735" onclick="uniprot.analytics('DR-lines', 'click', 'DR-MIM');">155735.</a> gene. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>neXtProt; the human protein knowledge platform</p><a href="/database/DB-0161">More...</a></span>neXtProt<sup>i</sup></span></td><td><a href="https://www.nextprot.org/entry/NX_P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-neXtProt');">NX_P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Open Targets</p><a href="/database/DB-0219">More...</a></span>OpenTargets<sup>i</sup></span></td><td><a href="https://www.targetvalidation.org/target/ENSG00000076706/associations" onclick="uniprot.analytics('DR-lines', 'click', 'DR-OpenTargets');">ENSG00000076706.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>The Pharmacogenetics and Pharmacogenomics Knowledge Base</p><a href="/database/DB-0074">More...</a></span>PharmGKB<sup>i</sup></span></td><td><a href="https://www.pharmgkb.org/gene/PA30678" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PharmGKB');">PA30678.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>GenAtlas: human gene database</p><a href="/database/DB-0027">More...</a></span>GenAtlas<sup>i</sup></span></td><td><a href="http://genatlas.medecine.univ-paris5.fr/fiche.php?symbol=MCAM" onclick="uniprot.analytics('DR-lines', 'click', 'DR-GenAtlas');">Search...</a><br/></td></tr></table><h4 id="section_x-ref_phylogenomic" class="xref_category">Phylogenomic databases</h4><table class="databaseTable PHYLOGENOMIC"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>evolutionary genealogy of genes: Non-supervised Orthologous Groups</p><a href="/database/DB-0152">More...</a></span>eggNOG<sup>i</sup></span></td><td><a href="http://eggnogdb.embl.de/#/app/results?seqid=P43121&amp;target_nogs=ENOG410IEC2" onclick="uniprot.analytics('DR-lines', 'click', 'DR-eggNOG');">ENOG410IEC2.</a> Eukaryota. <br/><a href="http://eggnogdb.embl.de/#/app/results?seqid=P43121&amp;target_nogs=ENOG4110RPG" onclick="uniprot.analytics('DR-lines', 'click', 'DR-eggNOG');">ENOG4110RPG.</a> LUCA. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Ensembl GeneTree</p><a href="/database/DB-0162">More...</a></span>GeneTree<sup>i</sup></span></td><td><a href="http://www.ensemblgenomes.org/id-genetree/ENSGT00530000063457" onclick="uniprot.analytics('DR-lines', 'click', 'DR-GeneTree');">ENSGT00530000063457.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>The HOGENOM Database of Homologous Genes from Fully Sequenced Organisms</p><a href="/database/DB-0044">More...</a></span>HOGENOM<sup>i</sup></span></td><td><a href="http://pbil.univ-lyon1.fr/cgi-bin/acnuc-ac2tree?query=P43121&amp;db=HOGENOM" onclick="uniprot.analytics('DR-lines', 'click', 'DR-HOGENOM');">HOG000015427.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>The HOVERGEN Database of Homologous Vertebrate Genes</p><a href="/database/DB-0045">More...</a></span>HOVERGEN<sup>i</sup></span></td><td><a href="http://pbil.univ-lyon1.fr/cgi-bin/acnuc-ac2tree?query=P43121&amp;db=HOVERGEN" onclick="uniprot.analytics('DR-lines', 'click', 'DR-HOVERGEN');">HBG002808.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>InParanoid: Eukaryotic Ortholog Groups</p><a href="/database/DB-0146">More...</a></span>InParanoid<sup>i</sup></span></td><td><a href="http://inparanoid.sbc.su.se/cgi-bin/gene_search.cgi?id=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InParanoid');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>KEGG Orthology (KO)</p><a href="/database/DB-0164">More...</a></span>KO<sup>i</sup></span></td><td><a href="http://www.genome.jp/dbget-bin/www_bget?ko:K06534" onclick="uniprot.analytics('DR-lines', 'click', 'DR-KO');">K06534.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Identification of Orthologs from Complete Genome Data</p><a href="/database/DB-0137">More...</a></span>OMA<sup>i</sup></span></td><td><a href="http://omabrowser.org/cgi-bin/gateway.pl?f=DisplayGroup&amp;p1=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-OMA');">PSRKSEF.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Database of Orthologous Groups</p><a href="/database/DB-0143">More...</a></span>OrthoDB<sup>i</sup></span></td><td><a href="http://www.orthodb.org/?query=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-OrthoDB');">EOG091G072E.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Database for complete collections of gene phylogenies</p><a href="/database/DB-0144">More...</a></span>PhylomeDB<sup>i</sup></span></td><td><a href="http://phylomedb.org/?q=search_tree&amp;seqid=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PhylomeDB');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>TreeFam database of animal gene trees</p><a href="/database/DB-0185">More...</a></span>TreeFam<sup>i</sup></span></td><td><a href="http://www.treefam.org/family/TF330534" onclick="uniprot.analytics('DR-lines', 'click', 'DR-TreeFam');">TF330534.</a> <br/></td></tr></table><h4 id="section_x-ref_other" class="xref_category">Miscellaneous databases</h4><table class="databaseTable OTHER"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>ChiTaRS: a database of human, mouse and fruit fly chimeric transcripts and RNA-sequencing data</p><a href="/database/DB-0176">More...</a></span>ChiTaRS<sup>i</sup></span></td><td><a href="http://chitars.bioinfo.cnio.es/cgi-bin/search.pl?searchtype=gene_name&amp;searchstr=MCAM&amp;human=1" onclick="uniprot.analytics('DR-lines', 'click', 'DR-ChiTaRS');">MCAM.</a> human. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>The Gene Wiki collection of pages on human genes and proteins</p><a href="/database/DB-0180">More...</a></span>GeneWiki<sup>i</sup></span></td><td><a href="https://en.wikipedia.org/wiki/CD146" onclick="uniprot.analytics('DR-lines', 'click', 'DR-GeneWiki');">CD146.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Database of phenotypes from RNA interference screens in Drosophila and Homo sapiens</p><a href="/database/DB-0169">More...</a></span>GenomeRNAi<sup>i</sup></span></td><td><a href="http://genomernai.org/genedetails/4162" onclick="uniprot.analytics('DR-lines', 'click', 'DR-GenomeRNAi');">4162.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Protein Ontology</p><a href="/database/DB-0181">More...</a></span>PRO<sup>i</sup></span></td><td><a href="http://purl.obolibrary.org/obo/PR_P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PRO');">P43121.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>The Stanford Online Universal Resource for Clones and ESTs</p><a href="/database/DB-0099">More...</a></span>SOURCE<sup>i</sup></span></td><td><a href="https://puma.princeton.edu/cgi-bin/source/sourceResult?criteria=MCAM&amp;choice=Gene&amp;option=Symbol&amp;organism=Hs" onclick="uniprot.analytics('DR-lines', 'click', 'DR-SOURCE');">Search...</a><br/></td></tr></table><h4 id="section_x-ref_expression" class="xref_category">Gene expression databases</h4><table class="databaseTable EXPRESSION"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Bgee dataBase for Gene Expression Evolution</p><a href="/database/DB-0133">More...</a></span>Bgee<sup>i</sup></span></td><td><a href="http://bgee.org/?page=gene&amp;gene_id=ENSG00000076706" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Bgee');">ENSG00000076706.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>CleanEx database of gene expression profiles</p><a href="/database/DB-0008">More...</a></span>CleanEx<sup>i</sup></span></td><td><a href="http://cleanex.vital-it.ch/cgi-bin/get_doc?db=cleanex&amp;format=nice&amp;entry=HS_MCAM" onclick="uniprot.analytics('DR-lines', 'click', 'DR-CleanEx');">HS_MCAM.</a> <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>ExpressionAtlas, Differential and Baseline Expression</p><a href="/database/DB-0004">More...</a></span>ExpressionAtlas<sup>i</sup></span></td><td><a href="http://www.ebi.ac.uk/gxa/query?geneQuery=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-ExpressionAtlas');">P43121.</a> baseline and differential. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Genevisible search portal to normalized and curated expression data from Genevestigator</p><a href="/database/DB-0194">More...</a></span>Genevisible<sup>i</sup></span></td><td><a href="https://genevisible.com/tissues/HS/UniProt/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Genevisible');">P43121.</a> HS. <br/></td></tr></table><h4 id="section_x-ref_domain" class="xref_category">Family and domain databases</h4><table class="databaseTable DOMAIN"><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Gene3D Structural and Functional Annotation of Protein Families</p><a href="/database/DB-0029">More...</a></span>Gene3D<sup>i</sup></span></td><td><a href="http://www.cathdb.info/superfamily/2.60.40.10" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Gene3D');">2.60.40.10.</a> 5 hits. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Integrated resource of protein families, domains and functional sites</p><a href="/database/DB-0052">More...</a></span>InterPro<sup>i</sup></span></td><td><a href="http://www.ebi.ac.uk/interpro/protein/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">View protein in InterPro</a><br/><a href="http://www.ebi.ac.uk/interpro/entry/IPR013162" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">IPR013162.</a> CD80_C2-set. <br/><a href="http://www.ebi.ac.uk/interpro/entry/IPR007110" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">IPR007110.</a> Ig-like_dom. <br/><a href="http://www.ebi.ac.uk/interpro/entry/IPR013783" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">IPR013783.</a> Ig-like_fold. <br/><a href="http://www.ebi.ac.uk/interpro/entry/IPR003599" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">IPR003599.</a> Ig_sub. <br/><a href="http://www.ebi.ac.uk/interpro/entry/IPR003598" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">IPR003598.</a> Ig_sub2. <br/><a href="http://www.ebi.ac.uk/interpro/entry/IPR013106" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">IPR013106.</a> Ig_V-set. <br/><a href="http://www.ebi.ac.uk/interpro/entry/IPR013151" onclick="uniprot.analytics('DR-lines', 'click', 'DR-InterPro');">IPR013151.</a> Immunoglobulin. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Pfam protein domain database</p><a href="/database/DB-0073">More...</a></span>Pfam<sup>i</sup></span></td><td><a href="http://pfam.xfam.org/protein/P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Pfam-Graphical-View');">View protein in Pfam</a><br/><a href="http://pfam.xfam.org/family/PF08205" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Pfam');">PF08205.</a> C2-set_2. 1 hit. <br/><a href="http://pfam.xfam.org/family/PF00047" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Pfam');">PF00047.</a> ig. 1 hit. <br/><a href="http://pfam.xfam.org/family/PF07686" onclick="uniprot.analytics('DR-lines', 'click', 'DR-Pfam');">PF07686.</a> V-set. 1 hit. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Simple Modular Architecture Research Tool; a protein domain database</p><a href="/database/DB-0097">More...</a></span>SMART<sup>i</sup></span></td><td><a href="http://smart.embl.de/smart/show_motifs.pl?ID=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-SMART-Graphical-View');">View protein in SMART</a><br/><a href="http://smart.embl.de/smart/do_annotation.pl?DOMAIN=SM00409" onclick="uniprot.analytics('DR-lines', 'click', 'DR-SMART');">SM00409.</a> IG. 5 hits. <br/><a href="http://smart.embl.de/smart/do_annotation.pl?DOMAIN=SM00408" onclick="uniprot.analytics('DR-lines', 'click', 'DR-SMART');">SM00408.</a> IGc2. 3 hits. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>Superfamily database of structural and functional annotation</p><a href="/database/DB-0155">More...</a></span>SUPFAM<sup>i</sup></span></td><td><a href="http://supfam.org/SUPERFAMILY/cgi-bin/scop.cgi?ipid=SSF48726" onclick="uniprot.analytics('DR-lines', 'click', 'DR-SUPFAM');">SSF48726.</a> SSF48726. 5 hits. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>PROSITE; a protein domain and family database</p><a href="/database/DB-0084">More...</a></span>PROSITE<sup>i</sup></span></td><td><a href="http://prosite.expasy.org/cgi-bin/prosite/PSScan.cgi?seq=P43121&amp;output=nice" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PROSITE-Graphical-View');">View protein in PROSITE</a><br/><a href="http://prosite.expasy.org/doc/PS50835" onclick="uniprot.analytics('DR-lines', 'click', 'DR-PROSITE');">PS50835.</a> IG_LIKE. 5 hits. <br/></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none"><p>ProtoNet; Automatic hierarchical classification of proteins</p><a href="/database/DB-0085">More...</a></span>ProtoNet<sup>i</sup></span></td><td><a href="http://www.protonet.cs.huji.ac.il/sp.php?prot=P43121" onclick="uniprot.analytics('DR-lines', 'click', 'DR-ProtoNet');">Search...</a><br/></td></tr></table></div><div class="section " id="entry_information"><h2><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This section provides general information on the entry.&lt;p>&lt;a href='/help/entry_information_section' target='_top'>More...&lt;/a>&lt;/p></span>Entry information<sup>i</sup></span></h2><table><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Entry information’ section provides a mnemonic identifier for a UniProtKB entry, but it is not a stable identifier. Each reviewed entry is assigned a unique entry name upon integration into UniProtKB/Swiss-Prot.&lt;p>&lt;a href='/help/entry_name' target='_top'>More...&lt;/a>&lt;/p></span>Entry name<sup>i</sup></span></td><td colspan="2">MUC18_HUMAN</td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Entry information’ section provides one or more accession number(s). These are stable identifiers and should be used to cite UniProtKB entries. Upon integration into UniProtKB, each entry is assigned a unique accession number, which is called ‘Primary (citable) accession number’.&lt;p>&lt;a href='/help/accession_numbers' target='_top'>More...&lt;/a>&lt;/p></span>Accession<sup>i</sup></span></td><td colspan="2"><span class="hidden" property="schema:about"><a href="http://purl.uniprot.org/uniprot/P43121">P43121</a></span>Primary (citable) accession number: <strong property="schema:entryID">P43121</strong><br/>Secondary accession number(s): O95812<span class="clickable"> <div id="ac_img1" class="icon_expand secondaryAcs" title="Show all accession numbers"></div> </span><span style="display:none">, Q59E86, Q6PHR3, </span>Q6ZTR2</td></tr><tr id="history"><td rowspan="4"><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Entry information’ section shows the date of integration of the entry into UniProtKB, the date of the last sequence update and the date of the last annotation modification (‘Last modified’). The version number for both the entry and the &lt;a href="http://www.uniprot.org/help/canonical_and_isoforms">canonical sequence&lt;/a> are also displayed.&lt;p>&lt;a href='/help/entry_history' target='_top'>More...&lt;/a>&lt;/p></span>Entry history<sup>i</sup></span></td><td style="width: 18em;">Integrated into UniProtKB/Swiss-Prot: </td><td><span property="schema:dateCreated">November 1, 1995</span></td></tr><tr><td style="width: 18em;">Last sequence update: </td><td>January 10, 2006</td></tr><tr><td style="width: 18em;">Last modified: </td><td><span property="schema:dateModified">March 15, 2017</span></td></tr><tr><td colspan="2">This is version <span property="schema:version">153</span> of the entry and version 2 of the sequence. <a href="/uniprot/P43121?version=*">See complete history.</a></td></tr><tr><td><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This subsection of the ‘Entry information’ section indicates whether the entry has been manually annotated and reviewed by UniProtKB curators or not, in other words, if the entry belongs to the Swiss-Prot section of UniProtKB (&lt;strong>reviewed&lt;/strong>) or to the computer-annotated TrEMBL section (&lt;strong>unreviewed&lt;/strong>).&lt;p>&lt;a href='/help/entry_status' target='_top'>More...&lt;/a>&lt;/p></span>Entry status<sup>i</sup></span></td><td colspan="2">Reviewed (UniProtKB/Swiss-Prot)</td></tr><tr><td><span class="acronym" title="Annotation program in charge of the current entry.">Annotation program</span></td><td colspan="2"><a href="/program/Chordata/" onclick="uniprot.analytics('UniProt-Entry-View', 'project', 'Chordata');">Chordata Protein Annotation Program</a></td></tr><tr class="achtung"><td>Disclaimer</td><td colspan="2">Any medical or genetic information present in this entry is provided for research, educational and informational purposes only. It is not in any way intended to be used as a substitute for professional medical advice, diagnosis, treatment or care.</td></tr></table></div><div class="section " id="miscellaneous"><h2><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This section contains any relevant information that doesn’t fit in any other defined sections&lt;p>&lt;a href='/help/miscellaneous_section' target='_top'>More...&lt;/a>&lt;/p></span>Miscellaneous<sup>i</sup></span></h2><h4><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>UniProtKB Keywords constitute a &lt;a href="http://www.uniprot.org/keywords">controlled vocabulary&lt;/a> with a hierarchical structure. Keywords summarise the content of a UniProtKB entry and facilitate the search for proteins of interest.&lt;p>&lt;a href='/help/keywords' target='_top'>More...&lt;/a>&lt;/p></span>Keywords - Technical term<sup>i</sup></span></h4><span><a href="/keywords/KW-0181">Complete proteome</a>, <a href="/keywords/KW-0903">Direct protein sequencing</a>, <a href="/keywords/KW-1185">Reference proteome</a></span><h4>Documents</h4><ol id="documentList" class="noNumbering"><li><a href="/docs/cdlist">Human cell differentiation molecules</a><br/>
CD nomenclature of surface proteins of human leucocytes and list of entries    </li><li><a href="/docs/humchr11">Human chromosome 11</a><br/>
Human chromosome 11: entries, gene names and cross-references to MIM    </li><li><a href="/docs/humpvar">Human entries with polymorphisms or disease mutations</a><br/>
List of human entries with polymorphisms or disease mutations    </li><li><a href="/docs/humsavar">Human polymorphisms and disease mutations</a><br/>
Index of human polymorphisms and disease mutations    </li><li><a href="/docs/mimtosp">MIM cross-references</a><br/>
Online Mendelian Inheritance in Man (MIM) cross-references in UniProtKB/Swiss-Prot    </li></ol></div><div class="section " id="similar_proteins"><h2><span class="context-help tooltipped-click"><span class="toolTipContent" style="display:none">
&lt;p>This section provides links to the UniProt Reference Clusters (&lt;a href="http://www.uniprot.org/help/uniref">UniRef&lt;/a>). UniRef consists of clusters for UniProtKB sequences (including isoforms) and selected UniParc sequences, in order to obtain complete coverage of the sequence space at resolutions of 100%, 90% and 50% identity.&lt;p>&lt;a href='/help/similar_proteins_section' target='_top'>More...&lt;/a>&lt;/p></span>Similar proteins<sup>i</sup></span></h2><span>Links to similar proteins from the UniProt Reference Clusters (UniRef) at 100%, 90% and 50% sequence identity:</span><table><tr><td><a href="/uniref/?query=uniprot:P43121*&amp;fil=identity:1.0&amp;sort=score">100%</a></td><td>UniRef100 combines identical sequences and sub-fragments with 11 or more residues from any organism into one UniRef entry.</td></tr><tr><td><a href="/uniref/?query=uniprot:P43121*&amp;fil=identity:0.9&amp;sort=score">90%</a></td><td>UniRef90 is built by clustering UniRef100 sequences that have at least 90% sequence identity to, and 80% overlap with, the longest sequence (a.k.a seed sequence).</td></tr><tr><td><a href="/uniref/?query=uniprot:P43121*&amp;fil=identity:0.5&amp;sort=score">50%</a></td><td>UniRef50 is built by clustering UniRef90 seed sequences that have at least 50% sequence identity to, and 80% overlap with, the longest sequence in the cluster.</td></tr></table></div></div></div><a property="schema:author schema:publisher" href="/help/about#UniProt" style="display:none"></a><link href="http://ebi-uniprot.github.io/CDN/protvista/css/main.css" rel="stylesheet"/><script async="async" src="http://ebi-uniprot.github.io/CDN/protvista/protvista.min.js"></script><span id="advanced-Blast"><span class="toolTipContent" style="display:none">&lt;p>This will take you to the BLAST page where you can edit options &lt;/p>&lt;p>&lt;a href="/help/sequence-searches">More..&lt;/a>&lt;/p></span></span></main><footer id="page-footer"><section class="html5-section"><h5>Tools</h5><ul><li><a href="/blast">BLAST</a></li><li><a href="/align">Align</a></li><li><a href="/uploadlists">Retrieve/ID mapping</a></li><li><a href="/peptidesearch">Peptide search</a></li></ul></section><section class="html5-section"><h5>Core data</h5><ul><li><a href="/uniprot">Protein knowledgebase (UniProtKB)</a></li><li><a href="/uniref">Sequence clusters (UniRef)</a></li><li><a href="/uniparc">Sequence archive (UniParc)</a></li><li><a href="/proteomes">Proteomes</a></li></ul></section><section class="html5-section"><h5>Supporting data</h5><ul><li><a href="/citations">Literature citations</a></li><li><a href="/taxonomy">Taxonomy</a></li><li><a href="/keywords">Keywords</a></li><li><a href="/locations">Subcellular locations</a></li><li><a href="/database">Cross-referenced databases</a></li><li><a href="/diseases">Diseases</a></li></ul></section><section class="html5-section"><h5>Information</h5><ul><li><a href="/help/about">About UniProt</a></li><li><a href="/help">Help</a></li><li><a href="/help/?fil=section:faq">FAQ</a></li><li><a href="/help/?fil=section:manual">UniProtKB manual</a></li><li><a href="/help/technical">Technical corner</a></li><li><a href="/help/?query=*&amp;fil=section:biocuration">Expert biocuration</a></li></ul></section><section class="html5-footer small"><div class="uniprotlogo"><img src="/images/logos/uniprot-rgb-optimized.svg"/></div>
			&copy; 2002 &ndash;
			<span property="schema:copyrightYear">2017</span>
			<a accesskey="9" href="/help/about">UniProt Consortium</a> | <a href="/help/license">License &amp; Disclaimer</a><div resource="/help/about#UniProt" id="logos" typeof="schema:NGO" property="schema:creator"><span resource="/help/about#EBI" typeof="schema:Organization" property="schema:member"><span class="hidden" property="schema:name">European Bioinformatics Institute</span><a href="http://www.ebi.ac.uk/" class="ebi" property="schema:url"></a></span><span resource="/help/about#PIR" typeof="schema:Organization" property="schema:member"><span class="hidden" property="schema:name">Protein Information Resource</span><a href="http://pir.georgetown.edu/" class="pir" property="schema:url"></a></span><span resource="/help/about#SIB" typeof="schema:Organization" property="schema:member"><span class="hidden" property="schema:name">SIB Swiss Institute of Bioinformatics</span><a href="http://www.sib.swiss/" class="sib" property="schema:url"></a></span></div></section></footer><script src="https://www.google-analytics.com/analytics.js" async="async"></script><script>
						window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
						ga('create', 'UA-6228219-1', 'auto');
						ga('set', 'dimension1', 'true');
						ga('set', 'dimension2', '9606');
						ga('send', 'pageview');
					</script></body></html>